System.register(["./vendor/svelte-C_apNiMW.js","./vendor/carbon-components-svelte-CB8VNYQX.js","./vendor/carbon-icons-svelte-DkM4cpjm.js"],(function(e){"use strict";var t,a,l,n,_,d,s,r,i,o,c,u,p,v,P,m,A,g,C,O,R,f,$,S,b,h,T,L,j,B,M,D,G,F,N,y,x,U,H,w,I,E,k,X,V,K,Y,z,q,Q,W,Z,J,ee,te,ae,le,ne,_e,de,se,re,ie,oe,ce,ue,pe,ve,Pe,me,Ae,ge,Ce,Oe,Re,fe,$e,Se,be,he,Te,Le,je,Be,Me,De,Ge,Fe,Ne,ye,xe,Ue,He,we,Ie,Ee,ke,Xe,Ve,Ke,Ye,ze,qe,Qe,We,Ze;return{setters:[function(e){t=e.S,a=e.i,l=e.s,n=e.a1,_=e.l,d=e.O,s=e.a,r=e.b,i=e.g,o=e.c,c=e.t,u=e.d,p=e.a5,v=e.B,P=e.e,m=e.G,A=e.H,g=e.I,C=e.y,O=e.K,R=e.T,f=e.U,$=e.L,S=e.D,b=e.M,h=e.z,T=e.Q,L=e.k,j=e.u,B=e.n,M=e.o,D=e.h,G=e.p,F=e.a0,N=e.f,y=e.j,x=e.A,U=e.m,H=e.q,w=e.aa,I=e.ab,E=e.x,k=e.W,X=e.ac,V=e.R,K=e.$,Y=e.v,z=e.Y,q=e.Z,Q=e.F,W=e.ad,Z=e.w,J=e.J},function(e){ee=e.S,te=e.a,ae=e.b,le=e.H,ne=e.c,_e=e.T,de=e.d,se=e.e,re=e.f,ie=e.g,oe=e.h,ce=e.M,ue=e.I,pe=e.i,ve=e.j,Pe=e.k,me=e.C,Ae=e.l,ge=e.m,Ce=e.G,Oe=e.R,Re=e.n,fe=e.B,$e=e.o,Se=e.A,be=e.p,he=e.D,Te=e.P,Le=e.q,je=e.r,Be=e.s,Me=e.t,De=e.u},function(e){Ge=e.C,Fe=e.a,Ne=e.D,ye=e.A,xe=e.S,Ue=e.I,He=e.b,we=e.c,Ie=e.V,Ee=e.d,ke=e.T,Xe=e.G,Ve=e.J,Ke=e.P,Ye=e.e,ze=e.f,qe=e.O,Qe=e.g,We=e.h,Ze=e.i}],execute:function(){function Je(e,t,a){const l=e.slice();return l[4]=t[a],l}function et(e,t,a){const l=e.slice();return l[7]=t[a],l}function tt(e){let t,a;return t=new ee({props:{href:`${e[1](e[4].page)}#${e[4].slug}`,class:e[2](e[4].page),title:e[4].text,text:e[4].text,expanded:!0,$$slots:{default:[nt]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};1&a&&(l.href=`${e[1](e[4].page)}#${e[4].slug}`),1&a&&(l.class=e[2](e[4].page)),1&a&&(l.title=e[4].text),1&a&&(l.text=e[4].text),1025&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function at(e){let t,a;return t=new te({props:{href:`${e[1](e[4].page)}#${e[4].slug}`,class:e[2](e[4].page),title:e[4].text,text:e[4].text}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};1&a&&(l.href=`${e[1](e[4].page)}#${e[4].slug}`),1&a&&(l.class=e[2](e[4].page)),1&a&&(l.title=e[4].text),1&a&&(l.text=e[4].text),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function lt(e){let t,a;return t=new ae({props:{href:`${e[1](e[7].page)}#${e[7].slug}`,class:e[2](e[7].page),title:e[7].text,text:e[7].text}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};1&a&&(l.href=`${e[1](e[7].page)}#${e[7].slug}`),1&a&&(l.class=e[2](e[7].page)),1&a&&(l.title=e[7].text),1&a&&(l.text=e[7].text),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function nt(e){let t,a,l=n(e[4].children),_=[];for(let t=0;t<l.length;t+=1)_[t]=lt(et(e,l,t));const d=e=>c(_[e],1,1,(()=>{_[e]=null}));return{c(){for(let e=0;e<_.length;e+=1)_[e].c();t=C()},m(e,l){for(let t=0;t<_.length;t+=1)_[t]&&_[t].m(e,l);s(e,t,l),a=!0},p(e,a){if(7&a){let s;for(l=n(e[4].children),s=0;s<l.length;s+=1){const n=et(e,l,s);_[s]?(_[s].p(n,a),r(_[s],1)):(_[s]=lt(n),_[s].c(),r(_[s],1),_[s].m(t.parentNode,t))}for(i(),s=l.length;s<_.length;s+=1)d(s);o()}},i(e){if(!a){for(let e=0;e<l.length;e+=1)r(_[e]);a=!0}},o(e){_=_.filter(Boolean);for(let e=0;e<_.length;e+=1)c(_[e]);a=!1},d(e){e&&u(t),p(_,e)}}}function _t(e){let t,a,l,n;const _=[at,tt],d=[];function p(e,t){return 0==e[4].children.length?0:1}return t=p(e),a=d[t]=_[t](e),{c(){a.c(),l=P()},m(e,a){d[t].m(e,a),s(e,l,a),n=!0},p(e,n){let s=t;t=p(e),t===s?d[t].p(e,n):(i(),c(d[s],1,1,(()=>{d[s]=null})),o(),a=d[t],a?a.p(e,n):(a=d[t]=_[t](e),a.c()),r(a,1),a.m(l.parentNode,l))},i(e){n||(r(a),n=!0)},o(e){c(a),n=!1},d(e){e&&u(l),d[t].d(e)}}}function dt(e){let t,a,l=n(e[0]),v=[];for(let t=0;t<l.length;t+=1)v[t]=_t(Je(e,l,t));const P=e=>c(v[e],1,1,(()=>{v[e]=null}));return{c(){t=_("ul");for(let e=0;e<v.length;e+=1)v[e].c();d(t,"class","bx--side-nav__items pipen-report-sidenav svelte-12gpq3")},m(e,l){s(e,t,l);for(let e=0;e<v.length;e+=1)v[e]&&v[e].m(t,null);a=!0},p(e,[a]){if(7&a){let _;for(l=n(e[0]),_=0;_<l.length;_+=1){const n=Je(e,l,_);v[_]?(v[_].p(n,a),r(v[_],1)):(v[_]=_t(n),v[_].c(),r(v[_],1),v[_].m(t,null))}for(i(),_=l.length;_<v.length;_+=1)P(_);o()}},i(e){if(!a){for(let e=0;e<l.length;e+=1)r(v[e]);a=!0}},o(e){v=v.filter(Boolean);for(let e=0;e<v.length;e+=1)c(v[e]);a=!1},d(e){e&&u(t),p(v,e)}}}function st(e,t,a){let{toc:l}=t,{page:n}=t;return v((()=>{let e,t=new URL(window.location.href);e=t.hash.length>0?document.querySelector(`[href="${t.hash}"]`):document.querySelector(".toc-current-page"),e.scrollIntoView({behavior:"smooth"})})),e.$$set=e=>{"toc"in e&&a(0,l=e.toc),"page"in e&&a(3,n=e.page)},[l,e=>{if(n===e)return"";let t=new URL(window.location.href);return t.hash="",t.searchParams.set("page",e),t.toString()},e=>e!=n?"toc-noncurrent-page":"toc-current-page",n]}class rt extends t{constructor(e){super(),a(this,e,st,dt,l,{toc:0,page:3})}}function it(e,t,a){const l=e.slice();return l[7]=t[a],l}function ot(e,t,a){const l=e.slice();return l[10]=t[a],l}function ct(e){let t,a;return t=new se({props:{isOpen:Bt,transition:e[5],$$slots:{default:[$t]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};8197&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function ut(e){let t;return{c(){t=$("Go to home:")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function pt(e){let t;return{c(){t=$(e[0])},m(e,a){s(e,t,a)},p(e,a){1&a&&b(t,e[0])},d(e){e&&u(t)}}}function vt(e){let t;return{c(){t=$("Switch to process:")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function Pt(e){let t,a;return t=new oe({props:{title:e[7].name,href:`?proc=${e[7].name}`,$$slots:{default:[At]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};4&a&&(l.title=e[7].name),4&a&&(l.href=`?proc=${e[7].name}`),8196&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function mt(e){let t,a,l,_;t=new oe({props:{class:"procgroup",$$slots:{default:[gt]},$$scope:{ctx:e}}});let d=n(e[7].procs),v=[];for(let t=0;t<d.length;t+=1)v[t]=Ot(ot(e,d,t));const O=e=>c(v[e],1,1,(()=>{v[e]=null}));return{c(){m(t.$$.fragment),a=C();for(let e=0;e<v.length;e+=1)v[e].c();l=P()},m(e,n){A(t,e,n),s(e,a,n);for(let t=0;t<v.length;t+=1)v[t]&&v[t].m(e,n);s(e,l,n),_=!0},p(e,a){const _={};if(8196&a&&(_.$$scope={dirty:a,ctx:e}),t.$set(_),4&a){let t;for(d=n(e[7].procs),t=0;t<d.length;t+=1){const n=ot(e,d,t);v[t]?(v[t].p(n,a),r(v[t],1)):(v[t]=Ot(n),v[t].c(),r(v[t],1),v[t].m(l.parentNode,l))}for(i(),t=d.length;t<v.length;t+=1)O(t);o()}},i(e){if(!_){r(t.$$.fragment,e);for(let e=0;e<d.length;e+=1)r(v[e]);_=!0}},o(e){c(t.$$.fragment,e),v=v.filter(Boolean);for(let e=0;e<v.length;e+=1)c(v[e]);_=!1},d(e){e&&(u(a),u(l)),g(t,e),p(v,e)}}}function At(e){let t,a,l=e[7].name+"";return{c(){t=$(l),a=C()},m(e,l){s(e,t,l),s(e,a,l)},p(e,a){4&a&&l!==(l=e[7].name+"")&&b(t,l)},d(e){e&&(u(t),u(a))}}}function gt(e){let t,a=e[7].name+"";return{c(){t=$(a)},m(e,a){s(e,t,a)},p(e,l){4&l&&a!==(a=e[7].name+"")&&b(t,a)},d(e){e&&u(t)}}}function Ct(e){let t,a,l,n=e[10].name+"";return{c(){t=$("- "),a=$(n),l=C()},m(e,n){s(e,t,n),s(e,a,n),s(e,l,n)},p(e,t){4&t&&n!==(n=e[10].name+"")&&b(a,n)},d(e){e&&(u(t),u(a),u(l))}}}function Ot(e){let t,a;return t=new oe({props:{title:e[10].name,class:"procgroup-proc",href:`?proc=${e[10].name}`,$$slots:{default:[Ct]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};4&a&&(l.title=e[10].name),4&a&&(l.href=`?proc=${e[10].name}`),8196&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function Rt(e){let t,a,l,n;const _=[mt,Pt],d=[];function p(e,t){return e[7].procs?0:1}return t=p(e),a=d[t]=_[t](e),{c(){a.c(),l=P()},m(e,a){d[t].m(e,a),s(e,l,a),n=!0},p(e,n){let s=t;t=p(e),t===s?d[t].p(e,n):(i(),c(d[s],1,1,(()=>{d[s]=null})),o(),a=d[t],a?a.p(e,n):(a=d[t]=_[t](e),a.c()),r(a,1),a.m(l.parentNode,l))},i(e){n||(r(a),n=!0)},o(e){c(a),n=!1},d(e){e&&u(l),d[t].d(e)}}}function ft(e){let t,a,l,_,d,v,O,R;t=new ie({props:{$$slots:{default:[ut]},$$scope:{ctx:e}}}),l=new oe({props:{href:"index.html",$$slots:{default:[pt]},$$scope:{ctx:e}}}),d=new ie({props:{$$slots:{default:[vt]},$$scope:{ctx:e}}});let f=n(e[2]),$=[];for(let t=0;t<f.length;t+=1)$[t]=Rt(it(e,f,t));const S=e=>c($[e],1,1,(()=>{$[e]=null}));return{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment),_=C(),m(d.$$.fragment),v=C();for(let e=0;e<$.length;e+=1)$[e].c();O=P()},m(e,n){A(t,e,n),s(e,a,n),A(l,e,n),s(e,_,n),A(d,e,n),s(e,v,n);for(let t=0;t<$.length;t+=1)$[t]&&$[t].m(e,n);s(e,O,n),R=!0},p(e,a){const _={};8192&a&&(_.$$scope={dirty:a,ctx:e}),t.$set(_);const s={};8193&a&&(s.$$scope={dirty:a,ctx:e}),l.$set(s);const c={};if(8192&a&&(c.$$scope={dirty:a,ctx:e}),d.$set(c),4&a){let t;for(f=n(e[2]),t=0;t<f.length;t+=1){const l=it(e,f,t);$[t]?($[t].p(l,a),r($[t],1)):($[t]=Rt(l),$[t].c(),r($[t],1),$[t].m(O.parentNode,O))}for(i(),t=f.length;t<$.length;t+=1)S(t);o()}},i(e){if(!R){r(t.$$.fragment,e),r(l.$$.fragment,e),r(d.$$.fragment,e);for(let e=0;e<f.length;e+=1)r($[e]);R=!0}},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),c(d.$$.fragment,e),$=$.filter(Boolean);for(let e=0;e<$.length;e+=1)c($[e]);R=!1},d(e){e&&(u(a),u(_),u(v),u(O)),g(t,e),g(l,e),g(d,e),p($,e)}}}function $t(e){let t,a;return t=new re({props:{$$slots:{default:[ft]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};8197&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function St(e){let t,a,l,n;t=new de({props:{render:"select",theme:"g10",persist:!0,persistKey:"__pipen-report-theme",select:{themes:["g10","white","g80","g90","g100"],labelText:"Theme",inline:!0}}});let _=e[2].length>0&&ct(e);return{c(){m(t.$$.fragment),a=C(),_&&_.c(),l=P()},m(e,d){A(t,e,d),s(e,a,d),_&&_.m(e,d),s(e,l,d),n=!0},p(e,t){e[2].length>0?_?(_.p(e,t),4&t&&r(_,1)):(_=ct(e),_.c(),r(_,1),_.m(l.parentNode,l)):_&&(i(),c(_,1,1,(()=>{_=null})),o())},i(e){n||(r(t.$$.fragment,e),r(_),n=!0)},o(e){c(t.$$.fragment,e),c(_),n=!1},d(e){e&&(u(a),u(l)),g(t,e),_&&_.d(e)}}}function bt(e){let t,a;return t=new ne({props:{class:"header-util",$$slots:{default:[St]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};8197&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function ht(e){let t;return{c(){t=$(e[4])},m(e,a){s(e,t,a)},p(e,a){16&a&&b(t,e[4])},d(e){e&&u(t)}}}function Tt(e){let t,a,l,n,d=null!==e[4]&&ht(e);return{c(){t=_("h1"),a=$(e[3]),l=C(),d&&d.c(),n=P()},m(e,_){s(e,t,_),S(t,a),s(e,l,_),d&&d.m(e,_),s(e,n,_)},p(e,t){8&t&&b(a,e[3]),null!==e[4]?d?d.p(e,t):(d=ht(e),d.c(),d.m(n.parentNode,n)):d&&(d.d(1),d=null)},d(e){e&&(u(t),u(l),u(n)),d&&d.d(e)}}}function Lt(e){let t,a;return t=new _e({props:{class:"header-tile",slot:"platform",$$slots:{default:[Tt]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};8216&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function jt(e){let t,a,l;function n(t){e[6](t)}let _={$$slots:{platform:[Lt],default:[bt]},$$scope:{ctx:e}};return void 0!==e[1]&&(_.isSideNavOpen=e[1]),t=new le({props:_}),O.push((()=>R(t,"isSideNavOpen",n))),{c(){m(t.$$.fragment)},m(e,a){A(t,e,a),l=!0},p(e,[l]){const n={};8221&l&&(n.$$scope={dirty:l,ctx:e}),!a&&2&l&&(a=!0,n.isSideNavOpen=e[1],f((()=>a=!1))),t.$set(n)},i(e){l||(r(t.$$.fragment,e),l=!0)},o(e){c(t.$$.fragment,e),l=!1},d(e){g(t,e)}}}let Bt=!1;function Mt(e,t,a){let{entries:l}=t,{logo:n}=t,{logotext:_=null}=t,{pipeline_name:d}=t,{isSideNavOpen:s}=t;return d=d||n,e.$$set=e=>{"entries"in e&&a(2,l=e.entries),"logo"in e&&a(3,n=e.logo),"logotext"in e&&a(4,_=e.logotext),"pipeline_name"in e&&a(0,d=e.pipeline_name),"isSideNavOpen"in e&&a(1,s=e.isSideNavOpen)},[d,s,l,n,_,{duration:200},function(e){s=e,a(1,s)}]}class Dt extends t{constructor(e){super(),a(this,e,Mt,jt,l,{entries:2,logo:3,logotext:4,pipeline_name:0,isSideNavOpen:1})}}function Gt(e){let t,a,l;function n(t){e[3](t)}let _={modalHeading:"Running Information",passiveModal:!0,hasScrollingContent:!0,class:"runinfo",$$slots:{default:[Ht]},$$scope:{ctx:e}};return void 0!==e[2]&&(_.open=e[2]),t=new ce({props:_}),O.push((()=>R(t,"open",n))),{c(){m(t.$$.fragment)},m(e,a){A(t,e,a),l=!0},p(e,l){const n={};34&l&&(n.$$scope={dirty:l,ctx:e}),!a&&4&l&&(a=!0,n.open=e[2],f((()=>a=!1))),t.$set(n)},i(e){l||(r(t.$$.fragment,e),l=!0)},o(e){c(t.$$.fragment,e),l=!1},d(e){g(t,e)}}}function Ft(e){let t,a,l,n,_,d;return t=new ve({props:{label:"Session Information"}}),l=new ve({props:{label:"Time Spent"}}),_=new ve({props:{label:"Device Information"}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment),n=C(),m(_.$$.fragment)},m(e,r){A(t,e,r),s(e,a,r),A(l,e,r),s(e,n,r),A(_,e,r),d=!0},p:T,i(e){d||(r(t.$$.fragment,e),r(l.$$.fragment,e),r(_.$$.fragment,e),d=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),c(_.$$.fragment,e),d=!1},d(e){e&&(u(a),u(n)),g(t,e),g(l,e),g(_,e)}}}function Nt(e){let t,a;return t=new me({props:{type:"multi",code:e[1].session}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};2&a&&(l.code=e[1].session),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function yt(e){let t,a;return t=new me({props:{type:"multi",code:e[1].time}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};2&a&&(l.code=e[1].time),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function xt(e){let t,a;return t=new me({props:{type:"multi",code:e[1].device}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};2&a&&(l.code=e[1].device),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function Ut(e){let t,a,l,n,_,d;return t=new Pe({props:{$$slots:{default:[Nt]},$$scope:{ctx:e}}}),l=new Pe({props:{$$slots:{default:[yt]},$$scope:{ctx:e}}}),_=new Pe({props:{$$slots:{default:[xt]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment),n=C(),m(_.$$.fragment)},m(e,r){A(t,e,r),s(e,a,r),A(l,e,r),s(e,n,r),A(_,e,r),d=!0},p(e,a){const n={};34&a&&(n.$$scope={dirty:a,ctx:e}),t.$set(n);const d={};34&a&&(d.$$scope={dirty:a,ctx:e}),l.$set(d);const s={};34&a&&(s.$$scope={dirty:a,ctx:e}),_.$set(s)},i(e){d||(r(t.$$.fragment,e),r(l.$$.fragment,e),r(_.$$.fragment,e),d=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),c(_.$$.fragment,e),d=!1},d(e){e&&(u(a),u(n)),g(t,e),g(l,e),g(_,e)}}}function Ht(e){let t,a,l,n;return t=new ue({props:{kind:"warning",lowContrast:!0,hideCloseButton:!0,title:"Showing the first job only. Check the workdir for information of other jobs if any."}}),l=new pe({props:{$$slots:{content:[Ut],default:[Ft]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,t){const a={};34&t&&(a.$$scope={dirty:t,ctx:e}),l.$set(a)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function wt(e){let t,a,l;return{c(){t=_("a"),t.textContent="> Running Information",d(t,"href","javascript:void(0);")},m(n,_){s(n,t,_),a||(l=h(t,"click",e[4]),a=!0)},p:T,d(e){e&&u(t),a=!1,l()}}}function It(e){let t,a,l,n,p,v,P,m,A=e[1]&&Gt(e),g=e[1]&&wt(e);return{c(){A&&A.c(),t=C(),a=_("div"),l=_("div"),g&&g.c(),n=C(),p=_("div"),v=$("Powered by "),P=_("span"),d(l,"class","runinfo"),d(P,"class","pipen-report-footer-versions"),d(p,"class","versions"),d(a,"class","pipen-report-footer bx--grid")},m(_,d){A&&A.m(_,d),s(_,t,d),s(_,a,d),S(a,l),g&&g.m(l,null),S(a,n),S(a,p),S(p,v),S(p,P),P.innerHTML=e[0],m=!0},p(e,[a]){e[1]?A?(A.p(e,a),2&a&&r(A,1)):(A=Gt(e),A.c(),r(A,1),A.m(t.parentNode,t)):A&&(i(),c(A,1,1,(()=>{A=null})),o()),e[1]?g?g.p(e,a):(g=wt(e),g.c(),g.m(l,null)):g&&(g.d(1),g=null),(!m||1&a)&&(P.innerHTML=e[0])},i(e){m||(r(A),m=!0)},o(e){c(A),m=!1},d(e){e&&(u(t),u(a)),A&&A.d(e),g&&g.d()}}}function Et(e,t,a){let{versions:l}=t,{runinfo:n}=t,_=!1;return e.$$set=e=>{"versions"in e&&a(0,l=e.versions),"runinfo"in e&&a(1,n=e.runinfo)},[l,n,_,function(e){_=e,a(2,_)},()=>{a(2,_=!0)}]}class kt extends t{constructor(e){super(),a(this,e,Et,It,l,{versions:0,runinfo:1})}}function Xt(e){let t,a,l;function n(t){e[12](t)}let _={$$slots:{default:[Vt]},$$scope:{ctx:e}};return void 0!==e[0]&&(_.isOpen=e[0]),t=new ge({props:_}),O.push((()=>R(t,"isOpen",n))),{c(){m(t.$$.fragment)},m(e,a){A(t,e,a),l=!0},p(e,l){const n={};8576&l&&(n.$$scope={dirty:l,ctx:e}),!a&&1&l&&(a=!0,n.isOpen=e[0],f((()=>a=!1))),t.$set(n)},i(e){l||(r(t.$$.fragment,e),l=!0)},o(e){c(t.$$.fragment,e),l=!1},d(e){g(t,e)}}}function Vt(e){let t,a;return t=new rt({props:{toc:e[7],page:e[8]}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};128&a&&(l.toc=e[7]),256&a&&(l.page=e[8]),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function Kt(e){let t;const a=e[10].default,l=L(a,e,e[13],null);return{c(){l&&l.c()},m(e,a){l&&l.m(e,a),t=!0},p(e,n){l&&l.p&&(!t||8192&n)&&j(l,a,e,e[13],t?M(a,e[13],n,null):B(e[13]),null)},i(e){t||(r(l,e),t=!0)},o(e){c(l,e),t=!1},d(e){l&&l.d(e)}}}function Yt(e){let t,a;return t=new Re({props:{$$slots:{default:[Kt]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};8192&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function zt(e){let t,a;return t=new Oe({props:{$$slots:{default:[Yt]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};8192&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function qt(e){let t,a,l,n;return t=new Ce({props:{$$slots:{default:[zt]},$$scope:{ctx:e}}}),l=new kt({props:{versions:e[3],runinfo:e[9]}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const n={};8192&a&&(n.$$scope={dirty:a,ctx:e}),t.$set(n);const _={};8&a&&(_.versions=e[3]),512&a&&(_.runinfo=e[9]),l.$set(_)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Qt(e){let t,a,l,n,_,d;function p(t){e[11](t)}let v={logo:e[1],logotext:e[2],entries:e[4],pipeline_name:e[5]};void 0!==e[0]&&(v.isSideNavOpen=e[0]),t=new Dt({props:v}),O.push((()=>R(t,"isSideNavOpen",p)));let P=e[6]&&e[7].length>0&&Xt(e);return _=new Ae({props:{$$slots:{default:[qt]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),l=C(),P&&P.c(),n=C(),m(_.$$.fragment)},m(e,a){A(t,e,a),s(e,l,a),P&&P.m(e,a),s(e,n,a),A(_,e,a),d=!0},p(e,[l]){const d={};2&l&&(d.logo=e[1]),4&l&&(d.logotext=e[2]),16&l&&(d.entries=e[4]),32&l&&(d.pipeline_name=e[5]),!a&&1&l&&(a=!0,d.isSideNavOpen=e[0],f((()=>a=!1))),t.$set(d),e[6]&&e[7].length>0?P?(P.p(e,l),192&l&&r(P,1)):(P=Xt(e),P.c(),r(P,1),P.m(n.parentNode,n)):P&&(i(),c(P,1,1,(()=>{P=null})),o());const s={};8712&l&&(s.$$scope={dirty:l,ctx:e}),_.$set(s)},i(e){d||(r(t.$$.fragment,e),r(P),r(_.$$.fragment,e),d=!0)},o(e){c(t.$$.fragment,e),c(P),c(_.$$.fragment,e),d=!1},d(e){e&&(u(l),u(n)),g(t,e),P&&P.d(e),g(_,e)}}}function Wt(e,t,a){let l,{$$slots:n={},$$scope:_}=t,{logo:d}=t,{logotext:s=null}=t,{versions:r}=t,{entries:i}=t,{pipeline_name:o}=t,{report_toc:c}=t,{toc:u}=t,{page:p}=t,{isSideNavOpen:P}=t,{runinfo:m}=t;return v((()=>{const e=new URL(window.location.href);let t;t=e.hash.length>0?document.querySelector(`li:has(>[href="${e.hash}"])`):document.querySelector("li:has(>.toc-current-page)"),t&&(t.classList.add("toc-active"),t.scrollIntoView());const a=document.querySelectorAll(".pipen-report-toc-anchor");l=new IntersectionObserver((e=>{e.forEach((e=>{const t=e.target.getAttribute("id"),a=document.querySelector(`li:has(>[href="#${t}"])`);a&&(e.isIntersecting?a.classList.add("toc-active"):a.classList.remove("toc-active"))}))}),{root:null,rootMargin:"0px",threshold:.5}),a.forEach((e=>{l.observe(e)}))})),e.$$set=e=>{"logo"in e&&a(1,d=e.logo),"logotext"in e&&a(2,s=e.logotext),"versions"in e&&a(3,r=e.versions),"entries"in e&&a(4,i=e.entries),"pipeline_name"in e&&a(5,o=e.pipeline_name),"report_toc"in e&&a(6,c=e.report_toc),"toc"in e&&a(7,u=e.toc),"page"in e&&a(8,p=e.page),"isSideNavOpen"in e&&a(0,P=e.isSideNavOpen),"runinfo"in e&&a(9,m=e.runinfo),"$$scope"in e&&a(13,_=e.$$scope)},[P,d,s,r,i,o,c,u,p,m,n,function(e){P=e,a(0,P)},function(e){P=e,a(0,P)},_]}class Zt extends t{constructor(e){super(),a(this,e,Wt,Qt,l,{logo:1,logotext:2,versions:3,entries:4,pipeline_name:5,report_toc:6,toc:7,page:8,isSideNavOpen:0,runinfo:9})}}function Jt(e){let t;return{c(){t=$(e[2])},m(e,a){s(e,t,a)},p:T,d(e){e&&u(t)}}}function ea(e){let t,a,l,n;return a=new fe({props:{icon:e[1],kind:"tertiary",href:e[3](),$$slots:{default:[Jt]},$$scope:{ctx:e}}}),{c(){t=_("div"),m(a.$$.fragment),d(t,"class",l=`pagenav ${e[0]}`)},m(e,l){s(e,t,l),A(a,t,null),n=!0},p(e,[_]){const s={};16&_&&(s.$$scope={dirty:_,ctx:e}),a.$set(s),(!n||1&_&&l!==(l=`pagenav ${e[0]}`))&&d(t,"class",l)},i(e){n||(r(a.$$.fragment,e),n=!0)},o(e){c(a.$$.fragment,e),n=!1},d(e){e&&u(t),g(a)}}}function ta(e,t,a){let{dir:l}=t,n="up"===l?Ge:Fe,_="up"===l?"Go to previous page":"Go to next page";return e.$$set=e=>{"dir"in e&&a(0,l=e.dir)},[l,n,_,()=>{let e=new URL(window.location.href);e.hash="";const t=parseInt(e.searchParams.get("page")||0),a="up"===l?t-1:t+1;return 0===a?e.searchParams.delete("page"):e.searchParams.set("page",a),e.toString()}]}class aa extends t{constructor(e){super(),a(this,e,ta,ea,l,{dir:0})}}function la(e){let t,a;const l=[{tooltipText:e[0]},{icon:e[1]},{direction:"bottom"},{align:"start"},e[3]];let n={};for(let e=0;e<l.length;e+=1)n=D(n,l[e]);return t=new $e({props:n}),t.$on("click",e[2]),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,[a]){const n=11&a?G(l,[1&a&&{tooltipText:e[0]},2&a&&{icon:e[1]},l[2],l[3],8&a&&F(e[3])]):{};t.$set(n)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function na(e,t,a){const l=["src","tip","icon"];let n,_=N(t,l),{src:d=""}=t,{tip:s=""}=t,{icon:r="auto"}=t,i=d.split(".").at(-1).toLocaleLowerCase();if("string"==typeof r)switch("auto"===r&&(r=i),r){case"pdf":case"DocumentPdf":n=ze;break;case"html":case"htm":case"ChartLineData":n=Ye;break;case"png":case"Png":n=Ke;break;case"jpg":case"jpeg":case"Jpg":n=Ve;break;case"gif":case"Gif":n=Xe;break;case"tif":case"tiff":case"Tif":n=ke;break;case"eps":case"DatabaseElastic":n=Ee;break;case"mp4":case"avi":case"mov":case"mkv":case"webm":case"VideoPlayer":n=Ie;break;case"Image":n=we;break;case"Data":case"DataVolume":n=He;break;case"Code":case"IbmWatsonxCodeAssistantForZRefactor":n=Ue;break;case"svg":case"Svg":n=xe;break;case"zip":case"tar":case"gz":case"bz2":case"7z":case"rar":case"xz":case"Archive":n=ye;break;default:n=Ne}const o={..._,class:[_.class,"pipen-report-image-download"].filter(Boolean).join(" ")};return e.$$set=e=>{t=D(D({},t),y(e)),a(7,_=N(t,l)),"src"in e&&a(5,d=e.src),"tip"in e&&a(0,s=e.tip),"icon"in e&&a(4,r=e.icon)},[s,n,()=>{const e=document.createElement("a");e.href=d,e.download=d.split("/").pop(),e.click()},o,r,d]}class _a extends t{constructor(e){super(),a(this,e,na,la,l,{src:5,tip:0,icon:4})}}const{Boolean:da}=w;function sa(e,t,a){const l=e.slice();return l[5]=t[a].src,l[6]=t[a].tip,l[7]=t[a].icon,l[9]=a,l}function ra(e){let t,a,l,d,v=n(e[0]),P=[];for(let t=0;t<v.length;t+=1)P[t]=ia(sa(e,v,t));const m=e=>c(P[e],1,1,(()=>{P[e]=null}));let A=[e[1],{role:"button"},{tabindex:"0"}],g={};for(let e=0;e<A.length;e+=1)g=D(g,A[e]);return{c(){t=_("div");for(let e=0;e<P.length;e+=1)P[e].c();U(t,g),H(t,"svelte-14lcnln",!0)},m(n,_){s(n,t,_);for(let e=0;e<P.length;e+=1)P[e]&&P[e].m(t,null);a=!0,l||(d=h(t,"mouseenter",e[3]),l=!0)},p(e,a){if(1&a){let l;for(v=n(e[0]),l=0;l<v.length;l+=1){const n=sa(e,v,l);P[l]?(P[l].p(n,a),r(P[l],1)):(P[l]=ia(n),P[l].c(),r(P[l],1),P[l].m(t,null))}for(i(),l=v.length;l<P.length;l+=1)m(l);o()}H(t,"svelte-14lcnln",!0)},i(e){if(!a){for(let e=0;e<v.length;e+=1)r(P[e]);a=!0}},o(e){P=P.filter(da);for(let e=0;e<P.length;e+=1)c(P[e]);a=!1},d(e){e&&u(t),p(P,e),l=!1,d()}}}function ia(e){let t,a;return t=new _a({props:{src:e[5],tip:e[6],icon:e[7]}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};1&a&&(l.src=e[5]),1&a&&(l.tip=e[6]),1&a&&(l.icon=e[7]),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function oa(e){let t,a,l=e[0].length>0&&ra(e);return{c(){l&&l.c(),t=P()},m(e,n){l&&l.m(e,n),s(e,t,n),a=!0},p(e,[a]){e[0].length>0?l?(l.p(e,a),1&a&&r(l,1)):(l=ra(e),l.c(),r(l,1),l.m(t.parentNode,t)):l&&(i(),c(l,1,1,(()=>{l=null})),o())},i(e){a||(r(l),a=!0)},o(e){c(l),a=!1},d(e){e&&u(t),l&&l.d(e)}}}function ca(e,t,a){const l=["data"];let n=N(t,l),{data:_=[]}=t;Array.isArray(_)||(_=[_]),_=_.map((e=>"string"==typeof e?{src:e,tip:`Download the ${e.split(".").at(-1)} format.`,icon:"auto"}:(e.tip=e.tip||`Download the ${e.src.split(".").at(-1)} format.`,e.icon=e.icon||"auto",e)));const d={...n,class:[n.class,"pipen-report-image-download-list"].filter(Boolean).join(" ")},s=x();return e.$$set=e=>{t=D(D({},t),y(e)),a(4,n=N(t,l)),"data"in e&&a(0,_=e.data)},[_,d,s,()=>s("mouseenter")]}class ua extends t{constructor(e){super(),a(this,e,ca,oa,l,{data:0})}}const pa=e=>({}),va=e=>({}),Pa=e=>({}),ma=e=>({}),Aa=e=>({}),ga=e=>({}),Ca=e=>({}),Oa=e=>({});function Ra(e){let t,a;return t=new Se({props:{ratio:e[6],$$slots:{default:[Ta]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};64&a&&(l.ratio=e[6]),33556159&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function fa(e){let t,a,l,n,_=e[2]&&La(e),d=e[0]&&ja(e),p=e[1]&&Ma(e);return{c(){_&&_.c(),t=C(),d&&d.c(),a=C(),p&&p.c(),l=P()},m(e,r){_&&_.m(e,r),s(e,t,r),d&&d.m(e,r),s(e,a,r),p&&p.m(e,r),s(e,l,r),n=!0},p(e,n){e[2]?_?(_.p(e,n),4&n&&r(_,1)):(_=La(e),_.c(),r(_,1),_.m(t.parentNode,t)):_&&(i(),c(_,1,1,(()=>{_=null})),o()),e[0]?d?(d.p(e,n),1&n&&r(d,1)):(d=ja(e),d.c(),r(d,1),d.m(a.parentNode,a)):d&&(i(),c(d,1,1,(()=>{d=null})),o()),e[1]?p?(p.p(e,n),2&n&&r(p,1)):(p=Ma(e),p.c(),r(p,1),p.m(l.parentNode,l)):p&&(i(),c(p,1,1,(()=>{p=null})),o())},i(e){n||(r(_),r(d),r(p),n=!0)},o(e){c(_),c(d),c(p),n=!1},d(e){e&&(u(t),u(a),u(l)),_&&_.d(e),d&&d.d(e),p&&p.d(e)}}}function $a(e){let t;const a=e[15].loading,l=L(a,e,e[25],ma);return{c(){l&&l.c()},m(e,a){l&&l.m(e,a),t=!0},p(e,n){l&&l.p&&(!t||33554432&n)&&j(l,a,e,e[25],t?M(a,e[25],n,Pa):B(e[25]),ma)},i(e){t||(r(l,e),t=!0)},o(e){c(l,e),t=!1},d(e){l&&l.d(e)}}}function Sa(e){let t,a,l,n,d,p,v,m,A=[e[12],{src:a=e[3]},{alt:e[4]},{title:"Click to zoom in ..."}],g={};for(let e=0;e<A.length;e+=1)g=D(g,A[e]);let O=e[9]&&ba(e);return{c(){t=_("img"),n=C(),O&&O.c(),d=P(),U(t,g)},m(a,l){s(a,t,l),s(a,n,l),O&&O.m(a,l),s(a,d,l),p=!0,v||(m=[h(t,"click",e[11]),h(t,"mouseenter",e[19]),h(t,"mouseleave",e[20])],v=!0)},p(l,n){U(t,g=G(A,[(e=l)[12],(!p||8&n&&!I(t.src,a=e[3]))&&{src:a},(!p||16&n)&&{alt:e[4]},{title:"Click to zoom in ..."}])),e[9]?O?(O.p(e,n),512&n&&r(O,1)):(O=ba(e),O.c(),r(O,1),O.m(d.parentNode,d)):O&&(i(),c(O,1,1,(()=>{O=null})),o())},i(a){p||(a&&E((()=>{p&&(l||(l=k(t,X,{duration:e[7]?Fa:0},!0)),l.run(1))})),r(O),p=!0)},o(a){a&&(l||(l=k(t,X,{duration:e[7]?Fa:0},!1)),l.run(0)),c(O),p=!1},d(e){e&&(u(t),u(n),u(d)),e&&l&&l.end(),O&&O.d(e),v=!1,V(m)}}}function ba(e){let t,a;return t=new ua({props:{data:e[5]}}),t.$on("mouseenter",e[21]),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};32&a&&(l.data=e[5]),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function ha(e){let t;const a=e[15].error,l=L(a,e,e[25],va);return{c(){l&&l.c()},m(e,a){l&&l.m(e,a),t=!0},p(e,n){l&&l.p&&(!t||33554432&n)&&j(l,a,e,e[25],t?M(a,e[25],n,pa):B(e[25]),va)},i(e){t||(r(l,e),t=!0)},o(e){c(l,e),t=!1},d(e){l&&l.d(e)}}}function Ta(e){let t,a,l,n,_=e[2]&&$a(e),d=e[0]&&Sa(e),p=e[1]&&ha(e);return{c(){_&&_.c(),t=C(),d&&d.c(),a=C(),p&&p.c(),l=P()},m(e,r){_&&_.m(e,r),s(e,t,r),d&&d.m(e,r),s(e,a,r),p&&p.m(e,r),s(e,l,r),n=!0},p(e,n){e[2]?_?(_.p(e,n),4&n&&r(_,1)):(_=$a(e),_.c(),r(_,1),_.m(t.parentNode,t)):_&&(i(),c(_,1,1,(()=>{_=null})),o()),e[0]?d?(d.p(e,n),1&n&&r(d,1)):(d=Sa(e),d.c(),r(d,1),d.m(a.parentNode,a)):d&&(i(),c(d,1,1,(()=>{d=null})),o()),e[1]?p?(p.p(e,n),2&n&&r(p,1)):(p=ha(e),p.c(),r(p,1),p.m(l.parentNode,l)):p&&(i(),c(p,1,1,(()=>{p=null})),o())},i(e){n||(r(_),r(d),r(p),n=!0)},o(e){c(_),c(d),c(p),n=!1},d(e){e&&(u(t),u(a),u(l)),_&&_.d(e),d&&d.d(e),p&&p.d(e)}}}function La(e){let t;const a=e[15].loading,l=L(a,e,e[25],Oa);return{c(){l&&l.c()},m(e,a){l&&l.m(e,a),t=!0},p(e,n){l&&l.p&&(!t||33554432&n)&&j(l,a,e,e[25],t?M(a,e[25],n,Ca):B(e[25]),Oa)},i(e){t||(r(l,e),t=!0)},o(e){c(l,e),t=!1},d(e){l&&l.d(e)}}}function ja(e){let t,a,l,n,d,p,v,m,A=[e[12],{src:a=e[3]},{alt:e[4]},{title:"Click to zoom in ..."}],g={};for(let e=0;e<A.length;e+=1)g=D(g,A[e]);let O=e[9]&&Ba(e);return{c(){t=_("img"),n=C(),O&&O.c(),d=P(),U(t,g)},m(a,l){s(a,t,l),s(a,n,l),O&&O.m(a,l),s(a,d,l),p=!0,v||(m=[h(t,"click",e[11]),h(t,"mouseenter",e[16]),h(t,"mouseleave",e[17])],v=!0)},p(l,n){U(t,g=G(A,[(e=l)[12],(!p||8&n&&!I(t.src,a=e[3]))&&{src:a},(!p||16&n)&&{alt:e[4]},{title:"Click to zoom in ..."}])),e[9]?O?(O.p(e,n),512&n&&r(O,1)):(O=Ba(e),O.c(),r(O,1),O.m(d.parentNode,d)):O&&(i(),c(O,1,1,(()=>{O=null})),o())},i(a){p||(a&&E((()=>{p&&(l||(l=k(t,X,{duration:e[7]?Fa:0},!0)),l.run(1))})),r(O),p=!0)},o(a){a&&(l||(l=k(t,X,{duration:e[7]?Fa:0},!1)),l.run(0)),c(O),p=!1},d(e){e&&(u(t),u(n),u(d)),e&&l&&l.end(),O&&O.d(e),v=!1,V(m)}}}function Ba(e){let t,a;return t=new ua({props:{data:e[5]}}),t.$on("mouseenter",e[18]),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};32&a&&(l.data=e[5]),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function Ma(e){let t;const a=e[15].error,l=L(a,e,e[25],ga);return{c(){l&&l.c()},m(e,a){l&&l.m(e,a),t=!0},p(e,n){l&&l.p&&(!t||33554432&n)&&j(l,a,e,e[25],t?M(a,e[25],n,Aa):B(e[25]),ga)},i(e){t||(r(l,e),t=!0)},o(e){c(l,e),t=!1},d(e){l&&l.d(e)}}}function Da(e){let t,a,l,n;return{c(){t=_("img"),d(t,"class","pipen-report-image-lg"),I(t.src,a=e[3])||d(t,"src",a),d(t,"alt",e[4])},m(e,a){s(e,t,a),n=!0},p(l,_){e=l,(!n||8&_&&!I(t.src,a=e[3]))&&d(t,"src",a),(!n||16&_)&&d(t,"alt",e[4])},i(a){n||(a&&E((()=>{n&&(l||(l=k(t,X,{duration:e[7]?Fa:0},!0)),l.run(1))})),n=!0)},o(a){a&&(l||(l=k(t,X,{duration:e[7]?Fa:0},!1)),l.run(0)),n=!1},d(e){e&&u(t),e&&l&&l.end()}}}function Ga(e){let t,a,l,n,_,d;const p=[fa,Ra],v=[];function P(e,t){return void 0===e[6]?0:1}function $(t){e[22](t)}t=P(e),a=v[t]=p[t](e);let S={modalHeading:e[4],primaryButtonText:"Open in new window ...",secondaryButtonText:"Close",$$slots:{default:[Da]},$$scope:{ctx:e}};return void 0!==e[8]&&(S.open=e[8]),n=new ce({props:S}),O.push((()=>R(n,"open",$))),n.$on("click:button--secondary",e[23]),n.$on("click:button--primary",e[24]),{c(){a.c(),l=C(),m(n.$$.fragment)},m(e,a){v[t].m(e,a),s(e,l,a),A(n,e,a),d=!0},p(e,[d]){let s=t;t=P(e),t===s?v[t].p(e,d):(i(),c(v[s],1,1,(()=>{v[s]=null})),o(),a=v[t],a?a.p(e,d):(a=v[t]=p[t](e),a.c()),r(a,1),a.m(l.parentNode,l));const u={};16&d&&(u.modalHeading=e[4]),33554584&d&&(u.$$scope={dirty:d,ctx:e}),!_&&256&d&&(_=!0,u.open=e[8],f((()=>_=!1))),n.$set(u)},i(e){d||(r(a),r(n.$$.fragment,e),d=!0)},o(e){c(a),c(n.$$.fragment,e),d=!1},d(e){e&&u(l),v[t].d(e),g(n,e)}}}const Fa=110;function Na(e,t,a){const l=["src","alt","download","ratio","loading","loaded","error","fadeIn","border","loadImage"];let n=N(t,l),{$$slots:_={},$$scope:d}=t,{src:s=""}=t,{alt:r=""}=t,{download:i=[]}=t,{ratio:o}=t,{loading:c=!1}=t,{loaded:u=!1}=t,{error:p=!1}=t,{fadeIn:P=!1}=t,{border:m=!0}=t,A=!1,g=!1;const C=e=>{null!=f&&(f=null),a(0,u=!1),a(1,p=!1),f=new Image,f.src=e||s,f.onload=()=>a(0,u=!0),f.onerror=()=>a(1,p=!0)},O=x();let R,f=null;v((()=>()=>f=null));const $={...n,class:[n.class,"pipen-report-image",m?"":"pipen-report-image-noborder"].filter(Boolean).join(" ")};return e.$$set=e=>{t=D(D({},t),y(e)),a(28,n=N(t,l)),"src"in e&&a(3,s=e.src),"alt"in e&&a(4,r=e.alt),"download"in e&&a(5,i=e.download),"ratio"in e&&a(6,o=e.ratio),"loading"in e&&a(2,c=e.loading),"loaded"in e&&a(0,u=e.loaded),"error"in e&&a(1,p=e.error),"fadeIn"in e&&a(7,P=e.fadeIn),"border"in e&&a(13,m=e.border),"$$scope"in e&&a(25,d=e.$$scope)},e.$$.update=()=>{3&e.$$.dirty&&a(2,c=!u&&!p),8&e.$$.dirty&&s&&"undefined"!=typeof window&&C(),1&e.$$.dirty&&u&&O("load"),2&e.$$.dirty&&p&&O("error")},[u,p,c,s,r,i,o,P,A,g,R,()=>{a(8,A=!0)},$,m,C,_,()=>{a(9,g=!0)},()=>{a(10,R=setTimeout((()=>{a(9,g=!1)}),100))},()=>{clearTimeout(R),a(9,g=!0)},()=>{a(9,g=!0)},()=>{a(10,R=setTimeout((()=>{a(9,g=!1)}),100))},()=>{clearTimeout(R),a(9,g=!0)},function(e){A=e,a(8,A)},()=>{a(8,A=!1)},()=>{a(8,A=!1),window.open(s)},d]}class ya extends t{constructor(e){super(),a(this,e,Na,Ga,l,{src:3,alt:4,download:5,ratio:6,loading:2,loaded:0,error:1,fadeIn:7,border:13,loadImage:14})}get loadImage(){return this.$$.ctx[14]}}function xa(e){let t,a,l=[e[2]],n={};for(let e=0;e<l.length;e+=1)n=D(n,l[e]);return{c(){t=_("div"),a=$("Image loading ..."),U(t,n)},m(e,l){s(e,t,l),S(t,a)},p:T,d(e){e&&u(t)}}}function Ua(e){let t,a,l=[e[3]],n={};for(let e=0;e<l.length;e+=1)n=D(n,l[e]);return{c(){t=_("div"),a=$("Image loading error!"),U(t,n)},m(e,l){s(e,t,l),S(t,a)},p:T,d(e){e&&u(t)}}}function Ha(e){let t,a,l;const n=[{src:e[1]},e[4]];let d={$$slots:{error:[Ua],loading:[xa]},$$scope:{ctx:e}};for(let e=0;e<n.length;e+=1)d=D(d,n[e]);a=new ya({props:d});let i=[e[0]],o={};for(let e=0;e<i.length;e+=1)o=D(o,i[e]);return{c(){t=_("div"),m(a.$$.fragment),U(t,o)},m(e,n){s(e,t,n),A(a,t,null),l=!0},p(e,[l]){const _=18&l?G(n,[2&l&&{src:e[1]},16&l&&F(e[4])]):{};256&l&&(_.$$scope={dirty:l,ctx:e}),a.$set(_),U(t,o=G(i,[1&l&&e[0]]))},i(e){l||(r(a.$$.fragment,e),l=!0)},o(e){c(a.$$.fragment,e),l=!1},d(e){e&&u(t),g(a)}}}function wa(e,t,a){const l=["frameProps","width","height","src"];let n=N(t,l),{frameProps:_={class:"pipen-report-image-frame"}}=t,{width:d=300}=t,{height:s=200}=t,{src:r}=t;_||(_={"style:position":"relative"});let i=`width: ${d}px; height: ${s}px; aspect-ratio: ${d} / ${s}`,o={...n,class:[n.class,"pipen-report-image-loading"].filter(Boolean).join(" "),style:n.style?`${n.style}; ${i}`:i},c={...o,class:[o.class,"error"].join(" ")};return e.$$set=e=>{t=D(D({},t),y(e)),a(4,n=N(t,l)),"frameProps"in e&&a(0,_=e.frameProps),"width"in e&&a(5,d=e.width),"height"in e&&a(6,s=e.height),"src"in e&&a(1,r=e.src)},[_,r,o,c,n,d,s]}let Ia=class extends t{constructor(e){super(),a(this,e,wa,Ha,l,{frameProps:0,width:5,height:6,src:1})}};const Ea=e=>({}),ka=e=>({});function Xa(e){let t,a;const l=e[25].default,n=L(l,e,e[24],null);return{c(){t=_("ul"),n&&n.c(),d(t,"role","menu"),d(t,"tabindex","-1"),d(t,"aria-label",e[14]),d(t,"data-floating-menu-direction",e[5]),d(t,"class",e[9]),H(t,"bx--overflow-menu-options",!0),H(t,"bx--overflow-menu--flip",e[7]),H(t,"bx--overflow-menu-options--open",e[0]),H(t,"bx--overflow-menu-options--light",e[6]),H(t,"bx--overflow-menu-options--sm","sm"===e[4]),H(t,"bx--overflow-menu-options--xl","xl"===e[4]),H(t,"bx--breadcrumb-menu-options",!!e[15])},m(l,_){s(l,t,_),n&&n.m(t,null),e[32](t),a=!0},p(e,_){n&&n.p&&(!a||16777216&_[0])&&j(n,l,e,e[24],a?M(l,e[24],_,null):B(e[24]),null),(!a||16384&_[0])&&d(t,"aria-label",e[14]),(!a||32&_[0])&&d(t,"data-floating-menu-direction",e[5]),(!a||512&_[0])&&d(t,"class",e[9]),(!a||512&_[0])&&H(t,"bx--overflow-menu-options",!0),(!a||640&_[0])&&H(t,"bx--overflow-menu--flip",e[7]),(!a||513&_[0])&&H(t,"bx--overflow-menu-options--open",e[0]),(!a||576&_[0])&&H(t,"bx--overflow-menu-options--light",e[6]),(!a||528&_[0])&&H(t,"bx--overflow-menu-options--sm","sm"===e[4]),(!a||528&_[0])&&H(t,"bx--overflow-menu-options--xl","xl"===e[4]),(!a||33280&_[0])&&H(t,"bx--breadcrumb-menu-options",!!e[15])},i(e){a||(r(n,e),a=!0)},o(e){c(n,e),a=!1},d(a){a&&u(t),n&&n.d(a),e[32](null)}}}function Va(e){let t,a,l,n,d,p,v,O;const R=e[25].menu,f=L(R,e,e[24],ka),$=f||function(e){let t,a,l;var n=e[1];function _(e,t){return{props:{"aria-label":e[11],title:e[11],class:"bx--overflow-menu__icon "+e[10]}}}return n&&(t=Q(n,_(e))),{c(){t&&m(t.$$.fragment),a=P()},m(e,n){t&&A(t,e,n),s(e,a,n),l=!0},p(e,l){if(2&l[0]&&n!==(n=e[1])){if(t){i();const e=t;c(e.$$.fragment,1,0,(()=>{g(e,1)})),o()}n?(t=Q(n,_(e)),m(t.$$.fragment),r(t.$$.fragment,1),A(t,a.parentNode,a)):t=null}else if(n){const a={};2048&l[0]&&(a["aria-label"]=e[11]),2048&l[0]&&(a.title=e[11]),1024&l[0]&&(a.class="bx--overflow-menu__icon "+e[10]),t.$set(a)}},i(e){l||(t&&r(t.$$.fragment,e),l=!0)},o(e){t&&c(t.$$.fragment,e),l=!1},d(e){e&&u(a),t&&g(t,e)}}}(e);let b=e[0]&&Xa(e),T=[{type:"button"},{"aria-haspopup":""},{"aria-expanded":e[0]},{"aria-label":e[14]},{id:e[12]},e[20]],F={};for(let e=0;e<T.length;e+=1)F=D(F,T[e]);return{c(){t=new W(!1),a=P(),l=C(),n=_("button"),$.c(),d=C(),b&&b.c(),t.a=a,U(n,F),H(n,"bx--overflow-menu",!0),H(n,"bx--overflow-menu--open",e[0]),H(n,"bx--overflow-menu--light",e[6]),H(n,"bx--overflow-menu--sm","sm"===e[4]),H(n,"bx--overflow-menu--xl","xl"===e[4])},m(_,r){t.m(e[13],document.head),S(document.head,a),s(_,l,r),s(_,n,r),$.m(n,null),S(n,d),b&&b.m(n,null),n.autofocus&&n.focus(),e[33](n),p=!0,v||(O=[h(window,"click",e[31]),h(n,"click",e[26]),h(n,"click",e[34]),h(n,"mouseover",e[27]),h(n,"mouseenter",e[28]),h(n,"mouseleave",e[29]),h(n,"keydown",e[30]),h(n,"keydown",e[35]),h(n,"focusout",e[36])],v=!0)},p(e,a){(!p||8192&a[0])&&t.p(e[13]),f?f.p&&(!p||16777216&a[0])&&j(f,R,e,e[24],p?M(R,e[24],a,Ea):B(e[24]),ka):$.p&&(!p||3074&a[0])&&$.p(e,p?a:[-1,-1]),e[0]?b?(b.p(e,a),1&a[0]&&r(b,1)):(b=Xa(e),b.c(),r(b,1),b.m(n,null)):b&&(i(),c(b,1,1,(()=>{b=null})),o()),U(n,F=G(T,[{type:"button"},{"aria-haspopup":""},(!p||1&a[0])&&{"aria-expanded":e[0]},(!p||16384&a[0])&&{"aria-label":e[14]},(!p||4096&a[0])&&{id:e[12]},1048576&a[0]&&e[20]])),H(n,"bx--overflow-menu",!0),H(n,"bx--overflow-menu--open",e[0]),H(n,"bx--overflow-menu--light",e[6]),H(n,"bx--overflow-menu--sm","sm"===e[4]),H(n,"bx--overflow-menu--xl","xl"===e[4])},i(e){p||(r($,e),r(b),p=!0)},o(e){c($,e),c(b),p=!1},d(_){_&&(t.d(),u(l),u(n)),u(a),$.d(_),b&&b.d(),e[33](null),v=!1,V(O)}}}function Ka(e,t,a){let l,n;const _=["size","direction","open","light","flipped","persistentClickItems","menuOptionsClass","icon","iconClass","iconDescription","id","buttonRef","menuRef"];let d,s,r,i=N(t,_),{$$slots:o={},$$scope:c}=t,{size:u}=t,{direction:p="bottom"}=t,{open:v=!1}=t,{light:P=!1}=t,{flipped:m=!1}=t,{persistentClickItems:A=!1}=t,{menuOptionsClass:g}=t,{icon:C=Qe}=t,{iconClass:R}=t,{iconDescription:f="Open and close list of options"}=t,{id:$="ccs-"+Math.random().toString(36)}=t,{buttonRef:S=null}=t,{menuRef:b=null}=t;const h=K("BreadcrumbItem"),T=x(),L=Z([]);Y(e,L,(e=>a(23,s=e)));const j=Z(void 0);Y(e,j,(e=>a(38,r=e)));const B=Z(void 0),M=Z(-1);let G;Y(e,M,(e=>a(22,d=e)));let F=!0;z("OverflowMenu",{focusedId:B,add:({id:e,text:t,primaryFocus:a,disabled:l})=>{L.update((n=>(a&&M.set(n.length),[...n,{id:e,text:t,primaryFocus:a,disabled:l,index:n.length}])))},update:e=>{j.set(e)},change:e=>{let t=d+e;t<0?t=s.length-1:t>=s.length&&(t=0);let a=s[t].disabled;for(;a;)t+=e,t<0?t=s.length-1:t>=s.length&&(t=0),a=s[t].disabled;M.set(t)}}),q((()=>{if(r){const{index:e,text:t}=s.filter((e=>e.id===r))[0];T("close",{index:e,text:t}),a(0,v=!1)}if(v){const{width:e,height:t}=S.getBoundingClientRect();a(21,G=e),!F&&d<0&&b.focus(),m&&(a(3,b.style.left="auto",b),a(3,b.style.right=0,b)),"top"===p?(a(3,b.style.top="auto",b),a(3,b.style.bottom=t+"px",b)):"bottom"===p&&a(3,b.style.top=t+"px",b),h&&(a(3,b.style.top=t+10+"px",b),a(3,b.style.left="-11px",b))}v||(L.set([]),j.set(void 0),M.set(0)),F=!1}));return e.$$set=e=>{a(40,t=D(D({},t),y(e))),a(20,i=N(t,_)),"size"in e&&a(4,u=e.size),"direction"in e&&a(5,p=e.direction),"open"in e&&a(0,v=e.open),"light"in e&&a(6,P=e.light),"flipped"in e&&a(7,m=e.flipped),"persistentClickItems"in e&&a(8,A=e.persistentClickItems),"menuOptionsClass"in e&&a(9,g=e.menuOptionsClass),"icon"in e&&a(1,C=e.icon),"iconClass"in e&&a(10,R=e.iconClass),"iconDescription"in e&&a(11,f=e.iconDescription),"id"in e&&a(12,$=e.id),"buttonRef"in e&&a(2,S=e.buttonRef),"menuRef"in e&&a(3,b=e.menuRef),"$$scope"in e&&a(24,c=e.$$scope)},e.$$.update=()=>{a(14,l=t["aria-label"]||"menu"),12582912&e.$$.dirty[0]&&s[d]&&B.set(s[d].id),2101248&e.$$.dirty[0]&&a(13,n=`<style>\n    #${$} .bx--overflow-menu-options.bx--overflow-menu-options:after {\n      width: ${G?G+"px":"2rem"};\n    }\n  </style>`)},h&&a(1,C=qe),t=y(t),[v,C,S,b,u,p,P,m,A,g,R,f,$,n,l,h,T,L,j,M,i,G,d,s,c,o,function(t){J.call(this,e,t)},function(t){J.call(this,e,t)},function(t){J.call(this,e,t)},function(t){J.call(this,e,t)},function(t){J.call(this,e,t)},({target:e})=>{S&&S.contains(e)||b&&!b.contains(e)&&a(0,v=!1)},function(e){O[e?"unshift":"push"]((()=>{b=e,a(3,b)}))},function(e){O[e?"unshift":"push"]((()=>{S=e,a(2,S)}))},e=>{A&&e.stopPropagation(),b&&b.contains(e.target)||(a(0,v=!v),v||T("close"))},e=>{v&&(["ArrowDown","ArrowLeft","ArrowRight","ArrowUp"].includes(e.key)?e.preventDefault():"Escape"===e.key&&(e.stopPropagation(),T("close"),a(0,v=!1),S.focus()))},e=>{v&&(S.contains(e.relatedTarget)||(T("close"),a(0,v=!1)))}]}class Ya extends t{constructor(e){super(),a(this,e,Ka,Va,l,{size:4,direction:5,open:0,light:6,flipped:7,persistentClickItems:8,menuOptionsClass:9,icon:1,iconClass:10,iconDescription:11,id:12,buttonRef:2,menuRef:3},null,[-1,-1])}}function za(e,t,a){const l=e.slice();return l[3]=t[a],l[4]=t,l[5]=a,l}function qa(e){let t,a,l,n,i;function o(t){e[1](t,e[3])}let p={labelText:e[3].value};return void 0!==e[3].selected&&(p.checked=e[3].selected),a=new be({props:p}),O.push((()=>R(a,"checked",o))),a.$on("change",e[2]),{c(){t=_("li"),m(a.$$.fragment),n=C(),d(t,"class","pipen-report-column-selector-item svelte-147jw2r")},m(e,l){s(e,t,l),A(a,t,null),S(t,n),i=!0},p(t,n){e=t;const _={};1&n&&(_.labelText=e[3].value),!l&&1&n&&(l=!0,_.checked=e[3].selected,f((()=>l=!1))),a.$set(_)},i(e){i||(r(a.$$.fragment,e),i=!0)},o(e){c(a.$$.fragment,e),i=!1},d(e){e&&u(t),g(a)}}}function Qa(e){let t,a,l=n(e[0]),_=[];for(let t=0;t<l.length;t+=1)_[t]=qa(za(e,l,t));const d=e=>c(_[e],1,1,(()=>{_[e]=null}));return{c(){for(let e=0;e<_.length;e+=1)_[e].c();t=P()},m(e,l){for(let t=0;t<_.length;t+=1)_[t]&&_[t].m(e,l);s(e,t,l),a=!0},p(e,a){if(1&a){let s;for(l=n(e[0]),s=0;s<l.length;s+=1){const n=za(e,l,s);_[s]?(_[s].p(n,a),r(_[s],1)):(_[s]=qa(n),_[s].c(),r(_[s],1),_[s].m(t.parentNode,t))}for(i(),s=l.length;s<_.length;s+=1)d(s);o()}},i(e){if(!a){for(let e=0;e<l.length;e+=1)r(_[e]);a=!0}},o(e){_=_.filter(Boolean);for(let e=0;e<_.length;e+=1)c(_[e]);a=!1},d(e){e&&u(t),p(_,e)}}}function Wa(e){let t,a;return t=new Ya({props:{class:"pipen-report-overflowmenu",persistentClickItems:!0,size:"sm",icon:We,iconDescription:"Select visible columns ...",$$slots:{default:[Qa]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,[a]){const l={};65&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function Za(e,t,a){let{allColumns:l}=t;return e.$$set=e=>{"allColumns"in e&&a(0,l=e.allColumns)},[l,function(t,n){e.$$.not_equal(n.selected,t)&&(n.selected=t,a(0,l))},()=>{a(0,l)}]}class Ja extends t{constructor(e){super(),a(this,e,Za,Wa,l,{allColumns:0})}}function el(e){let t,a;return t=new fe({props:{iconDescription:"Right click and save as to download the entire data",tooltipPosition:"top",icon:Ze,href:e[2]}}),t.$on("click",e[13]),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};4&a&&(l.href=e[2]),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function tl(e){let t,a,l,n,_,d,p,v,$,S;function b(t){e[17](t)}let h={persistent:!0};function T(t){e[18](t)}void 0!==e[7]&&(h.value=e[7]),t=new Be({props:h}),O.push((()=>R(t,"value",b)));let L={};void 0!==e[6]&&(L.allColumns=e[6]),n=new Ja({props:L}),O.push((()=>R(n,"allColumns",T))),p=new fe({props:{iconDescription:`Download current ${e[11].length} records (may be partial of the entire data)`,tooltipPosition:"top",icon:Ne}}),p.$on("click",e[14]);let j=!!e[2]&&el(e);return{c(){m(t.$$.fragment),l=C(),m(n.$$.fragment),d=C(),m(p.$$.fragment),v=C(),j&&j.c(),$=P()},m(e,a){A(t,e,a),s(e,l,a),A(n,e,a),s(e,d,a),A(p,e,a),s(e,v,a),j&&j.m(e,a),s(e,$,a),S=!0},p(e,l){const d={};!a&&128&l&&(a=!0,d.value=e[7],f((()=>a=!1))),t.$set(d);const s={};!_&&64&l&&(_=!0,s.allColumns=e[6],f((()=>_=!1))),n.$set(s);const u={};2048&l&&(u.iconDescription=`Download current ${e[11].length} records (may be partial of the entire data)`),p.$set(u),e[2]?j?(j.p(e,l),4&l&&r(j,1)):(j=el(e),j.c(),r(j,1),j.m($.parentNode,$)):j&&(i(),c(j,1,1,(()=>{j=null})),o())},i(e){S||(r(t.$$.fragment,e),r(n.$$.fragment,e),r(p.$$.fragment,e),r(j),S=!0)},o(e){c(t.$$.fragment,e),c(n.$$.fragment,e),c(p.$$.fragment,e),c(j),S=!1},d(e){e&&(u(l),u(d),u(v),u($)),g(t,e),g(n,e),g(p,e),j&&j.d(e)}}}function al(e){let t,a;return t=new je({props:{$$slots:{default:[tl]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};134219972&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function ll(e){let t,a;return t=new Le({props:{size:e[12],$$slots:{default:[al]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};134219972&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function nl(e){let t;return{c(){t=_("p"),t.textContent='You must use "Save as" or "Save link as" from the context menu (by right-clicking the button) to download the entire data.'},m(e,a){s(e,t,a)},p:T,d(e){e&&u(t)}}}function _l(e){let t,a,l,n,_,d,i,o,p;const v=[{size:e[0]},{zebra:e[3]},{sortable:e[4]},{page:e[8]},{pageSize:e[9]},{rows:e[11]},{headers:e[1]},e[15]];let P={$$slots:{default:[ll]},$$scope:{ctx:e}};for(let e=0;e<v.length;e+=1)P=D(P,v[e]);function $(t){e[19](t)}function S(t){e[20](t)}t=new he({props:P});let b={pageSizes:e[5],totalItems:e[11].length};function h(t){e[21](t)}void 0!==e[9]&&(b.pageSize=e[9]),void 0!==e[8]&&(b.page=e[8]),l=new Te({props:b}),O.push((()=>R(l,"pageSize",$))),O.push((()=>R(l,"page",S)));let T={passiveModal:!0,modalHeading:"Downloading entire data",$$slots:{default:[nl]},$$scope:{ctx:e}};return void 0!==e[10]&&(T.open=e[10]),i=new ce({props:T}),O.push((()=>R(i,"open",h))),i.$on("open",e[22]),i.$on("close",e[23]),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment),d=C(),m(i.$$.fragment)},m(e,n){A(t,e,n),s(e,a,n),A(l,e,n),s(e,d,n),A(i,e,n),p=!0},p(e,[a]){const d=35611&a?G(v,[1&a&&{size:e[0]},8&a&&{zebra:e[3]},16&a&&{sortable:e[4]},256&a&&{page:e[8]},512&a&&{pageSize:e[9]},2048&a&&{rows:e[11]},2&a&&{headers:e[1]},32768&a&&F(e[15])]):{};134219972&a&&(d.$$scope={dirty:a,ctx:e}),t.$set(d);const s={};32&a&&(s.pageSizes=e[5]),2048&a&&(s.totalItems=e[11].length),!n&&512&a&&(n=!0,s.pageSize=e[9],f((()=>n=!1))),!_&&256&a&&(_=!0,s.page=e[8],f((()=>_=!1))),l.$set(s);const r={};134217728&a&&(r.$$scope={dirty:a,ctx:e}),!o&&1024&a&&(o=!0,r.open=e[10],f((()=>o=!1))),i.$set(r)},i(e){p||(r(t.$$.fragment,e),r(l.$$.fragment,e),r(i.$$.fragment,e),p=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),c(i.$$.fragment,e),p=!1},d(e){e&&(u(a),u(d)),g(t,e),g(l,e),g(i,e)}}}function dl(e,t,a){const l=["size","src","zebra","sortable","rows","headers","pageSizes"];let n=N(t,l),{size:_="sm"}=t,{src:d}=t;_={sm:"short",small:"short",short:"short",md:"medium",medium:"medium",lg:"tall",large:"tall",tall:"tall",compact:"compact"}[_];let s={short:"sm",compact:"sm",medium:"default",tall:"default"}[_],{zebra:r=!0}=t,{sortable:i=!0}=t,{rows:o=[]}=t,{headers:c=[]}=t,{pageSizes:u=[20,50,75,100]}=t,p=c.map((e=>({...e,selected:!0}))),v=1,P=u[0],m="",A=!1,g=o;return e.$$set=e=>{t=D(D({},t),y(e)),a(15,n=N(t,l)),"size"in e&&a(0,_=e.size),"src"in e&&a(2,d=e.src),"zebra"in e&&a(3,r=e.zebra),"sortable"in e&&a(4,i=e.sortable),"rows"in e&&a(16,o=e.rows),"headers"in e&&a(1,c=e.headers),"pageSizes"in e&&a(5,u=e.pageSizes)},e.$$.update=()=>{64&e.$$.dirty&&a(1,c=p.filter((e=>e.selected))),65664&e.$$.dirty&&(m.length>0?a(11,g=o.filter((e=>Object.values(e).filter((e=>!!e&&e.toString().includes(m))).length>0))):a(11,g=o))},[_,c,d,r,i,u,p,m,v,P,A,g,s,e=>{e.preventDefault(),a(10,A=!0)},()=>{((e,t)=>{const a=document.createElement("a");a.href=e,a.download=t,document.body.appendChild(a),a.click(),document.body.removeChild(a)})("data:application/vnd.ms-excel;charset=utf-8,\ufeff"+encodeURIComponent(function(){const e=c.map((e=>e.value)),t=[];return g.forEach((e=>{const a=c.map((t=>`"${e[t.key]}"`));t.push(a.join(","))})),e.join(",")+"\r\n"+t.join("\r\n")}()),d&&d.length>0?d.split("/").pop().split(".")[0]+".xls":"data-subset.xls")},n,o,function(e){m=e,a(7,m)},function(e){p=e,a(6,p)},function(e){P=e,a(9,P)},function(e){v=e,a(8,v)},function(e){A=e,a(10,A)},function(t){J.call(this,e,t)},function(t){J.call(this,e,t)}]}class sl extends t{constructor(e){super(),a(this,e,dl,_l,l,{size:0,src:2,zebra:3,sortable:4,rows:16,headers:1,pageSizes:5})}}function rl(e){let t,a,l;const n=[{headers:e[0]},{rows:e[1]},e[3]];let d={};for(let e=0;e<n.length;e+=1)d=D(d,n[e]);a=new sl({props:d});let i=[e[2]],o={};for(let e=0;e<i.length;e+=1)o=D(o,i[e]);return{c(){t=_("div"),m(a.$$.fragment),U(t,o)},m(e,n){s(e,t,n),A(a,t,null),l=!0},p(e,l){const _=11&l?G(n,[1&l&&{headers:e[0]},2&l&&{rows:e[1]},8&l&&F(e[3])]):{};a.$set(_),U(t,o=G(i,[4&l&&e[2]]))},i(e){l||(r(a.$$.fragment,e),l=!0)},o(e){c(a.$$.fragment,e),l=!1},d(e){e&&u(t),g(a)}}}function il(e){let t,a;const l=[{headers:e[0]},{rows:e[1]},e[3]];let n={};for(let e=0;e<l.length;e+=1)n=D(n,l[e]);return t=new sl({props:n}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const n=11&a?G(l,[1&a&&{headers:e[0]},2&a&&{rows:e[1]},8&a&&F(e[3])]):{};t.$set(n)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function ol(e){let t,a,l,n;const _=[il,rl],d=[];function p(e,t){return e[2]?1:0}return t=p(e),a=d[t]=_[t](e),{c(){a.c(),l=P()},m(e,a){d[t].m(e,a),s(e,l,a),n=!0},p(e,[n]){let s=t;t=p(e),t===s?d[t].p(e,n):(i(),c(d[s],1,1,(()=>{d[s]=null})),o(),a=d[t],a?a.p(e,n):(a=d[t]=_[t](e),a.c()),r(a,1),a.m(l.parentNode,l))},i(e){n||(r(a),n=!0)},o(e){c(a),n=!1},d(e){e&&u(l),d[t].d(e)}}}function cl(e,t,a){const l=["frameProps","data","headers","rows"];let n=N(t,l),{frameProps:_={class:"pipen-report-datatable-frame"}}=t,{data:d={}}=t,{headers:s=[]}=t,{rows:r=[]}=t;if(s.length+r.length>0&&d.length>0)throw"Can only specify `data` or `headers`/`rows` for `DataTable`!";return d.length>0&&(r=d,s=Object.keys(r[0]).filter((e=>"id"!==e)).map((e=>({key:e,value:e})))),e.$$set=e=>{t=D(D({},t),y(e)),a(3,n=N(t,l)),"frameProps"in e&&a(2,_=e.frameProps),"data"in e&&a(4,d=e.data),"headers"in e&&a(0,s=e.headers),"rows"in e&&a(1,r=e.rows)},[s,r,_,n,d]}class ul extends t{constructor(e){super(),a(this,e,cl,ol,l,{frameProps:2,data:4,headers:0,rows:1})}}function pl(e){let t,a;return{c(){t=_("div"),a=$(e[0]),d(t,"class","pipen-report-descr-title svelte-1xzxuva")},m(e,l){s(e,t,l),S(t,a)},p(e,t){1&t&&b(a,e[0])},d(e){e&&u(t)}}}function vl(e){let t,a,l,n=e[0]&&pl(e);const d=e[3].default,i=L(d,e,e[2],null);let o=[e[1]],p={};for(let e=0;e<o.length;e+=1)p=D(p,o[e]);return{c(){t=_("div"),n&&n.c(),a=C(),i&&i.c(),U(t,p),H(t,"svelte-1xzxuva",!0)},m(e,_){s(e,t,_),n&&n.m(t,null),S(t,a),i&&i.m(t,null),l=!0},p(e,[_]){e[0]?n?n.p(e,_):(n=pl(e),n.c(),n.m(t,a)):n&&(n.d(1),n=null),i&&i.p&&(!l||4&_)&&j(i,d,e,e[2],l?M(d,e[2],_,null):B(e[2]),null),U(t,p=G(o,[2&_&&e[1]])),H(t,"svelte-1xzxuva",!0)},i(e){l||(r(i,e),l=!0)},o(e){c(i,e),l=!1},d(e){e&&u(t),n&&n.d(),i&&i.d(e)}}}function Pl(e,t,a){const l=["title"];let n=N(t,l),{$$slots:_={},$$scope:d}=t,{title:s}=t,{klass:r,...i}=n;return r=r?`pipen-report-descr ${r}`:"pipen-report-descr",i||(i={}),i.class=r,e.$$set=e=>{t=D(D({},t),y(e)),a(5,n=N(t,l)),"title"in e&&a(0,s=e.title),"$$scope"in e&&a(2,d=e.$$scope)},[s,i,d,_]}class ml extends t{constructor(e){super(),a(this,e,Pl,vl,l,{title:0})}}function Al(e){let t,a,l,n,_,d;return t=new ve({props:{label:"Significant Markers"}}),l=new ve({props:{label:"Volcano Plot"}}),_=new ve({props:{label:"Dot Plot"}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment),n=C(),m(_.$$.fragment)},m(e,r){A(t,e,r),s(e,a,r),A(l,e,r),s(e,n,r),A(_,e,r),d=!0},p:T,i(e){d||(r(t.$$.fragment,e),r(l.$$.fragment,e),r(_.$$.fragment,e),d=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),c(_.$$.fragment,e),d=!1},d(e){e&&(u(a),u(n)),g(t,e),g(l,e),g(_,e)}}}function gl(e){let t;return{c(){t=$("The markers are found using Seurat's FindMarkers function, and filtered by: p_val_adj < 0.05 & avg_log2FC > 0.1")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function Cl(e){let t,a,l,n;return t=new ml({props:{$$slots:{default:[gl]},$$scope:{ctx:e}}}),l=new ul({props:{src:"../ClusterMarkers/samples.markers/Cluster/c17/markers.txt",data:[{p_val:0,avg_log2FC:3.5233,pct_1:.731,pct_2:.147,p_val_adj:0,cluster:"c17",gene:"GZMB",id:0},{p_val:0,avg_log2FC:2.0778,pct_1:.642,pct_2:.279,p_val_adj:0,cluster:"c17",gene:"CD74",id:1},{p_val:0,avg_log2FC:4.0979,pct_1:.235,pct_2:.017,p_val_adj:0,cluster:"c17",gene:"STMN1",id:2},{p_val:0,avg_log2FC:4.0371,pct_1:.19,pct_2:.02,p_val_adj:0,cluster:"c17",gene:"IRF7",id:3},{p_val:0,avg_log2FC:4.158,pct_1:.157,pct_2:.005,p_val_adj:0,cluster:"c17",gene:"IGJ",id:4},{p_val:0,avg_log2FC:5.1541,pct_1:.145,pct_2:.006,p_val_adj:0,cluster:"c17",gene:"IRF8",id:5},{p_val:0,avg_log2FC:3.581,pct_1:.156,pct_2:.018,p_val_adj:0,cluster:"c17",gene:"NPC2",id:6},{p_val:0,avg_log2FC:7.0358,pct_1:.13,pct_2:.001,p_val_adj:0,cluster:"c17",gene:"PLD4",id:7},{p_val:0,avg_log2FC:8.334,pct_1:.112,pct_2:0,p_val_adj:0,cluster:"c17",gene:"LILRA4",id:8},{p_val:0,avg_log2FC:4.3315,pct_1:.114,pct_2:.007,p_val_adj:0,cluster:"c17",gene:"NAPSB",id:9},{p_val:0,avg_log2FC:5.2085,pct_1:.098,pct_2:.003,p_val_adj:0,cluster:"c17",gene:"TCF4",id:10},{p_val:0,avg_log2FC:4.9479,pct_1:.092,pct_2:.003,p_val_adj:0,cluster:"c17",gene:"CCDC50",id:11},{p_val:0,avg_log2FC:7.5621,pct_1:.088,pct_2:.001,p_val_adj:0,cluster:"c17",gene:"SERPINF1",id:12},{p_val:0,avg_log2FC:3.8407,pct_1:.092,pct_2:.007,p_val_adj:0,cluster:"c17",gene:"UGCG",id:13},{p_val:0,avg_log2FC:6.189,pct_1:.081,pct_2:.001,p_val_adj:0,cluster:"c17",gene:"TSPAN13",id:14},{p_val:0,avg_log2FC:5.3048,pct_1:.079,pct_2:.002,p_val_adj:0,cluster:"c17",gene:"APP",id:15},{p_val:0,avg_log2FC:5.993,pct_1:.076,pct_2:.003,p_val_adj:0,cluster:"c17",gene:"TCL1A",id:16},{p_val:0,avg_log2FC:4.0894,pct_1:.077,pct_2:.005,p_val_adj:0,cluster:"c17",gene:"MZB1",id:17},{p_val:0,avg_log2FC:6.0005,pct_1:.067,pct_2:.001,p_val_adj:0,cluster:"c17",gene:"DERL3",id:18},{p_val:0,avg_log2FC:4.2422,pct_1:.07,pct_2:.004,p_val_adj:0,cluster:"c17",gene:"KIAA0101",id:19},{p_val:0,avg_log2FC:8.397,pct_1:.064,pct_2:0,p_val_adj:0,cluster:"c17",gene:"IL3RA",id:20},{p_val:0,avg_log2FC:5.3448,pct_1:.065,pct_2:.002,p_val_adj:0,cluster:"c17",gene:"LOC101929219",id:21},{p_val:0,avg_log2FC:4.809,pct_1:.062,pct_2:.002,p_val_adj:0,cluster:"c17",gene:"SPIB",id:22},{p_val:0,avg_log2FC:4.5603,pct_1:.061,pct_2:.003,p_val_adj:0,cluster:"c17",gene:"TYMS",id:23},{p_val:0,avg_log2FC:8.8079,pct_1:.057,pct_2:0,p_val_adj:0,cluster:"c17",gene:"CLEC4C",id:24},{p_val:0,avg_log2FC:4.7983,pct_1:.057,pct_2:.002,p_val_adj:0,cluster:"c17",gene:"RNASE6",id:25},{p_val:0,avg_log2FC:4.3377,pct_1:.056,pct_2:.003,p_val_adj:0,cluster:"c17",gene:"RRM2",id:26},{p_val:0,avg_log2FC:4.4756,pct_1:.052,pct_2:.002,p_val_adj:0,cluster:"c17",gene:"UBE2C",id:27},{p_val:0,avg_log2FC:5.9557,pct_1:.051,pct_2:.001,p_val_adj:0,cluster:"c17",gene:"TPM2",id:28},{p_val:0,avg_log2FC:5.8436,pct_1:.048,pct_2:.001,p_val_adj:0,cluster:"c17",gene:"SMPD3",id:29},{p_val:0,avg_log2FC:6.3224,pct_1:.045,pct_2:.001,p_val_adj:0,cluster:"c17",gene:"LILRB4",id:30},{p_val:0,avg_log2FC:5.769,pct_1:.039,pct_2:.001,p_val_adj:0,cluster:"c17",gene:"BCL11A",id:31},{p_val:0,avg_log2FC:8.7375,pct_1:.038,pct_2:0,p_val_adj:0,cluster:"c17",gene:"SCT",id:32},{p_val:0,avg_log2FC:8.3448,pct_1:.032,pct_2:0,p_val_adj:0,cluster:"c17",gene:"PTCRA",id:33},{p_val:0,avg_log2FC:6.1447,pct_1:.028,pct_2:0,p_val_adj:0,cluster:"c17",gene:"PPP1R14A",id:34},{p_val:0,avg_log2FC:7.6444,pct_1:.025,pct_2:0,p_val_adj:0,cluster:"c17",gene:"LAMP5",id:35},{p_val:0,avg_log2FC:6.4155,pct_1:.023,pct_2:0,p_val_adj:0,cluster:"c17",gene:"FUT7",id:36},{p_val:0,avg_log2FC:7.284,pct_1:.018,pct_2:0,p_val_adj:0,cluster:"c17",gene:"MAP1A",id:37},{p_val:0,avg_log2FC:7.5855,pct_1:.017,pct_2:0,p_val_adj:0,cluster:"c17",gene:"MYBL2",id:38},{p_val:0,avg_log2FC:8.8585,pct_1:.014,pct_2:0,p_val_adj:0,cluster:"c17",gene:"SMIM5",id:39},{p_val:0,avg_log2FC:8.6444,pct_1:.013,pct_2:0,p_val_adj:0,cluster:"c17",gene:"KRT5",id:40},{p_val:0,avg_log2FC:9.3224,pct_1:.011,pct_2:0,p_val_adj:0,cluster:"c17",gene:"SHD",id:41},{p_val:6867e-302,avg_log2FC:2.8369,pct_1:.19,pct_2:.034,p_val_adj:133e-296,cluster:"c17",gene:"ITM2C",id:42},{p_val:6923e-298,avg_log2FC:4.3665,pct_1:.049,pct_2:.003,p_val_adj:1.341e-290,cluster:"c17",gene:"NUSAP1",id:43},{p_val:3189e-291,avg_log2FC:3.0342,pct_1:.252,pct_2:.059,p_val_adj:6178e-287,cluster:"c17",gene:"TUBA1B",id:44},{p_val:8118e-289,avg_log2FC:1.8061,pct_1:.404,pct_2:.13,p_val_adj:1572e-284,cluster:"c17",gene:"ALOX5AP",id:45},{p_val:3421e-282,avg_log2FC:2.5197,pct_1:.202,pct_2:.04,p_val_adj:6627e-278,cluster:"c17",gene:"HLA-DRB4",id:46},{p_val:1644e-266,avg_log2FC:3.542,pct_1:.081,pct_2:.008,p_val_adj:3184e-262,cluster:"c17",gene:"PPP1R14B",id:47},{p_val:3551e-260,avg_log2FC:2.4915,pct_1:.221,pct_2:.05,p_val_adj:6878e-256,cluster:"c17",gene:"SEC61B",id:48},{p_val:6033e-250,avg_log2FC:3.0047,pct_1:.136,pct_2:.022,p_val_adj:1169e-245,cluster:"c17",gene:"PLAC8",id:49},{p_val:106e-245,avg_log2FC:2.6062,pct_1:.152,pct_2:.026,p_val_adj:2053e-242,cluster:"c17",gene:"CST3",id:50},{p_val:1704e-246,avg_log2FC:2.8086,pct_1:.184,pct_2:.037,p_val_adj:3302e-242,cluster:"c17",gene:"TXN",id:51},{p_val:3111e-245,avg_log2FC:2.4391,pct_1:.184,pct_2:.037,p_val_adj:6026e-241,cluster:"c17",gene:"LOC100996809",id:52},{p_val:2523e-238,avg_log2FC:2.5009,pct_1:.291,pct_2:.086,p_val_adj:4886e-234,cluster:"c17",gene:"HMGN2",id:53},{p_val:1708e-236,avg_log2FC:3.0436,pct_1:.16,pct_2:.03,p_val_adj:3308e-232,cluster:"c17",gene:"CLIC3",id:54},{p_val:244e-235,avg_log2FC:2.3059,pct_1:.199,pct_2:.044,p_val_adj:4726e-232,cluster:"c17",gene:"C12orf75",id:55},{p_val:7592e-236,avg_log2FC:2.9789,pct_1:.187,pct_2:.04,p_val_adj:1471e-231,cluster:"c17",gene:"TUBB",id:56},{p_val:2011e-231,avg_log2FC:6.2155,pct_1:.013,pct_2:0,p_val_adj:3896e-227,cluster:"c17",gene:"DNASE1L3",id:57},{p_val:9213e-230,avg_log2FC:4.4168,pct_1:.036,pct_2:.002,p_val_adj:1785e-225,cluster:"c17",gene:"CDK1",id:58},{p_val:2408e-220,avg_log2FC:3.9054,pct_1:.046,pct_2:.003,p_val_adj:4664e-216,cluster:"c17",gene:"CYB561A3",id:59},{p_val:6355e-217,avg_log2FC:4.1569,pct_1:.037,pct_2:.002,p_val_adj:1231e-212,cluster:"c17",gene:"TOP2A",id:60},{p_val:1159e-211,avg_log2FC:4.011,pct_1:.042,pct_2:.003,p_val_adj:2246e-207,cluster:"c17",gene:"MPEG1",id:61},{p_val:3.48e-200,avg_log2FC:2.3465,pct_1:.174,pct_2:.039,p_val_adj:6741e-199,cluster:"c17",gene:"HSP90B1",id:62},{p_val:972e-195,avg_log2FC:3.5047,pct_1:.058,pct_2:.005,p_val_adj:1883e-191,cluster:"c17",gene:"SOX4",id:63},{p_val:1821e-192,avg_log2FC:4.6625,pct_1:.024,pct_2:.001,p_val_adj:3528e-188,cluster:"c17",gene:"CDC20",id:64},{p_val:1852e-179,avg_log2FC:2.156,pct_1:.173,pct_2:.042,p_val_adj:3588e-175,cluster:"c17",gene:"SPCS1",id:65},{p_val:2844e-178,avg_log2FC:1.9176,pct_1:.209,pct_2:.058,p_val_adj:551e-173,cluster:"c17",gene:"PARK7",id:66},{p_val:4252e-178,avg_log2FC:4.6971,pct_1:.023,pct_2:.001,p_val_adj:8237e-174,cluster:"c17",gene:"GAPT",id:67},{p_val:6463e-170,avg_log2FC:1.472,pct_1:.579,pct_2:.344,p_val_adj:1252e-165,cluster:"c17",gene:"GAPDH",id:68},{p_val:8637e-170,avg_log2FC:1.6732,pct_1:.196,pct_2:.053,p_val_adj:1673e-165,cluster:"c17",gene:"CXCL13",id:69},{p_val:1614e-167,avg_log2FC:4.962,pct_1:.019,pct_2:.001,p_val_adj:3.127e-160,cluster:"c17",gene:"TNFRSF21",id:70},{p_val:8975e-166,avg_log2FC:1.5679,pct_1:.303,pct_2:.11,p_val_adj:1739e-161,cluster:"c17",gene:"PPIB",id:71},{p_val:2241e-164,avg_log2FC:2.4569,pct_1:.17,pct_2:.044,p_val_adj:4341e-160,cluster:"c17",gene:"HMGB2",id:72},{p_val:1.033e-160,avg_log2FC:4.863,pct_1:.019,pct_2:.001,p_val_adj:2001e-159,cluster:"c17",gene:"CCNB1",id:73},{p_val:3.945e-160,avg_log2FC:2.1408,pct_1:.2,pct_2:.057,p_val_adj:7642e-159,cluster:"c17",gene:"GPX1",id:74},{p_val:3521e-156,avg_log2FC:2.772,pct_1:.08,pct_2:.012,p_val_adj:682e-151,cluster:"c17",gene:"HIGD1A",id:75},{p_val:5908e-154,avg_log2FC:2.6871,pct_1:.084,pct_2:.013,p_val_adj:1144e-149,cluster:"c17",gene:"VIMP",id:76},{p_val:3199e-150,avg_log2FC:4.4615,pct_1:.059,pct_2:.007,p_val_adj:6197e-146,cluster:"c17",gene:"PTGDS",id:77},{p_val:4296e-150,avg_log2FC:4.3075,pct_1:.024,pct_2:.001,p_val_adj:8322e-146,cluster:"c17",gene:"CLN8",id:78},{p_val:1.58e-140,avg_log2FC:3.8042,pct_1:.029,pct_2:.002,p_val_adj:306e-138,cluster:"c17",gene:"MKI67",id:79},{p_val:4.109e-140,avg_log2FC:4.3072,pct_1:.023,pct_2:.001,p_val_adj:7961e-139,cluster:"c17",gene:"TK1",id:80},{p_val:2492e-135,avg_log2FC:1.7678,pct_1:.191,pct_2:.058,p_val_adj:4827e-131,cluster:"c17",gene:"SLC25A5",id:81},{p_val:6087e-129,avg_log2FC:1.6757,pct_1:.322,pct_2:.142,p_val_adj:1179e-124,cluster:"c17",gene:"HMGB1",id:82},{p_val:6832e-128,avg_log2FC:5.055,pct_1:.013,pct_2:0,p_val_adj:1.323e-120,cluster:"c17",gene:"P2RY14",id:83},{p_val:2234e-127,avg_log2FC:3.7416,pct_1:.028,pct_2:.002,p_val_adj:4.327e-120,cluster:"c17",gene:"CENPF",id:84},{p_val:1247e-125,avg_log2FC:2.4553,pct_1:.091,pct_2:.018,p_val_adj:2416e-121,cluster:"c17",gene:"PTTG1",id:85},{p_val:5325e-124,avg_log2FC:1.4473,pct_1:.276,pct_2:.11,p_val_adj:1032e-119,cluster:"c17",gene:"LOC101929531",id:86},{p_val:6733e-124,avg_log2FC:1.6723,pct_1:.227,pct_2:.082,p_val_adj:1304e-119,cluster:"c17",gene:"TPI1",id:87},{p_val:2.262e-120,avg_log2FC:2.4183,pct_1:.1,pct_2:.021,p_val_adj:4382e-119,cluster:"c17",gene:"FABP5",id:88},{p_val:1923e-122,avg_log2FC:1.0853,pct_1:.469,pct_2:.252,p_val_adj:3725e-118,cluster:"c17",gene:"PPIA",id:89},{p_val:3531e-121,avg_log2FC:2.361,pct_1:.095,pct_2:.02,p_val_adj:6841e-117,cluster:"c17",gene:"CTSB",id:90},{p_val:8465e-119,avg_log2FC:1.0341,pct_1:.45,pct_2:.235,p_val_adj:164e-113,cluster:"c17",gene:"MYL6",id:91},{p_val:6857e-114,avg_log2FC:2.2558,pct_1:.095,pct_2:.021,p_val_adj:1328e-109,cluster:"c17",gene:"PTMS",id:92},{p_val:18e-110,avg_log2FC:1.9972,pct_1:.126,pct_2:.033,p_val_adj:3486e-108,cluster:"c17",gene:"H2AFV",id:93},{p_val:5775e-112,avg_log2FC:1.7083,pct_1:.16,pct_2:.049,p_val_adj:1119e-107,cluster:"c17",gene:"GNG5",id:94},{p_val:1174e-110,avg_log2FC:2.7943,pct_1:.054,pct_2:.008,p_val_adj:2274e-106,cluster:"c17",gene:"GRN",id:95},{p_val:1037e-109,avg_log2FC:3.8199,pct_1:.024,pct_2:.002,p_val_adj:2009e-105,cluster:"c17",gene:"FAM129C",id:96},{p_val:5736e-105,avg_log2FC:1.8366,pct_1:.136,pct_2:.04,p_val_adj:1.111e-97,cluster:"c17",gene:"LAG3",id:97},{p_val:5.238e-100,avg_log2FC:1.5856,pct_1:.18,pct_2:.062,p_val_adj:1015e-98,cluster:"c17",gene:"RAN",id:98},{p_val:1.744e-97,avg_log2FC:1.8382,pct_1:.199,pct_2:.076,p_val_adj:3379e-96,cluster:"c17",gene:"H2AFZ",id:99}]}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Ol(e){let t;return{c(){t=$("The volcano plot is generated using ggplot2. The significant markers are highlighted in blue and red, indicating up-regulated and down-regulated genes, respectively. The non-significant genes are shown in grey.")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function Rl(e){let t,a,l,n;return t=new ml({props:{$$slots:{default:[Ol]},$$scope:{ctx:e}}}),l=new Ia({props:{src:"../ClusterMarkers/samples.markers/Cluster/c17/volcano.png",download:"../ClusterMarkers/samples.markers/Cluster/c17/volcano.pdf",width:900,height:1200}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function fl(e){let t;return{c(){t=$("The dot plot is generated using Seurat's DotPlot function. The top significant markers are used as input. The sizes of the dots are proportional to the percentage of cells expressing the gene in each group. The colors of the dots are proportional to the average expression of the gene in each group.")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function $l(e){let t,a,l,n;return t=new ml({props:{$$slots:{default:[fl]},$$scope:{ctx:e}}}),l=new Ia({props:{src:"../ClusterMarkers/samples.markers/Cluster/c17/dotplot.png",download:"../ClusterMarkers/samples.markers/Cluster/c17/dotplot.pdf",width:600,height:1200}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Sl(e){let t,a,l,n,_,d;return t=new Pe({props:{$$slots:{default:[Cl]},$$scope:{ctx:e}}}),l=new Pe({props:{$$slots:{default:[Rl]},$$scope:{ctx:e}}}),_=new Pe({props:{$$slots:{default:[$l]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment),n=C(),m(_.$$.fragment)},m(e,r){A(t,e,r),s(e,a,r),A(l,e,r),s(e,n,r),A(_,e,r),d=!0},p(e,a){const n={};1&a&&(n.$$scope={dirty:a,ctx:e}),t.$set(n);const d={};1&a&&(d.$$scope={dirty:a,ctx:e}),l.$set(d);const s={};1&a&&(s.$$scope={dirty:a,ctx:e}),_.$set(s)},i(e){d||(r(t.$$.fragment,e),r(l.$$.fragment,e),r(_.$$.fragment,e),d=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),c(_.$$.fragment,e),d=!1},d(e){e&&(u(a),u(n)),g(t,e),g(l,e),g(_,e)}}}function bl(e){let t;return{c(){t=$("The enrichment analysis is done using Enrichr. The significant markers are used as input.")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function hl(e){let t,a,l,n;return t=new ve({props:{label:"Plot"}}),l=new ve({props:{label:"Table"}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p:T,i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Tl(e){let t;return{c(){t=$("The enrichment analysis results of the top biological pathways associated with the input gene set. Each bar represents a pathway, with the length of the bar indicating the number of input genes overlapping with genes in that pathway. The color intensity of the bars reflects the statistical significance of the enrichment (p-value).")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function Ll(e){let t,a,l,n;return t=new ml({props:{$$slots:{default:[Tl]},$$scope:{ctx:e}}}),l=new Ia({props:{src:"../ClusterMarkers/samples.markers/Cluster/c17/Enrichr-MSigDB_Hallmark_2020.png",download:"../ClusterMarkers/samples.markers/Cluster/c17/Enrichr-MSigDB_Hallmark_2020.pdf",width:1e3,height:1e3}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function jl(e){let t,a;return t=new ul({props:{src:"../ClusterMarkers/samples.markers/Cluster/c17/Enrichr-MSigDB_Hallmark_2020.txt",data:[{Term:"E2F Targets",Overlap:"56/200",P_value:4755e-40,Adjusted_P_value:233e-37,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:12.8187,Combined_Score:1072.1074,Genes:"TOP2A;MCM7;HMGB2;MKI67;SMC4;CKS1B;LMNB1;CDC20;PTTG1;STMN1;MYBL2;TK1;KPNA2;LBR;TMPO;TUBB;KIF22;DCK;DDX39A;MTHFD2;CKS2;SRSF2;MCM3;BIRC5;MCM5;MCM6;DCTPP1;ASF1B;SNRPB;SLBP;DNMT1;PCNA;RNASEH2A;CDCA8;AURKB;CCNB2;RAD21;RBBP7;RANBP1;DUT;RRM2;PLK1;ATAD2;HMGA1;NAP1L1;PA2G4;DEK;NASP;RPA3;UBE2T;ANP32E;CDK1;CENPM;RAN;MAD2L1;CDKN3",id:0},{Term:"Myc Targets V1",Overlap:"55/200",P_value:6057e-39,Adjusted_P_value:1484e-37,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:12.4809,Combined_Score:1012.0961,Genes:"SLC25A3;HSP90AB1;MCM7;HNRNPR;PSMD8;CDC20;SNRPD1;C1QBP;SNRPD3;KPNA2;ILF2;CCNA2;PSMA6;PSMA4;TFDP1;PSMA2;CANX;SRSF2;SNRPG;NHP2;SRSF3;VDAC1;ERH;MCM5;HPRT1;MCM6;PPIA;SET;PCNA;PHB;TYMS;COX5A;PSMA7;PRDX3;LDHA;PSMB3;PCBP1;PGK1;BUB3;HNRNPA1;RANBP1;NPM1;DUT;HNRNPA3;RRM1;CBX3;NAP1L1;PA2G4;DEK;EIF2S2;LSM2;LSM7;HNRNPA2B1;RAN;MAD2L1",id:1},{Term:"G2-M Checkpoint",Overlap:"44/200",P_value:1405e-27,Adjusted_P_value:2295e-26,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:9.1039,Combined_Score:499.9996,Genes:"TOP2A;DTYMK;MKI67;SMC4;AURKB;SMC2;LMNB1;CKS1B;CDC20;CCNB2;MT2A;PTTG1;SNRPD1;RAD21;STMN1;NUSAP1;MYBL2;BUB3;KPNA2;HMGN2;LBR;TMPO;UBE2C;PLK1;HMGA1;KIF22;CCNA2;TPX2;SLC7A5;CENPF;DDX39A;TFDP1;NASP;NCL;CKS2;SRSF2;CDK1;MCM3;BIRC5;MCM5;MCM6;KIF20B;MAD2L1;CDKN3",id:2},{Term:"Oxidative Phosphorylation",Overlap:"33/200",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:6.2588,Combined_Score:203.1628,Genes:"COX7B;SLC25A3;NDUFB8;NDUFB4;COX17;TIMM13;NDUFB2;NDUFB1;COX7A2;UQCR10;ETFB;COX6A1;COX5A;PRDX3;LDHA;UQCRFS1;IDH3A;COX8A;ATP6V0B;NDUFA7;NDUFA6;NDUFA4;IDH2;NDUFA1;NDUFC2;COX6C;COX6B1;NDUFS8;UQCRQ;NDUFS6;VDAC1;SLC25A5;ATP6V0C",id:3},{Term:"mTORC1 Signaling",Overlap:"31/200",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.7902,Combined_Score:166.9404,Genes:"FKBP2;ARPC5L;ENO1;CORO1A;HSP90B1;LDHA;PRDX1;PGK1;CTSC;ACTR3;TPI1;SDF2L1;RRM2;HSPA5;PLK1;EIF2S2;DHFR;CCT6A;SLC7A5;DDX39A;PSMA4;MTHFD2;PPA1;CANX;TCEA1;CD9;HPRT1;CALR;ALDOA;PPIA;GAPDH",id:4},{Term:"Interferon Gamma Response",Overlap:"30/200",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.561,Combined_Score:150.5645,Genes:"IFITM3;UBE2L6;IFI35;IFI44L;PSMB10;MT2A;HLA-DMA;CCL5;SLAMF7;CD74;P2RY14;GZMA;MX1;TAP1;ISG15;PSMB8;PSMB9;BST2;VAMP8;PLSCR1;MTHFD2;PSMA2;IRF4;PSME1;IRF7;PSME2;IRF8;LAP3;HLA-DRB1;LY6E",id:5},{Term:"Interferon Alpha Response",Overlap:"18/97",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:7.0747,Combined_Score:143.6717,Genes:"IFITM3;CD74;CSF1;MX1;TAP1;UBE2L6;ISG15;IFI35;IFI44L;PSMB8;PSMB9;BST2;PLSCR1;PSME1;IRF7;PSME2;LAP3;LY6E",id:6},{Term:"Allograft Rejection",Overlap:"23/200",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.0468,Combined_Score:64.5629,Genes:"EIF5A;CD74;NPM1;CSF1;GZMA;PRF1;TAP1;GZMB;CAPG;CXCL13;CD3D;PSMB10;HLA-DMA;HLA-DMB;IFNG;CD8A;IRF4;CXCR3;CCL5;CD7;IRF7;IRF8;CSK",id:7},{Term:"Complement",Overlap:"22/200",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.8427,Combined_Score:55.9035,Genes:"DUSP5;FCER1G;APOBEC3G;HSPA5;GZMA;ANXA5;PLEK;GZMB;PSMB9;LGALS3;PLSCR1;PPP4C;GNB2;CCL5;IRF7;CTSH;CALM3;LAP3;PFN1;CTSD;CTSC;CTSB",id:8},{Term:"Unfolded Protein Response",Overlap:"16/113",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.1004,Combined_Score:72.974,Genes:"NPM1;HSPA5;PDIA6;LSM4;YIF1A;HSP90B1;HERPUD1;CKS1B;SLC7A5;SPCS3;SPCS1;MTHFD2;NHP2;BANF1;CALR;ATF3",id:9},{Term:"Estrogen Response Late",Overlap:"17/200",P_value:2e-4,Adjusted_P_value:.001,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.8642,Combined_Score:23.7971,Genes:"TOP2A;CLIC3;RNASEH2A;ARL3;DCXR;IDH2;ETFB;COX6C;AREG;CDC20;SLC7A5;TSPAN13;FABP5;PPIF;CD9;HPRT1;NBL1",id:10},{Term:"Glycolysis",Overlap:"17/200",P_value:2e-4,Adjusted_P_value:.001,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.8642,Combined_Score:23.7971,Genes:"TPI1;HSPA5;PGAM1;TALDO1;ENO1;MIF;TXN;LDHA;PKM;NASP;CHST12;STMN1;PGK1;CDK1;TGFBI;ALDOA;PPIA",id:11},{Term:"PI3K/AKT/mTOR  Signaling",Overlap:"11/105",P_value:5e-4,Adjusted_P_value:.002,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.5902,Combined_Score:27.0571,Genes:"ACTR3;ARF1;ARPC3;CFL1;CSNK2B;CDK1;CALR;PFN1;AP2M1;HSP90B1;PPP1CA",id:12},{Term:"Mitotic Spindle",Overlap:"16/199",P_value:7e-4,Adjusted_P_value:.0024,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.6914,Combined_Score:19.613,Genes:"TOP2A;PLK1;KIF22;SMC4;CD2AP;LMNB1;CCDC88A;TPX2;CCNB2;CENPF;CAPZB;NUSAP1;CDK1;BIRC5;KIF20B;ARF6",id:13},{Term:"Hypoxia",Overlap:"16/200",P_value:7e-4,Adjusted_P_value:.0024,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.6767,Combined_Score:19.3605,Genes:"TPI1;ANXA2;HSPA5;ENO1;MIF;RBPJ;PLAC8;MT2A;LDHA;PRDX5;PGK1;TGFBI;ALDOA;GAPDH;ATF3;MT1E",id:14},{Term:"Reactive Oxygen Species Pathway",Overlap:"7/49",P_value:9e-4,Adjusted_P_value:.0027,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.0947,Combined_Score:35.8443,Genes:"NDUFA6;NDUFB4;PRDX1;LSP1;TXN;PRDX6;LAMTOR5",id:15},{Term:"Protein Secretion",Overlap:"10/96",P_value:.001,Adjusted_P_value:.0028,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.5632,Combined_Score:24.662,Genes:"CD63;LMAN1;ARF1;AP2S1;PPT1;ANP32E;CLTA;AP2M1;CTSC;COPE",id:16},{Term:"Myc Targets V2",Overlap:"7/58",P_value:.0024,Adjusted_P_value:.0063,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.1937,Combined_Score:25.2869,Genes:"NPM1;CBX3;PLK1;MCM5;PHB;PA2G4;DCTPP1",id:17},{Term:"Cholesterol Homeostasis",Overlap:"8/74",P_value:.0025,Adjusted_P_value:.0063,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.7065,Combined_Score:22.2808,Genes:"LGALS3;PLSCR1;FABP5;ANXA5;CD9;ATF3;S100A11;ACTG1",id:18},{Term:"DNA Repair",Overlap:"12/150",P_value:.0032,Adjusted_P_value:.0079,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.666,Combined_Score:15.3092,Genes:"FEN1;DUT;PCNA;ARL6IP1;RPA3;COX17;EDF1;HPRT1;TYMS;ZWINT;ADA;RBX1",id:19},{Term:"Fatty Acid Metabolism",Overlap:"12/158",P_value:.0049,Adjusted_P_value:.0109,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.5188,Combined_Score:13.4105,Genes:"LDHA;LGALS1;OSTC;PSME1;ERP29;UBE2L6;MIF;ALDOA;GLUL;PRDX6;S100A10;YWHAH",id:20},{Term:"IL-2/STAT5 Signaling",Overlap:"14/199",P_value:.0049,Adjusted_P_value:.0109,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.3219,Combined_Score:12.3499,Genes:"IFITM3;CSF1;GSTO1;TNFRSF9;TNFRSF18;CAPG;ITGAE;PLSCR1;SYNGR2;IRF4;MAPKAPK2;IL3RA;IRF8;TNFRSF21",id:21},{Term:"Apoptosis",Overlap:"12/161",P_value:.0057,Adjusted_P_value:.012,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.4677,Combined_Score:12.7725,Genes:"IFITM3;TOP2A;APP;LGALS3;GNA15;GPX1;SLC20A1;PRF1;PPT1;HMGB2;TAP1;ATF3",id:22},{Term:"Adipogenesis",Overlap:"13/200",P_value:.0123,Adjusted_P_value:.0223,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.1293,Combined_Score:9.3662,Genes:"COX8A;COX7B;MTCH2;TALDO1;UQCR10;ETFB;COX6A1;DNAJC15;PRDX3;UQCRQ;UCP2;ALDOA;IDH3A",id:23},{Term:"Apical Junction",Overlap:"13/200",P_value:.0123,Adjusted_P_value:.0223,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.1293,Combined_Score:9.3662,Genes:"CAP1;SYK;LAYN;ACTB;MYL12B;ACTG1;ARPC2;RAC2;ZYX;TGFBI;PFN1;CD99;YWHAH",id:24},{Term:"Inflammatory Response",Overlap:"13/200",P_value:.0123,Adjusted_P_value:.0223,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.1293,Combined_Score:9.3662,Genes:"CSF1;CD82;CD70;TNFRSF9;CYBB;BST2;PTPRE;GNA15;RGS1;GPR183;CCL5;IRF7;LY6E",id:25},{Term:"p53 Pathway",Overlap:"13/200",P_value:.0123,Adjusted_P_value:.0223,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.1293,Combined_Score:9.3662,Genes:"APP;PCNA;CD82;DCXR;TAP1;SLC3A2;VAMP8;PTPRE;HINT1;CTSD;ATF3;ADA;S100A10",id:26},{Term:"TNF-alpha Signaling via NF-kB",Overlap:"12/200",P_value:.0275,Adjusted_P_value:.0464,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.9518,Combined_Score:7.0162,Genes:"DUSP4;DUSP5;PTPRE;CSF1;GPR183;TNFRSF9;CCL5;PLEK;TAP1;AREG;ATF3;GEM",id:27},{Term:"Xenobiotic Metabolism",Overlap:"12/200",P_value:.0275,Adjusted_P_value:.0464,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.9518,Combined_Score:7.0162,Genes:"MT2A;GSTO1;DDAH2;DCXR;SSR3;IRF8;TMBIM6;SPINT2;HPRT1;PTGDS;COMT;PSMB10",id:28},{Term:"UV Response Up",Overlap:"10/158",P_value:.0307,Adjusted_P_value:.0501,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.0639,Combined_Score:7.1926,Genes:"FEN1;PSMC3;PPIF;PPT1;AP2S1;TAP1;RAB27A;TMBIM6;ALDOA;ATF3",id:29},{Term:"TGF-beta Signaling",Overlap:"4/54",P_value:.0936,Adjusted_P_value:.1479,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.4329,Combined_Score:5.7639,Genes:"FKBP1A;SLC20A1;RHOA;PPP1CA",id:30},{Term:"Androgen Response",Overlap:"6/100",P_value:.101,Adjusted_P_value:.1546,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.9428,Combined_Score:4.4548,Genes:"LMAN1;SPCS3;DBI;PA2G4;MYL12A;UBE2J1",id:31},{Term:"Epithelial Mesenchymal Transition",Overlap:"10/200",P_value:.1088,Adjusted_P_value:.1616,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.6041,Combined_Score:3.5583,Genes:"LGALS1;TPM4;MCM7;TPM2;CAPG;TGFBI;VIM;PPIB;AREG;GEM",id:32},{Term:"Pperoxisome",Overlap:"6/104",P_value:.116,Adjusted_P_value:.1672,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.8631,Combined_Score:4.0135,Genes:"TOP2A;PRDX5;PRDX1;IDH2;CLN8;YWHAH",id:33},{Term:"Pancreas Beta Cells",Overlap:"3/40",P_value:.135,Adjusted_P_value:.1889,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.4635,Combined_Score:4.9338,Genes:"SPCS1;SRP14;SRP9",id:34},{Term:"IL-6/JAK/STAT3 Signaling",Overlap:"5/87",P_value:.1457,Adjusted_P_value:.1982,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.8542,Combined_Score:3.5721,Genes:"CSF1;IL3RA;CD9;CXCL13;TNFRSF21",id:35},{Term:"Coagulation",Overlap:"6/138",P_value:.2796,Adjusted_P_value:.3703,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.3808,Combined_Score:1.7597,Genes:"WDR1;GNB2;PLEK;CTSH;CD9;CTSB",id:36},{Term:"Angiogenesis",Overlap:"2/36",P_value:.3202,Adjusted_P_value:.4129,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.7847,Combined_Score:2.0324,Genes:"APP;TNFRSF21",id:37},{Term:"Spermatogenesis",Overlap:"5/135",P_value:.4333,Adjusted_P_value:.5444,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1666,Combined_Score:.9757,Genes:"CCNB2;GFI1;CDK1;TALDO1;CDKN3",id:38},{Term:"Estrogen Response Early",Overlap:"7/200",P_value:.4574,Adjusted_P_value:.5603,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1,Combined_Score:.8605,Genes:"UGCG;SLC7A5;CLIC3;ARL3;PPIF;NBL1;AREG",id:39},{Term:"Myogenesis",Overlap:"6/200",P_value:.6192,Adjusted_P_value:.7224,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9365,Combined_Score:.4489,Genes:"APP;MEF2C;SYNGR2;TPM3;TPM2;LSP1",id:40},{Term:"KRAS Signaling Dn",Overlap:"6/200",P_value:.6192,Adjusted_P_value:.7224,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9365,Combined_Score:.4489,Genes:"TCL1A;IFNG;MX1;KRT5;PDCD1;IFI44L",id:41},{Term:"Notch Signaling",Overlap:"1/32",P_value:.6465,Adjusted_P_value:.7367,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9773,Combined_Score:.4263,Genes:"RBX1",id:42},{Term:"Wnt-beta Catenin Signaling",Overlap:"1/42",P_value:.7447,Adjusted_P_value:.8024,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7386,Combined_Score:.2177,Genes:"RBPJ",id:43},{Term:"Apical Surface",Overlap:"1/44",P_value:.7608,Adjusted_P_value:.8024,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7042,Combined_Score:.1925,Genes:"APP",id:44},{Term:"heme Metabolism",Overlap:"5/200",P_value:.7697,Adjusted_P_value:.8024,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7751,Combined_Score:.2029,Genes:"UCP2;TCEA1;ACP5;FBXO7;CTSB",id:45},{Term:"KRAS Signaling Up",Overlap:"5/200",P_value:.7697,Adjusted_P_value:.8024,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7751,Combined_Score:.2029,Genes:"FCER1G;TSPAN13;IRF8;LCP1;PSMB8",id:46},{Term:"Bile Acid Metabolism",Overlap:"2/112",P_value:.877,Adjusted_P_value:.8953,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.5495,Combined_Score:.0721,Genes:"PRDX5;IDH2",id:47},{Term:"UV Response Dn",Overlap:"2/144",P_value:.947,Adjusted_P_value:.947,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.4249,Combined_Score:.0231,Genes:"ANXA2;MT1E",id:48}]}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p:T,i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function Bl(e){let t,a,l,n;return t=new Pe({props:{$$slots:{default:[Ll]},$$scope:{ctx:e}}}),l=new Pe({props:{$$slots:{default:[jl]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const n={};1&a&&(n.$$scope={dirty:a,ctx:e}),t.$set(n);const _={};1&a&&(_.$$scope={dirty:a,ctx:e}),l.$set(_)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Ml(e){let t,a;return t=new pe({props:{$$slots:{content:[Bl],default:[hl]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function Dl(e){let t,a,l,n;return t=new ve({props:{label:"Plot"}}),l=new ve({props:{label:"Table"}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p:T,i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Gl(e){let t;return{c(){t=$("The enrichment analysis results of the top biological pathways associated with the input gene set. Each bar represents a pathway, with the length of the bar indicating the number of input genes overlapping with genes in that pathway. The color intensity of the bars reflects the statistical significance of the enrichment (p-value).")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function Fl(e){let t,a,l,n;return t=new ml({props:{$$slots:{default:[Gl]},$$scope:{ctx:e}}}),l=new Ia({props:{src:"../ClusterMarkers/samples.markers/Cluster/c17/Enrichr-KEGG_2021_Human.png",download:"../ClusterMarkers/samples.markers/Cluster/c17/Enrichr-KEGG_2021_Human.pdf",width:1e3,height:1e3}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Nl(e){let t,a;return t=new ul({props:{src:"../ClusterMarkers/samples.markers/Cluster/c17/Enrichr-KEGG_2021_Human.txt",data:[{Term:"Parkinson disease",Overlap:"47/249",P_value:3783e-26,Adjusted_P_value:9608e-24,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:7.53,Combined_Score:388.769,Genes:"COX7B;NDUFA13;NDUFB8;NDUFA11;NDUFB11;NDUFB4;NDUFB2;NDUFB1;UBE2L6;COX7A2;TXN;UQCR10;PARK7;COX6A1;COX5A;PSMA7;UBE2J1;PSMD8;PSMB6;TUBA1C;TUBA1B;PSMB3;UQCRFS1;COX8A;NDUFA7;NDUFA6;HSPA5;NDUFA4;TUBB;NDUFA1;NDUFC2;COX6C;TUBB4B;COX6B1;PSMA5;PSMA6;PSMA4;NDUFS8;PSMC3;PSMA2;UQCRQ;NDUFS6;PPIF;VDAC1;CALM3;SLC25A5;CALM2",id:0},{Term:"Prion disease",Overlap:"46/273",P_value:1.452e-20,Adjusted_P_value:1844e-21,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:6.5386,Combined_Score:298.6727,Genes:"COX7B;NDUFA13;NDUFB8;NDUFA11;NDUFB11;NDUFB4;NDUFB2;NDUFB1;COX7A2;UQCR10;COX6A1;COX5A;PSMA7;PSMD8;PSMB6;TUBA1C;TUBA1B;PSMB3;CCL5;RAC2;UQCRFS1;COX8A;NDUFA7;NDUFA6;HSPA5;NDUFA4;TUBB;NDUFA1;CYBB;NDUFC2;CYBA;COX6C;TUBB4B;COX6B1;PSMA5;PSMA6;PSMA4;NDUFS8;PSMC3;PSMA2;UQCRQ;NDUFS6;CSNK2B;PPIF;VDAC1;SLC25A5",id:1},{Term:"Huntington disease",Overlap:"47/306",P_value:2796e-22,Adjusted_P_value:2368e-20,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.8554,Combined_Score:250.1465,Genes:"COX7B;NDUFA13;NDUFB8;NDUFA11;NDUFB11;NDUFB4;DCTN3;NDUFB2;CLTA;NDUFB1;COX7A2;UQCR10;COX6A1;COX5A;PSMA7;PSMD8;PSMB6;TUBA1C;TUBA1B;PSMB3;ATG101;AP2S1;UQCRFS1;AP2M1;POLR2L;COX8A;NDUFA7;GPX1;NDUFA6;NDUFA4;TUBB;NDUFA1;NDUFC2;COX6C;TUBB4B;COX6B1;PSMA5;PSMA6;PSMA4;NDUFS8;PSMC3;PSMA2;UQCRQ;NDUFS6;PPIF;VDAC1;SLC25A5",id:2},{Term:"Amyotrophic lateral sclerosis",Overlap:"51/364",P_value:4818e-22,Adjusted_P_value:3059e-20,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.2783,Combined_Score:222.6238,Genes:"COX7B;NDUFA13;NDUFA11;COX6A1;ACTB;ACTG1;PSMD8;TUBA1C;TUBA1B;UQCRFS1;RAB8A;COX8A;GPX1;TUBB;NDUFC2;COX6B1;PSMA5;PSMA6;PSMA4;NDUFS8;PSMA2;NDUFS6;SRSF3;VDAC1;PFN1;NDUFB8;NDUFB11;NDUFB4;DCTN3;NDUFB2;NDUFB1;COX7A2;UQCR10;COX5A;GABARAP;PSMA7;PSMB6;PSMB3;ATG101;HNRNPA1;HNRNPA3;NDUFA7;NDUFA6;HSPA5;NDUFA4;NDUFA1;COX6C;TUBB4B;PSMC3;UQCRQ;HNRNPA2B1",id:3},{Term:"Alzheimer disease",Overlap:"48/369",P_value:1145e-19,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.8174,Combined_Score:176.8266,Genes:"APP;COX7B;NDUFA13;NDUFB8;CSF1;NDUFA11;NDUFB11;NDUFB4;NDUFB2;NDUFB1;COX7A2;UQCR10;COX6A1;COX5A;PSMA7;PSMD8;PSMB6;TUBA1C;TUBA1B;PSMB3;ATG101;UQCRFS1;COX8A;NDUFA7;NDUFA6;NDUFA4;TUBB;NDUFA1;CYBB;NDUFC2;COX6C;TUBB4B;COX6B1;PSMA5;PSMA6;PSMA4;NDUFS8;PSMC3;PSMA2;UQCRQ;NDUFS6;CSNK2B;PPIF;VDAC1;CALM3;SLC25A5;CALM2;GAPDH",id:4},{Term:"Pathways of neurodegeneration",Overlap:"55/475",P_value:1187e-19,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.2472,Combined_Score:155.7458,Genes:"APP;COX7B;NDUFA13;CSF1;NDUFA11;UBE2L6;PARK7;COX6A1;PSMD8;TUBA1C;TUBA1B;UQCRFS1;RAB8A;COX8A;GPX1;TUBB;NDUFC2;CYBB;COX6B1;PSMA5;PSMA6;PSMA4;NDUFS8;PSMA2;NDUFS6;CSNK2B;PPIF;VDAC1;SLC25A5;NDUFB8;NDUFB11;NDUFB4;DCTN3;NDUFB2;NDUFB1;COX7A2;UQCR10;COX5A;GABARAP;PSMA7;UBE2J1;PSMB6;PSMB3;ATG101;NDUFA7;NDUFA6;HSPA5;NDUFA4;NDUFA1;TUBB4B;COX6C;PSMC3;UQCRQ;CALM3;CALM2",id:5},{Term:"Oxidative phosphorylation",Overlap:"28/133",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:8.4041,Combined_Score:287.1957,Genes:"NDUFA13;COX7B;NDUFB8;NDUFA11;NDUFB11;NDUFB4;COX17;NDUFB2;NDUFB1;COX7A2;UQCR10;COX6A1;COX5A;UQCRFS1;COX8A;ATP6V0B;NDUFA7;NDUFA6;NDUFA4;NDUFA1;NDUFC2;COX6C;COX6B1;NDUFS8;PPA1;UQCRQ;NDUFS6;ATP6V0C",id:6},{Term:"Diabetic cardiomyopathy",Overlap:"33/203",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:6.1474,Combined_Score:196.8098,Genes:"COX7B;NDUFA13;NDUFB8;NDUFA11;NDUFB11;NDUFB4;NDUFB2;NDUFB1;COX7A2;UQCR10;COX6A1;COX5A;RAC2;UQCRFS1;CTSD;COX8A;NDUFA7;NDUFA6;NDUFA4;NDUFA1;CYBB;NDUFC2;CYBA;COX6C;PPP1CA;COX6B1;NDUFS8;UQCRQ;NDUFS6;PPIF;VDAC1;SLC25A5;GAPDH",id:7},{Term:"Proteasome",Overlap:"16/46",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:16.5487,Combined_Score:472.4861,Genes:"POMP;PSMA7;PSMB10;PSMB8;PSMB9;PSMB6;PSMD8;PSMA5;PSMA6;PSMA4;IFNG;PSMC3;PSMB3;PSMA2;PSME1;PSME2",id:8},{Term:"Antigen processing and presentation",Overlap:"19/78",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:10.0256,Combined_Score:264.7351,Genes:"PDIA3;CD74;HLA-DRB5;HSP90AB1;HLA-DRB4;HSPA5;TAP1;HLA-DMA;HLA-DMB;IFNG;CD8A;CANX;PSME1;PSME2;CALR;HLA-DRB3;HLA-DRB1;CTSB;HLA-DQB1",id:9},{Term:"Non-alcoholic fatty liver disease",Overlap:"24/155",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.7285,Combined_Score:129.5736,Genes:"COX8A;NDUFA13;COX7B;NDUFB8;NDUFA7;NDUFA6;NDUFA11;NDUFB11;NDUFA4;NDUFB4;NDUFB2;NDUFA1;NDUFC2;NDUFB1;COX7A2;UQCR10;COX6A1;COX6C;COX5A;COX6B1;NDUFS8;UQCRQ;NDUFS6;UQCRFS1",id:10},{Term:"Phagosome",Overlap:"23/152",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.5665,Combined_Score:118.3886,Genes:"HLA-DRB5;ATP6V0B;HLA-DRB4;TUBB;CYBB;TAP1;CYBA;TUBB4B;CORO1A;ACTB;ACTG1;TUBA1C;TUBA1B;HLA-DMA;HLA-DMB;SEC61G;CANX;CALR;SEC61B;HLA-DRB3;ATP6V0C;HLA-DRB1;HLA-DQB1",id:11},{Term:"Thermogenesis",Overlap:"28/232",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.3034,Combined_Score:86.9343,Genes:"COX7B;NDUFA13;NDUFB8;SMARCB1;NDUFA11;NDUFB11;NDUFB4;COX17;NDUFB2;NDUFB1;COX7A2;UQCR10;COX6A1;COX5A;ACTB;ACTG1;UQCRFS1;COX8A;NDUFA7;NDUFA6;NDUFA4;NDUFA1;NDUFC2;COX6C;COX6B1;NDUFS8;UQCRQ;NDUFS6",id:12},{Term:"Cell cycle",Overlap:"20/124",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.9827,Combined_Score:118.5911,Genes:"PCNA;MCM7;PLK1;RBX1;ANAPC11;CDC20;CCNA2;MAD2L2;CCNB2;CCNB1;TFDP1;PTTG1;RAD21;CDK1;MCM3;MCM5;MCM6;BUB3;YWHAH;MAD2L1",id:13},{Term:"Salmonella infection",Overlap:"28/249",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.9689,Combined_Score:73.8628,Genes:"ARF1;HSP90AB1;ARPC1B;DCTN3;ARPC5L;BRK1;TXN;ACTB;MYL12A;MYL12B;ACTG1;HSP90B1;PYCARD;TUBA1C;TUBA1B;S100A10;ACTR3;ANXA2;TUBB;ARPC5;DYNLL1;TUBB4B;RHOA;ARPC2;ARPC3;PFN1;GAPDH;ARF6",id:14},{Term:"Protein export",Overlap:"9/23",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:19.7418,Combined_Score:352.1096,Genes:"SPCS3;SPCS2;SPCS1;HSPA5;SEC61G;SEC61B;SRP14;SRP9;SEC11C",id:15},{Term:"Protein processing in endoplasmic reticulum",Overlap:"22/171",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.5975,Combined_Score:79.7254,Genes:"PDIA3;TRAM1;HSP90AB1;RPN2;HSPA5;SSR4;DERL3;SSR3;RRBP1;PDIA6;PDIA4;RBX1;HSP90B1;UBE2J1;HERPUD1;LMAN1;SEC61G;CANX;ERP29;CALR;SEC61B;P4HB",id:16},{Term:"DNA replication",Overlap:"10/36",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:11.8228,Combined_Score:188.0237,Genes:"RNASEH2C;FEN1;RNASEH2B;PCNA;RNASEH2A;MCM7;RPA3;MCM3;MCM5;MCM6",id:17},{Term:"Allograft rejection",Overlap:"10/38",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:10.9772,Combined_Score:168.3978,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;HLA-DRB4;IFNG;PRF1;GZMB;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:18},{Term:"Asthma",Overlap:"9/31",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:12.5578,Combined_Score:186.5986,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;FCER1G;HLA-DRB4;FCER1A;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:19},{Term:"Epstein-Barr virus infection",Overlap:"22/202",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.7996,Combined_Score:54.6248,Genes:"PDIA3;ENTPD1;HLA-DRB5;HLA-DRB4;SYK;TAP1;ISG15;RBPJ;RUNX3;CD3D;PSMD8;CCNA2;HLA-DMA;HLA-DMB;PSMC3;BLNK;IRF7;CALR;VIM;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:20},{Term:"Graft-versus-host disease",Overlap:"10/42",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:9.603,Combined_Score:137.5212,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;HLA-DRB4;IFNG;PRF1;GZMB;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:21},{Term:"Cardiac muscle contraction",Overlap:"14/87",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.9185,Combined_Score:84.4978,Genes:"COX8A;COX7B;TPM4;TPM3;TPM2;COX7A2;UQCR10;COX6A1;COX6C;COX5A;COX6B1;UQCRQ;FXYD2;UQCRFS1",id:22},{Term:"Type I diabetes mellitus",Overlap:"10/43",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:9.3116,Combined_Score:131.1506,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;HLA-DRB4;IFNG;PRF1;GZMB;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:23},{Term:"Pyrimidine metabolism",Overlap:"11/56",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:7.5186,Combined_Score:101.8519,Genes:"ENTPD1;DTYMK;DUT;RRM1;RRM2;NME2;TK1;TYMS;DCK;DCTPP1;TYMP",id:24},{Term:"Fluid shear stress and atherosclerosis",Overlap:"17/139",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.31,Combined_Score:56.0928,Genes:"MEF2C;HSP90AB1;GSTO1;GSTP1;MGST2;CYBA;TXN;ACTB;RHOA;ACTG1;HSP90B1;IFNG;SUMO3;SUMO2;RAC2;CALM3;CALM2",id:25},{Term:"Viral myocarditis",Overlap:"11/60",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:6.9034,Combined_Score:88.5635,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;HLA-DRB4;PRF1;RAC2;HLA-DRB3;ACTB;HLA-DRB1;ACTG1;HLA-DQB1",id:26},{Term:"Tuberculosis",Overlap:"19/180",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.6546,Combined_Score:44.3987,Genes:"CD74;HLA-DRB5;ATP6V0B;HLA-DRB4;FCER1G;SYK;LSP1;CORO1A;RHOA;HLA-DMA;HLA-DMB;IFNG;CALM3;HLA-DRB3;CALM2;CTSD;ATP6V0C;HLA-DRB1;HLA-DQB1",id:27},{Term:"Pathogenic Escherichia coli infection",Overlap:"20/197",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.5019,Combined_Score:42.4326,Genes:"ACTR3;ARF1;ARPC1B;TUBB;ARPC5L;BRK1;ARPC5;TUBB4B;ACTB;RHOA;ACTG1;PYCARD;TUBA1C;TUBA1B;ARPC2;ARPC3;NCL;TMBIM6;GAPDH;ARF6",id:28},{Term:"Bacterial invasion of epithelial cells",Overlap:"12/77",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.6816,Combined_Score:68.6864,Genes:"ACTR3;MAD2L2;ARPC2;ARPC3;ARPC1B;ARPC5L;CLTA;ARPC5;ACTB;RHOA;ACTG1;CD2AP",id:29},{Term:"Human T-cell leukemia virus 1 infection",Overlap:"21/219",P_value:0,Adjusted_P_value:1e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.2887,Combined_Score:38.6576,Genes:"RANBP1;HLA-DRB5;HLA-DRB4;CD3D;ANAPC11;CDC20;CCNA2;CCNB2;HLA-DMA;HLA-DMB;PTTG1;CANX;VDAC1;CALR;BUB3;HLA-DRB3;SLC25A5;HLA-DRB1;RAN;HLA-DQB1;MAD2L1",id:30},{Term:"Rheumatoid arthritis",Overlap:"13/93",P_value:0,Adjusted_P_value:1e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.0051,Combined_Score:58.772,Genes:"ATP6V0B;HLA-DRB5;HLA-DRB4;CSF1;HLA-DMA;HLA-DMB;IFNG;CCL5;ACP5;HLA-DRB3;ATP6V0C;HLA-DRB1;HLA-DQB1",id:31},{Term:"Hematopoietic cell lineage",Overlap:"13/99",P_value:0,Adjusted_P_value:1e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.6544,Combined_Score:51.4273,Genes:"HLA-DRB5;HLA-DRB4;CSF1;CD3D;HLA-DMA;HLA-DMB;CD8A;IL3RA;CD7;CD9;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:32},{Term:"Retrograde endocannabinoid signaling",Overlap:"16/148",P_value:0,Adjusted_P_value:2e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.7412,Combined_Score:40.2159,Genes:"NDUFA13;NDUFB8;NDUFA7;NDUFA6;NDUFA11;NDUFB11;NDUFA4;NDUFB4;NDUFB2;NDUFA1;NDUFC2;NDUFB1;NDUFS8;GNG5;NDUFS6;GNB2",id:33},{Term:"Spliceosome",Overlap:"16/150",P_value:0,Adjusted_P_value:2e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.685,Combined_Score:38.9943,Genes:"HNRNPA3;LSM5;LSM4;LSM2;LSM7;HNRNPK;SNRPD1;PCBP1;SNRPG;SRSF2;SRSF3;SNRPE;SNRPD3;SNRPF;HNRNPA1;SNRPB",id:34},{Term:"Influenza A",Overlap:"17/172",P_value:0,Adjusted_P_value:3e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.3866,Combined_Score:34.427,Genes:"HLA-DRB5;HLA-DRB4;MX1;ACTB;ACTG1;PYCARD;HLA-DMA;HLA-DMB;IFNG;CCL5;IRF7;VDAC1;KPNA2;HLA-DRB3;SLC25A5;HLA-DRB1;HLA-DQB1",id:35},{Term:"Autoimmune thyroid disease",Overlap:"9/53",P_value:0,Adjusted_P_value:3e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:6.2718,Combined_Score:63.3047,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;HLA-DRB4;PRF1;GZMB;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:36},{Term:"Lysosome",Overlap:"14/128",P_value:1e-4,Adjusted_P_value:4e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.7819,Combined_Score:36.7041,Genes:"CD164;CD63;ATP6V0B;CLTA;NPC2;PPT1;ACP5;CTSH;MAN2B1;CD68;CTSD;ATP6V0C;CTSC;CTSB",id:37},{Term:"Oocyte meiosis",Overlap:"14/129",P_value:1e-4,Adjusted_P_value:4e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.7488,Combined_Score:36.0619,Genes:"PLK1;RBX1;ANAPC11;PPP1CA;CDC20;MAD2L2;CCNB2;CCNB1;PTTG1;CDK1;CALM3;CALM2;YWHAH;MAD2L1",id:38},{Term:"Glutathione metabolism",Overlap:"9/57",P_value:1e-4,Adjusted_P_value:5e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.7479,Combined_Score:54.6003,Genes:"RRM1;GPX1;RRM2;GSTO1;GSTP1;IDH2;MGST2;LAP3;PRDX6",id:39},{Term:"Tight junction",Overlap:"16/169",P_value:1e-4,Adjusted_P_value:7e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.2242,Combined_Score:29.4565,Genes:"ACTR3;PCNA;ARPC1B;ARPC5L;ARPC5;ACTB;MYL12A;RHOA;ACTG1;MYL12B;TUBA1C;MYL6;TUBA1B;ARPC2;ARPC3;RAB8A",id:40},{Term:"Shigellosis",Overlap:"20/246",P_value:1e-4,Adjusted_P_value:8e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.7356,Combined_Score:24.4123,Genes:"ACTR3;ARF1;CBX3;ARPC1B;ARPC5L;ARPC5;ACTB;GABARAP;MYL12A;RHOA;MYL12B;ACTG1;RBX1;PYCARD;ARPC2;ARPC3;CCL5;VDAC1;PFN1;ARF6",id:41},{Term:"Leishmaniasis",Overlap:"10/77",P_value:2e-4,Adjusted_P_value:.001,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.5782,Combined_Score:39.8794,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;HLA-DRB4;IFNG;CYBB;CYBA;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:42},{Term:"Th1 and Th2 cell differentiation",Overlap:"11/92",P_value:2e-4,Adjusted_P_value:.001,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.1692,Combined_Score:36.2596,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;HLA-DRB4;IFNG;RBPJ;HLA-DRB3;CD3D;RUNX3;HLA-DRB1;HLA-DQB1",id:43},{Term:"Vibrio cholerae infection",Overlap:"8/50",P_value:2e-4,Adjusted_P_value:.001,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.8317,Combined_Score:50.6002,Genes:"ARF1;ATP6V0B;SEC61G;SEC61B;ATP6V0C;ACTB;ACTG1;PDIA4",id:44},{Term:"Spinocerebellar ataxia",Overlap:"14/143",P_value:2e-4,Adjusted_P_value:.0011,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.3395,Combined_Score:28.4246,Genes:"RBPJ;PSMA7;PSMD8;PSMA5;PSMB6;PSMA6;PSMA4;PSMC3;ATG101;PSMA2;PSMB3;PPIF;VDAC1;SLC25A5",id:45},{Term:"Regulation of actin cytoskeleton",Overlap:"18/218",P_value:2e-4,Adjusted_P_value:.0013,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.777,Combined_Score:23.207,Genes:"ACTR3;ARPC1B;ARPC5L;BRK1;ARPC5;ITGAE;ACTB;MYL12A;RHOA;MYL12B;PPP1CA;ACTG1;ARPC2;ARPC3;TMSB4X;CFL1;RAC2;PFN1",id:46},{Term:"Yersinia infection",Overlap:"13/137",P_value:5e-4,Adjusted_P_value:.0024,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.2217,Combined_Score:24.7902,Genes:"ACTR3;ARPC1B;ARPC5L;ARPC5;ACTB;RHOA;ACTG1;PYCARD;ARPC2;CD8A;ARPC3;RAC2;ARF6",id:47},{Term:"Th17 cell differentiation",Overlap:"11/107",P_value:6e-4,Adjusted_P_value:.0032,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.515,Combined_Score:25.9256,Genes:"HLA-DRB5;HLA-DMA;HLA-DRB4;HSP90AB1;HLA-DMB;IFNG;IRF4;HLA-DRB3;CD3D;HLA-DRB1;HLA-DQB1",id:48},{Term:"Apoptosis",Overlap:"13/142",P_value:6e-4,Adjusted_P_value:.0033,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.096,Combined_Score:22.7625,Genes:"PRF1;GZMB;ACTB;ACTG1;LMNB1;TUBA1C;TUBA1B;IL3RA;CTSH;BIRC5;CTSD;CTSC;CTSB",id:49},{Term:"Intestinal immune network for IgA production",Overlap:"7/48",P_value:8e-4,Adjusted_P_value:.0039,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.2192,Combined_Score:37.3814,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;HLA-DRB4;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:50},{Term:"Inflammatory bowel disease",Overlap:"8/65",P_value:.0011,Adjusted_P_value:.0051,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.2937,Combined_Score:29.4336,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;HLA-DRB4;IFNG;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:51},{Term:"Fc gamma R-mediated phagocytosis",Overlap:"10/97",P_value:.0011,Adjusted_P_value:.0051,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.5221,Combined_Score:24.094,Genes:"ACTR3;ARPC2;SYK;ARPC3;ARPC1B;CFL1;ARPC5L;RAC2;ARPC5;ARF6",id:52},{Term:"Glycolysis / Gluconeogenesis",Overlap:"8/67",P_value:.0013,Adjusted_P_value:.0061,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.1477,Combined_Score:27.6006,Genes:"LDHA;TPI1;PKM;PGAM1;PGK1;ENO1;ALDOA;GAPDH",id:53},{Term:"Human immunodeficiency virus 1 infection",Overlap:"16/212",P_value:.0013,Adjusted_P_value:.0061,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.5112,Combined_Score:16.6129,Genes:"PDIA3;APOBEC3G;TAP1;CD3D;RBX1;BST2;CCNB2;CCNB1;GNG5;GNB2;CFL1;RAC2;CDK1;CALM3;CALR;CALM2",id:54},{Term:"Progesterone-mediated oocyte maturation",Overlap:"10/100",P_value:.0014,Adjusted_P_value:.0061,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.4042,Combined_Score:22.4894,Genes:"CCNA2;MAD2L2;CCNB2;CCNB1;HSP90AB1;PLK1;CDK1;KIF22;ANAPC11;MAD2L1",id:55},{Term:"Drug metabolism",Overlap:"10/108",P_value:.0024,Adjusted_P_value:.0107,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.125,Combined_Score:18.8368,Genes:"DUT;RRM1;RRM2;GSTO1;GSTP1;NME2;MGST2;HPRT1;TK1;TYMP",id:56},{Term:"Endocytosis",Overlap:"17/252",P_value:.0032,Adjusted_P_value:.0139,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.2244,Combined_Score:12.8008,Genes:"ACTR3;ARF1;ARPC1B;ARPC5L;CLTA;ARPC5;RHOA;SNX3;ARPC2;CAPZB;ARPC3;AP2S1;RAB8A;ARF5;AP2M1;ARF6;CHMP5",id:57},{Term:"Systemic lupus erythematosus",Overlap:"11/135",P_value:.0041,Adjusted_P_value:.0175,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.7174,Combined_Score:14.9561,Genes:"HLA-DRB5;HLA-DMA;HLA-DRB4;HLA-DMB;IFNG;SNRPD1;SNRPD3;HLA-DRB3;HLA-DRB1;SNRPB;HLA-DQB1",id:58},{Term:"Cellular senescence",Overlap:"12/156",P_value:.0044,Adjusted_P_value:.0186,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.5541,Combined_Score:13.8572,Genes:"CCNA2;CCNB2;CCNB1;RBBP4;MAPKAPK2;CDK1;VDAC1;CALM3;MYBL2;SLC25A5;CALM2;PPP1CA",id:59},{Term:"Platelet activation",Overlap:"10/124",P_value:.0065,Adjusted_P_value:.0269,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.6842,Combined_Score:13.5296,Genes:"VAMP8;FCER1G;SYK;ACTB;MYL12A;RHOA;ACTG1;MYL12B;FERMT3;PPP1CA",id:60},{Term:"Primary immunodeficiency",Overlap:"5/38",P_value:.0069,Adjusted_P_value:.0282,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.6191,Combined_Score:23.0002,Genes:"CD8A;BLNK;TAP1;CD3D;ADA",id:61},{Term:"Toxoplasmosis",Overlap:"9/112",P_value:.0098,Adjusted_P_value:.0393,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.671,Combined_Score:12.366,Genes:"HLA-DRB5;HLA-DMA;HLA-DRB4;HLA-DMB;IFNG;PPIF;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:62},{Term:"Leukocyte transendothelial migration",Overlap:"9/114",P_value:.0109,Adjusted_P_value:.0432,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.6199,Combined_Score:11.8418,Genes:"RAC2;CYBB;CYBA;ACTB;MYL12A;CD99;RHOA;ACTG1;MYL12B",id:63},{Term:"C-type lectin receptor signaling pathway",Overlap:"8/104",P_value:.0183,Adjusted_P_value:.0717,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.5442,Combined_Score:10.1741,Genes:"PYCARD;FCER1G;SYK;MAPKAPK2;CALM3;LSP1;CALM2;RHOA",id:64},{Term:"Cell adhesion molecules",Overlap:"10/148",P_value:.0206,Adjusted_P_value:.0788,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.2146,Combined_Score:8.5949,Genes:"HLA-DRB5;HLA-DMA;HLA-DRB4;HLA-DMB;CD8A;PDCD1;HLA-DRB3;CD99;HLA-DRB1;HLA-DQB1",id:65},{Term:"Osteoclast differentiation",Overlap:"9/127",P_value:.0208,Adjusted_P_value:.0788,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.3297,Combined_Score:9.025,Genes:"TYROBP;CSF1;SYK;IFNG;BLNK;ACP5;CYBA;LILRB4;LILRA4",id:66},{Term:"Purine metabolism",Overlap:"9/129",P_value:.0227,Adjusted_P_value:.0849,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.2906,Combined_Score:8.6673,Genes:"ENTPD1;RRM1;PKM;RRM2;NME2;HPRT1;NUDT5;DCK;ADA",id:67},{Term:"HIF-1 signaling pathway",Overlap:"8/109",P_value:.0236,Adjusted_P_value:.0869,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.4177,Combined_Score:9.0581,Genes:"LDHA;IFNG;PGK1;CYBB;ENO1;ALDOA;GAPDH;RBX1",id:68},{Term:"One carbon pool by folate",Overlap:"3/20",P_value:.0247,Adjusted_P_value:.0896,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.3674,Combined_Score:19.8674,Genes:"DHFR;MTHFD2;TYMS",id:69},{Term:"p53 signaling pathway",Overlap:"6/73",P_value:.0291,Adjusted_P_value:.1041,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.7296,Combined_Score:9.6552,Genes:"CCNB2;CCNB1;RRM2;CD82;SIVA1;CDK1",id:70},{Term:"Viral carcinogenesis",Overlap:"12/203",P_value:.0303,Adjusted_P_value:.107,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.9209,Combined_Score:6.7142,Genes:"CDC20;CCNA2;RANBP1;HNRNPK;PKM;SYK;MAPKAPK2;CDK1;IRF7;RBPJ;RHOA;YWHAH",id:71},{Term:"NOD-like receptor signaling pathway",Overlap:"11/181",P_value:.0312,Adjusted_P_value:.1087,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.9773,Combined_Score:6.8535,Genes:"PYCARD;HSP90AB1;CCL5;IRF7;CYBB;CYBA;VDAC1;TXN;GABARAP;RHOA;CTSB",id:72},{Term:"Necroptosis",Overlap:"10/159",P_value:.0318,Adjusted_P_value:.1091,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.0499,Combined_Score:7.0676,Genes:"PYCARD;HSP90AB1;IFNG;CYBB;VDAC1;HMGB1;SLC25A5;GLUL;PPIA;CHMP5",id:73},{Term:"Staphylococcus aureus infection",Overlap:"7/95",P_value:.0325,Adjusted_P_value:.1091,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.4258,Combined_Score:8.3139,Genes:"HLA-DMA;HLA-DRB5;HLA-DMB;HLA-DRB4;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:74},{Term:"Thyroid hormone synthesis",Overlap:"6/75",P_value:.0327,Adjusted_P_value:.1091,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.6502,Combined_Score:9.0687,Genes:"GPX1;HSPA5;FXYD2;CANX;PDIA4;HSP90B1",id:75},{Term:"Pertussis",Overlap:"6/76",P_value:.0345,Adjusted_P_value:.1139,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.6122,Combined_Score:8.7924,Genes:"PYCARD;CFL1;IRF8;CALM3;CALM2;RHOA",id:76},{Term:"cGMP-PKG signaling pathway",Overlap:"10/167",P_value:.0423,Adjusted_P_value:.1376,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.9446,Combined_Score:6.1529,Genes:"RGS2;MEF2C;FXYD2;PPIF;VDAC1;CALM3;SLC25A5;CALM2;RHOA;PPP1CA",id:77},{Term:"Oxytocin signaling pathway",Overlap:"9/154",P_value:.0592,Adjusted_P_value:.1903,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.8932,Combined_Score:5.3521,Genes:"RGS2;MEF2C;MYL6;CALM3;CALM2;ACTB;RHOA;ACTG1;PPP1CA",id:78},{Term:"Fc epsilon RI signaling pathway",Overlap:"5/68",P_value:.0661,Adjusted_P_value:.2098,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.4158,Combined_Score:6.5629,Genes:"FCER1G;SYK;ALOX5AP;RAC2;FCER1A",id:79},{Term:"Central carbon metabolism in cancer",Overlap:"5/70",P_value:.073,Adjusted_P_value:.2288,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.3412,Combined_Score:6.129,Genes:"SLC7A5;LDHA;PKM;PGAM1;IDH2",id:80},{Term:"Adherens junction",Overlap:"5/71",P_value:.0765,Adjusted_P_value:.2371,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.3056,Combined_Score:5.9256,Genes:"CSNK2B;RAC2;ACTB;RHOA;ACTG1",id:81},{Term:"Ubiquitin mediated proteolysis",Overlap:"8/140",P_value:.0803,Adjusted_P_value:.2458,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.8469,Combined_Score:4.6572,Genes:"CDC20;UBE2C;UBE2L6;UBE2A;SAE1;RBX1;ANAPC11;UBE2J1",id:82},{Term:"Neutrophil extracellular trap formation",Overlap:"10/189",P_value:.0822,Adjusted_P_value:.2487,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.7037,Combined_Score:4.2559,Genes:"SYK;PPIF;RAC2;CYBB;CYBA;VDAC1;HMGB1;SLC25A5;ACTB;ACTG1",id:83},{Term:"Lipid and atherosclerosis",Overlap:"11/215",P_value:.085,Adjusted_P_value:.2539,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.6449,Combined_Score:4.0554,Genes:"PYCARD;HSP90AB1;HSPA5;CCL5;IRF7;CYBB;CALM3;CYBA;CALM2;RHOA;HSP90B1",id:84},{Term:"Base excision repair",Overlap:"3/33",P_value:.0874,Adjusted_P_value:.2581,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.0395,Combined_Score:7.4082,Genes:"FEN1;PCNA;HMGB1",id:85},{Term:"Endocrine and other factor-regulated calcium reabsorption",Overlap:"4/53",P_value:.0887,Adjusted_P_value:.2591,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.4827,Combined_Score:6.0129,Genes:"FXYD2;AP2S1;CLTA;AP2M1",id:86},{Term:"Synaptic vesicle cycle",Overlap:"5/78",P_value:.1041,Adjusted_P_value:.3004,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.0837,Combined_Score:4.7145,Genes:"ATP6V0B;AP2S1;CLTA;ATP6V0C;AP2M1",id:87},{Term:"RNA degradation",Overlap:"5/79",P_value:.1084,Adjusted_P_value:.3093,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.0555,Combined_Score:4.5675,Genes:"LSM7;ENO1;LSM5;LSM4;LSM2",id:88},{Term:"B cell receptor signaling pathway",Overlap:"5/81",P_value:.1172,Adjusted_P_value:.3308,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.0012,Combined_Score:4.29,Genes:"SYK;BLNK;RAC2;LILRB4;LILRA4",id:89},{Term:"Natural killer cell mediated cytotoxicity",Overlap:"7/131",P_value:.1268,Adjusted_P_value:.3533,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.7183,Combined_Score:3.5483,Genes:"TYROBP;FCER1G;SYK;IFNG;PRF1;RAC2;GZMB",id:90},{Term:"Glucagon signaling pathway",Overlap:"6/107",P_value:.128,Adjusted_P_value:.3533,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.8075,Combined_Score:3.7161,Genes:"LDHA;PPP4C;PKM;PGAM1;CALM3;CALM2",id:91},{Term:"Cytosolic DNA-sensing pathway",Overlap:"4/63",P_value:.142,Adjusted_P_value:.3847,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.0608,Combined_Score:4.0231,Genes:"PYCARD;CCL5;IRF7;POLR2L",id:92},{Term:"Ferroptosis",Overlap:"3/41",P_value:.1424,Adjusted_P_value:.3847,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.3986,Combined_Score:4.6756,Genes:"PCBP1;CYBB;SLC3A2",id:93},{Term:"Autophagy",Overlap:"7/137",P_value:.1496,Adjusted_P_value:.3986,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.6385,Combined_Score:3.113,Genes:"VAMP8;ATG101;HMGB1;CTSD;GABARAP;RAB8A;CTSB",id:94},{Term:"Gap junction",Overlap:"5/88",P_value:.1507,Adjusted_P_value:.3986,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.8317,Combined_Score:3.467,Genes:"TUBA1C;TUBA1B;TUBB;CDK1;TUBB4B",id:95},{Term:"Hypertrophic cardiomyopathy",Overlap:"5/90",P_value:.1609,Adjusted_P_value:.4213,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.7885,Combined_Score:3.2677,Genes:"TPM4;TPM3;TPM2;ACTB;ACTG1",id:96},{Term:"Kaposi sarcoma-associated herpesvirus infection",Overlap:"9/193",P_value:.1662,Adjusted_P_value:.4269,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.4889,Combined_Score:2.6724,Genes:"CLEC2B;SYK;GNG5;GNB2;MAPKAPK2;IRF7;CALM3;CALM2;GABARAP",id:97},{Term:"Mismatch repair",Overlap:"2/23",P_value:.1664,Adjusted_P_value:.4269,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.8915,Combined_Score:5.186,Genes:"PCNA;RPA3",id:98},{Term:"TGF-beta signaling pathway",Overlap:"5/94",P_value:.1821,Adjusted_P_value:.4609,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.7077,Combined_Score:2.9084,Genes:"TFDP1;IFNG;NBL1;RHOA;RBX1",id:99},{Term:"Human cytomegalovirus infection",Overlap:"10/225",P_value:.1849,Adjusted_P_value:.4609,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.4158,Combined_Score:2.39,Genes:"PDIA3;GNG5;GNB2;CCL5;RAC2;TAP1;CALM3;CALR;CALM2;RHOA",id:100},{Term:"Epithelial cell signaling in Helicobacter pylori infection",Overlap:"4/70",P_value:.1851,Adjusted_P_value:.4609,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.8416,Combined_Score:3.1066,Genes:"ATP6V0B;CCL5;CSK;ATP6V0C",id:101},{Term:"Nucleotide excision repair",Overlap:"3/47",P_value:.1893,Adjusted_P_value:.4669,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.0709,Combined_Score:3.4464,Genes:"PCNA;RPA3;RBX1",id:102},{Term:"Dilated cardiomyopathy",Overlap:"5/96",P_value:.1931,Adjusted_P_value:.4716,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.67,Combined_Score:2.7463,Genes:"TPM4;TPM3;TPM2;ACTB;ACTG1",id:103},{Term:"Circadian entrainment",Overlap:"5/97",P_value:.1987,Adjusted_P_value:.4807,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.6518,Combined_Score:2.6693,Genes:"RASD1;GNG5;GNB2;CALM3;CALM2",id:104},{Term:"Adrenergic signaling in cardiomyocytes",Overlap:"7/150",P_value:.2047,Adjusted_P_value:.4906,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.4885,Combined_Score:2.3609,Genes:"TPM4;TPM3;TPM2;FXYD2;CALM3;CALM2;PPP1CA",id:105},{Term:"Phototransduction",Overlap:"2/28",P_value:.2248,Adjusted_P_value:.5283,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.3349,Combined_Score:3.4845,Genes:"CALM3;CALM2",id:106},{Term:"Gastric acid secretion",Overlap:"4/76",P_value:.225,Adjusted_P_value:.5283,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.6876,Combined_Score:2.5176,Genes:"CALM3;CALM2;ACTB;ACTG1",id:107},{Term:"Chagas disease",Overlap:"5/102",P_value:.2274,Adjusted_P_value:.5283,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.5662,Combined_Score:2.3198,Genes:"GNA15;IFNG;CCL5;CALR;CD3D",id:108},{Term:"Riboflavin metabolism",Overlap:"1/8",P_value:.2288,Adjusted_P_value:.5283,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.3336,Combined_Score:6.3916,Genes:"ACP5",id:109},{Term:"T cell receptor signaling pathway",Overlap:"5/104",P_value:.2392,Adjusted_P_value:.5473,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.5344,Combined_Score:2.195,Genes:"IFNG;CD8A;PDCD1;CD3D;RHOA",id:110},{Term:"Herpes simplex virus 1 infection",Overlap:"19/498",P_value:.2453,Adjusted_P_value:.5492,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.208,Combined_Score:1.6977,Genes:"PDIA3;CD74;HLA-DRB5;HLA-DRB4;SYK;TAP1;PPP1CA;BST2;HLA-DMA;HLA-DMB;IFNG;CCL5;SRSF2;IRF7;SRSF3;CALR;HLA-DRB3;HLA-DRB1;HLA-DQB1",id:111},{Term:"Dopaminergic synapse",Overlap:"6/132",P_value:.2471,Adjusted_P_value:.5492,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.447,Combined_Score:2.0226,Genes:"GNG5;GNB2;CALM3;COMT;CALM2;PPP1CA",id:112},{Term:"Citrate cycle (TCA cycle)",Overlap:"2/30",P_value:.2487,Adjusted_P_value:.5492,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.1679,Combined_Score:3.0168,Genes:"IDH2;IDH3A",id:113},{Term:"Pentose phosphate pathway",Overlap:"2/30",P_value:.2487,Adjusted_P_value:.5492,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.1679,Combined_Score:3.0168,Genes:"TALDO1;ALDOA",id:114},{Term:"Vascular smooth muscle contraction",Overlap:"6/133",P_value:.2525,Adjusted_P_value:.5528,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.4355,Combined_Score:1.9759,Genes:"PPP1R14A;MYL6;CALM3;CALM2;RHOA;PPP1CA",id:115},{Term:"Apelin signaling pathway",Overlap:"6/137",P_value:.2741,Adjusted_P_value:.5951,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.3914,Combined_Score:1.8007,Genes:"MEF2C;GNG5;GNB2;CALM3;CALM2;GABARAP",id:116},{Term:"Fructose and mannose metabolism",Overlap:"2/33",P_value:.2845,Adjusted_P_value:.6073,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.9578,Combined_Score:2.4608,Genes:"TPI1;ALDOA",id:117},{Term:"SNARE interactions in vesicular transport",Overlap:"2/33",P_value:.2845,Adjusted_P_value:.6073,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.9578,Combined_Score:2.4608,Genes:"VAMP8;STX11",id:118},{Term:"Colorectal cancer",Overlap:"4/86",P_value:.2954,Adjusted_P_value:.6252,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.481,Combined_Score:1.8061,Genes:"RAC2;BIRC5;AREG;RHOA",id:119},{Term:"Mineral absorption",Overlap:"3/60",P_value:.3004,Adjusted_P_value:.6307,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.5975,Combined_Score:1.921,Genes:"MT2A;FXYD2;MT1E",id:120},{Term:"Focal adhesion",Overlap:"8/201",P_value:.314,Adjusted_P_value:.6495,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2592,Combined_Score:1.4585,Genes:"RAC2;ZYX;ACTB;MYL12A;RHOA;MYL12B;PPP1CA;ACTG1",id:121},{Term:"GABAergic synapse",Overlap:"4/89",P_value:.3171,Adjusted_P_value:.6495,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.4285,Combined_Score:1.6408,Genes:"GNG5;GNB2;GLUL;GABARAP",id:122},{Term:"PD-L1 expression and PD-1 checkpoint pathway in cancer",Overlap:"4/89",P_value:.3171,Adjusted_P_value:.6495,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.4285,Combined_Score:1.6408,Genes:"IFNG;CSNK2B;PDCD1;CD3D",id:123},{Term:"Tyrosine metabolism",Overlap:"2/36",P_value:.3202,Adjusted_P_value:.6507,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.7847,Combined_Score:2.0324,Genes:"MIF;COMT",id:124},{Term:"Sphingolipid signaling pathway",Overlap:"5/119",P_value:.3318,Adjusted_P_value:.6688,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.3315,Combined_Score:1.4691,Genes:"FCER1G;RAC2;FCER1A;CTSD;RHOA",id:125},{Term:"Phospholipase D signaling pathway",Overlap:"6/148",P_value:.3355,Adjusted_P_value:.6711,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2829,Combined_Score:1.4009,Genes:"ARF1;FCER1G;SYK;FCER1A;RHOA;ARF6",id:126},{Term:"Pathways in cancer",Overlap:"19/531",P_value:.339,Adjusted_P_value:.6727,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1282,Combined_Score:1.2203,Genes:"HSP90AB1;GSTO1;TPM3;GSTP1;MGST2;RHOA;RBX1;HSP90B1;CKS1B;CCNA2;IFNG;GNG5;GNB2;IL3RA;CKS2;RAC2;BIRC5;CALM3;CALM2",id:127},{Term:"Non-homologous end-joining",Overlap:"1/13",P_value:.3444,Adjusted_P_value:.6764,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.5273,Combined_Score:2.6937,Genes:"FEN1",id:128},{Term:"Salivary secretion",Overlap:"4/93",P_value:.3462,Adjusted_P_value:.6764,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.364,Combined_Score:1.4469,Genes:"CST3;FXYD2;CALM3;CALM2",id:129},{Term:"Rap1 signaling pathway",Overlap:"8/210",P_value:.3571,Adjusted_P_value:.6924,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2025,Combined_Score:1.2382,Genes:"CSF1;RAC2;CALM3;PFN1;CALM2;ACTB;RHOA;ACTG1",id:130},{Term:"Long-term potentiation",Overlap:"3/67",P_value:.362,Adjusted_P_value:.6965,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.4222,Combined_Score:1.4452,Genes:"CALM3;CALM2;PPP1CA",id:131},{Term:"Amphetamine addiction",Overlap:"3/69",P_value:.3795,Adjusted_P_value:.7193,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.379,Combined_Score:1.3363,Genes:"CALM3;CALM2;PPP1CA",id:132},{Term:"Renin secretion",Overlap:"3/69",P_value:.3795,Adjusted_P_value:.7193,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.379,Combined_Score:1.3363,Genes:"CALM3;CALM2;CTSB",id:133},{Term:"Viral protein interaction with cytokine and cytokine receptor",Overlap:"4/100",P_value:.3972,Adjusted_P_value:.7472,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2641,Combined_Score:1.1673,Genes:"CSF1;CCL5;CXCR3;CXCL13",id:134},{Term:"Pancreatic secretion",Overlap:"4/102",P_value:.4116,Adjusted_P_value:.7674,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2382,Combined_Score:1.0991,Genes:"SCT;FXYD2;RAB8A;RHOA",id:135},{Term:"Chemokine signaling pathway",Overlap:"7/192",P_value:.4157,Adjusted_P_value:.7674,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1481,Combined_Score:1.0078,Genes:"GNG5;GNB2;CCL5;CXCR3;RAC2;CXCL13;RHOA",id:136},{Term:"Hippo signaling pathway",Overlap:"6/163",P_value:.4212,Adjusted_P_value:.7674,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1594,Combined_Score:1.0026,Genes:"BIRC5;AREG;ACTB;ACTG1;YWHAH;PPP1CA",id:137},{Term:"Nitrogen metabolism",Overlap:"1/17",P_value:.4243,Adjusted_P_value:.7674,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.8951,Combined_Score:1.6245,Genes:"GLUL",id:138},{Term:"Phenylalanine metabolism",Overlap:"1/17",P_value:.4243,Adjusted_P_value:.7674,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.8951,Combined_Score:1.6245,Genes:"MIF",id:139},{Term:"NF-kappa B signaling pathway",Overlap:"4/104",P_value:.426,Adjusted_P_value:.7674,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2133,Combined_Score:1.0353,Genes:"EDARADD;SYK;CSNK2B;BLNK",id:140},{Term:"Metabolism of xenobiotics by cytochrome P450",Overlap:"3/76",P_value:.4396,Adjusted_P_value:.781,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2463,Combined_Score:1.0242,Genes:"GSTO1;GSTP1;MGST2",id:141},{Term:"Other glycan degradation",Overlap:"1/18",P_value:.4427,Adjusted_P_value:.781,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.7835,Combined_Score:1.4531,Genes:"MAN2B1",id:142},{Term:"Pyruvate metabolism",Overlap:"2/47",P_value:.4456,Adjusted_P_value:.781,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.3477,Combined_Score:1.0893,Genes:"LDHA;PKM",id:143},{Term:"Estrogen signaling pathway",Overlap:"5/137",P_value:.4458,Adjusted_P_value:.781,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1489,Combined_Score:.9281,Genes:"HSP90AB1;CALM3;CALM2;CTSD;HSP90B1",id:144},{Term:"Hepatocellular carcinoma",Overlap:"6/168",P_value:.4495,Adjusted_P_value:.7821,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1233,Combined_Score:.8982,Genes:"SMARCB1;GSTO1;GSTP1;MGST2;ACTB;ACTG1",id:145},{Term:"Ribosome biogenesis in eukaryotes",Overlap:"4/108",P_value:.4545,Adjusted_P_value:.7853,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1664,Combined_Score:.9197,Genes:"CSNK2B;NHP2;SNORD3A;RAN",id:146},{Term:"Ras signaling pathway",Overlap:"8/232",P_value:.4638,Adjusted_P_value:.7932,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0831,Combined_Score:.8322,Genes:"CSF1;GNG5;GNB2;RAC2;CALM3;CALM2;RHOA;ARF6",id:147},{Term:"Sphingolipid metabolism",Overlap:"2/49",P_value:.4671,Adjusted_P_value:.7932,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2902,Combined_Score:.9822,Genes:"SMPD3;UGCG",id:148},{Term:"Cytokine-cytokine receptor interaction",Overlap:"10/295",P_value:.4697,Adjusted_P_value:.7932,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0641,Combined_Score:.8041,Genes:"CSF1;IFNG;CD70;CXCR3;TNFRSF9;TNFRSF18;CCL5;IL3RA;CXCL13;TNFRSF21",id:149},{Term:"Cholesterol metabolism",Overlap:"2/50",P_value:.4776,Adjusted_P_value:.7932,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2633,Combined_Score:.9335,Genes:"NPC2;VDAC1",id:150},{Term:"Cysteine and methionine metabolism",Overlap:"2/50",P_value:.4776,Adjusted_P_value:.7932,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2633,Combined_Score:.9335,Genes:"DNMT1;LDHA",id:151},{Term:"Steroid biosynthesis",Overlap:"1/20",P_value:.4778,Adjusted_P_value:.7932,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.5956,Combined_Score:1.1784,Genes:"LBR",id:152},{Term:"Peroxisome",Overlap:"3/82",P_value:.4894,Adjusted_P_value:.8072,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1513,Combined_Score:.8228,Genes:"PRDX5;PRDX1;IDH2",id:153},{Term:"Arginine biosynthesis",Overlap:"1/22",P_value:.5107,Adjusted_P_value:.8369,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.4435,Combined_Score:.97,Genes:"GLUL",id:154},{Term:"Fanconi anemia pathway",Overlap:"2/54",P_value:.5185,Adjusted_P_value:.8442,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1659,Combined_Score:.7658,Genes:"RPA3;UBE2T",id:155},{Term:"Proximal tubule bicarbonate reclamation",Overlap:"1/23",P_value:.5263,Adjusted_P_value:.8515,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.3778,Combined_Score:.8843,Genes:"FXYD2",id:156},{Term:"Neurotrophin signaling pathway",Overlap:"4/119",P_value:.5301,Adjusted_P_value:.8521,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0542,Combined_Score:.6691,Genes:"MAPKAPK2;CALM3;CALM2;RHOA",id:157},{Term:"Legionellosis",Overlap:"2/57",P_value:.5477,Adjusted_P_value:.8749,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1021,Combined_Score:.6635,Genes:"PYCARD;ARF1",id:158},{Term:"Notch signaling pathway",Overlap:"2/59",P_value:.5665,Adjusted_P_value:.8937,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0633,Combined_Score:.6043,Genes:"PTCRA;RBPJ",id:159},{Term:"VEGF signaling pathway",Overlap:"2/59",P_value:.5665,Adjusted_P_value:.8937,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0633,Combined_Score:.6043,Genes:"MAPKAPK2;RAC2",id:160},{Term:"Folate biosynthesis",Overlap:"1/26",P_value:.5704,Adjusted_P_value:.8943,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.2123,Combined_Score:.6807,Genes:"DHFR",id:161},{Term:"IL-17 signaling pathway",Overlap:"3/94",P_value:.5817,Adjusted_P_value:.8946,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9989,Combined_Score:.5412,Genes:"HSP90AB1;IFNG;HSP90B1",id:162},{Term:"Collecting duct acid secretion",Overlap:"1/27",P_value:.5841,Adjusted_P_value:.8946,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1656,Combined_Score:.6267,Genes:"ATP6V0C",id:163},{Term:"Fatty acid elongation",Overlap:"1/27",P_value:.5841,Adjusted_P_value:.8946,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1656,Combined_Score:.6267,Genes:"PPT1",id:164},{Term:"Arachidonic acid metabolism",Overlap:"2/61",P_value:.5847,Adjusted_P_value:.8946,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0272,Combined_Score:.5513,Genes:"GPX1;PTGDS",id:165},{Term:"MAPK signaling pathway",Overlap:"9/294",P_value:.599,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9562,Combined_Score:.49,Genes:"RPS6KA4;DUSP4;DUSP5;MEF2C;CSF1;STMN1;MAPKAPK2;RAC2;AREG",id:166},{Term:"Prostate cancer",Overlap:"3/97",P_value:.6031,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9668,Combined_Score:.4889,Genes:"HSP90AB1;GSTP1;HSP90B1",id:167},{Term:"FoxO signaling pathway",Overlap:"4/131",P_value:.6062,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.954,Combined_Score:.4775,Genes:"CCNB2;CCNB1;PLK1;GABARAP",id:168},{Term:"Inflammatory mediator regulation of TRP channels",Overlap:"3/98",P_value:.6101,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9566,Combined_Score:.4728,Genes:"CALM3;CALM2;PPP1CA",id:169},{Term:"Wnt signaling pathway",Overlap:"5/166",P_value:.6154,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9405,Combined_Score:.4565,Genes:"SERPINF1;CSNK2B;RAC2;RHOA;RBX1",id:170},{Term:"Glyoxylate and dicarboxylate metabolism",Overlap:"1/30",P_value:.6228,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0449,Combined_Score:.4948,Genes:"GLUL",id:171},{Term:"Circadian rhythm",Overlap:"1/31",P_value:.6348,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.01,Combined_Score:.4589,Genes:"RBX1",id:172},{Term:"RNA polymerase",Overlap:"1/31",P_value:.6348,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.01,Combined_Score:.4589,Genes:"POLR2L",id:173},{Term:"Amoebiasis",Overlap:"3/102",P_value:.6371,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9178,Combined_Score:.4138,Genes:"GNA15;IFNG;PRDX1",id:174},{Term:"Insulin signaling pathway",Overlap:"4/137",P_value:.6413,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9107,Combined_Score:.4046,Genes:"PRKAR1A;CALM3;CALM2;PPP1CA",id:175},{Term:"Mitophagy",Overlap:"2/68",P_value:.6439,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9179,Combined_Score:.4041,Genes:"CSNK2B;GABARAP",id:176},{Term:"Chemical carcinogenesis",Overlap:"7/239",P_value:.645,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9132,Combined_Score:.4004,Genes:"HSP90AB1;GSTO1;GSTP1;MGST2;BIRC5;KPNA2;HSP90B1",id:177},{Term:"Measles",Overlap:"4/139",P_value:.6525,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8971,Combined_Score:.383,Genes:"CSNK2B;MX1;IRF7;CD3D",id:178},{Term:"RIG-I-like receptor signaling pathway",Overlap:"2/70",P_value:.6595,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8908,Combined_Score:.3708,Genes:"IRF7;ISG15",id:179},{Term:"Parathyroid hormone synthesis, secretion and action",Overlap:"3/106",P_value:.6628,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8819,Combined_Score:.3628,Genes:"MEF2C;RUNX2;RHOA",id:180},{Term:"Pentose and glucuronate interconversions",Overlap:"1/34",P_value:.6688,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.918,Combined_Score:.3693,Genes:"DCXR",id:181},{Term:"Propanoate metabolism",Overlap:"1/34",P_value:.6688,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.918,Combined_Score:.3693,Genes:"LDHA",id:182},{Term:"PPAR signaling pathway",Overlap:"2/74",P_value:.6892,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8411,Combined_Score:.3131,Genes:"FABP5;DBI",id:183},{Term:"Mucin type O-glycan biosynthesis",Overlap:"1/36",P_value:.6896,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8655,Combined_Score:.3216,Genes:"GALNT2",id:184},{Term:"cAMP signaling pathway",Overlap:"6/216",P_value:.6919,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8644,Combined_Score:.3184,Genes:"FXYD2;RAC2;CALM3;CALM2;RHOA;PPP1CA",id:185},{Term:"Axon guidance",Overlap:"5/182",P_value:.6948,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8548,Combined_Score:.3113,Genes:"CFL1;RAC2;MYL12A;RHOA;MYL12B",id:186},{Term:"Glioma",Overlap:"2/75",P_value:.6962,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8296,Combined_Score:.3004,Genes:"CALM3;CALM2",id:187},{Term:"PI3K-Akt signaling pathway",Overlap:"10/354",P_value:.6981,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8789,Combined_Score:.3158,Genes:"TCL1A;HSP90AB1;SYK;CSF1;GNG5;GNB2;IL3RA;AREG;HSP90B1;YWHAH",id:188},{Term:"TNF signaling pathway",Overlap:"3/112",P_value:.6988,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8331,Combined_Score:.2986,Genes:"RPS6KA4;CSF1;CCL5",id:189},{Term:"African trypanosomiasis",Overlap:"1/37",P_value:.6996,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8414,Combined_Score:.3006,Genes:"IFNG",id:190},{Term:"Alanine, aspartate and glutamate metabolism",Overlap:"1/37",P_value:.6996,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8414,Combined_Score:.3006,Genes:"GLUL",id:191},{Term:"Aldosterone-regulated sodium reabsorption",Overlap:"1/37",P_value:.6996,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8414,Combined_Score:.3006,Genes:"FXYD2",id:192},{Term:"Thyroid cancer",Overlap:"1/37",P_value:.6996,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8414,Combined_Score:.3006,Genes:"TPM3",id:193},{Term:"Serotonergic synapse",Overlap:"3/113",P_value:.7045,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8255,Combined_Score:.2892,Genes:"APP;GNG5;GNB2",id:194},{Term:"Arrhythmogenic right ventricular cardiomyopathy",Overlap:"2/77",P_value:.71,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8074,Combined_Score:.2765,Genes:"ACTB;ACTG1",id:195},{Term:"Glutamatergic synapse",Overlap:"3/114",P_value:.7101,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.818,Combined_Score:.28,Genes:"GNG5;GNB2;GLUL",id:196},{Term:"Alcoholism",Overlap:"5/186",P_value:.7127,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8357,Combined_Score:.283,Genes:"GNG5;GNB2;CALM3;CALM2;PPP1CA",id:197},{Term:"RNA transport",Overlap:"5/186",P_value:.7127,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8357,Combined_Score:.283,Genes:"SUMO3;SUMO2;SNORD3A;EIF2S2;RAN",id:198},{Term:"Various types of N-glycan biosynthesis",Overlap:"1/39",P_value:.7185,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.797,Combined_Score:.2635,Genes:"RPN2",id:199},{Term:"Glycine, serine and threonine metabolism",Overlap:"1/40",P_value:.7275,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7765,Combined_Score:.247,Genes:"PGAM1",id:200},{Term:"mTOR signaling pathway",Overlap:"4/154",P_value:.7292,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8068,Combined_Score:.2548,Genes:"SLC7A5;SLC3A2;RHOA;LAMTOR5",id:201},{Term:"Bladder cancer",Overlap:"1/41",P_value:.7362,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7571,Combined_Score:.2318,Genes:"TYMP",id:202},{Term:"Homologous recombination",Overlap:"1/41",P_value:.7362,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7571,Combined_Score:.2318,Genes:"RPA3",id:203},{Term:"Transcriptional misregulation in cancer",Overlap:"5/192",P_value:.7383,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8086,Combined_Score:.2454,Genes:"PTCRA;CCNA2;MEF2C;GZMB;RUNX2",id:204},{Term:"Hepatitis C",Overlap:"4/157",P_value:.7429,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7908,Combined_Score:.235,Genes:"IFNG;MX1;IRF7;YWHAH",id:205},{Term:"Thyroid hormone signaling pathway",Overlap:"3/121",P_value:.7472,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7692,Combined_Score:.2242,Genes:"FXYD2;ACTB;ACTG1",id:206},{Term:"Vasopressin-regulated water reabsorption",Overlap:"1/44",P_value:.7608,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7042,Combined_Score:.1925,Genes:"DYNLL1",id:207},{Term:"Hepatitis B",Overlap:"4/162",P_value:.7646,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7656,Combined_Score:.2055,Genes:"CCNA2;PCNA;IRF7;BIRC5",id:208},{Term:"ABC transporters",Overlap:"1/45",P_value:.7684,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6881,Combined_Score:.1813,Genes:"TAP1",id:209},{Term:"Glycosphingolipid biosynthesis",Overlap:"1/45",P_value:.7684,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6881,Combined_Score:.1813,Genes:"FUT7",id:210},{Term:"Type II diabetes mellitus",Overlap:"1/46",P_value:.7758,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6728,Combined_Score:.1708,Genes:"PKM",id:211},{Term:"Calcium signaling pathway",Overlap:"6/240",P_value:.7825,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7748,Combined_Score:.19,Genes:"GNA15;PPIF;VDAC1;CALM3;SLC25A5;CALM2",id:212},{Term:"Carbohydrate digestion and absorption",Overlap:"1/47",P_value:.783,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6581,Combined_Score:.161,Genes:"FXYD2",id:213},{Term:"Other types of O-glycan biosynthesis",Overlap:"1/47",P_value:.783,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6581,Combined_Score:.161,Genes:"GALNT2",id:214},{Term:"Relaxin signaling pathway",Overlap:"3/129",P_value:.7848,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7201,Combined_Score:.1745,Genes:"GNA15;GNG5;GNB2",id:215},{Term:"Bile secretion",Overlap:"2/90",P_value:.7871,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6876,Combined_Score:.1647,Genes:"SCT;FXYD2",id:216},{Term:"Proteoglycans in cancer",Overlap:"5/205",P_value:.7878,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7556,Combined_Score:.1802,Genes:"CD63;ACTB;RHOA;PPP1CA;ACTG1",id:217},{Term:"Morphine addiction",Overlap:"2/91",P_value:.7922,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6799,Combined_Score:.1584,Genes:"GNG5;GNB2",id:218},{Term:"Ether lipid metabolism",Overlap:"1/49",P_value:.7967,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6306,Combined_Score:.1433,Genes:"PLD4",id:219},{Term:"Small cell lung cancer",Overlap:"2/92",P_value:.7972,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6723,Combined_Score:.1524,Genes:"CKS2;CKS1B",id:220},{Term:"GnRH signaling pathway",Overlap:"2/93",P_value:.8021,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6649,Combined_Score:.1466,Genes:"CALM3;CALM2",id:221},{Term:"Arginine and proline metabolism",Overlap:"1/50",P_value:.8032,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6177,Combined_Score:.1354,Genes:"LAP3",id:222},{Term:"Malaria",Overlap:"1/50",P_value:.8032,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6177,Combined_Score:.1354,Genes:"IFNG",id:223},{Term:"N-Glycan biosynthesis",Overlap:"1/50",P_value:.8032,Adjusted_P_value:.9067,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6177,Combined_Score:.1354,Genes:"RPN2",id:224},{Term:"Phosphatidylinositol signaling system",Overlap:"2/97",P_value:.8206,Adjusted_P_value:.9191,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6367,Combined_Score:.1259,Genes:"CALM3;CALM2",id:225},{Term:"Glycosaminoglycan biosynthesis",Overlap:"1/53",P_value:.8215,Adjusted_P_value:.9191,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.582,Combined_Score:.1144,Genes:"CHST12",id:226},{Term:"Aldosterone synthesis and secretion",Overlap:"2/98",P_value:.825,Adjusted_P_value:.9191,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6301,Combined_Score:.1212,Genes:"CALM3;CALM2",id:227},{Term:"Melanogenesis",Overlap:"2/101",P_value:.8376,Adjusted_P_value:.929,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6109,Combined_Score:.1083,Genes:"CALM3;CALM2",id:228},{Term:"Protein digestion and absorption",Overlap:"2/103",P_value:.8455,Adjusted_P_value:.9337,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.5987,Combined_Score:.1005,Genes:"FXYD2;SLC3A2",id:229},{Term:"Toll-like receptor signaling pathway",Overlap:"2/104",P_value:.8494,Adjusted_P_value:.9339,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.5928,Combined_Score:.0968,Genes:"CCL5;IRF7",id:230},{Term:"Steroid hormone biosynthesis",Overlap:"1/61",P_value:.8625,Adjusted_P_value:.9442,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.5042,Combined_Score:.0746,Genes:"COMT",id:231},{Term:"Cholinergic synapse",Overlap:"2/113",P_value:.8801,Adjusted_P_value:.9595,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.5445,Combined_Score:.0695,Genes:"GNG5;GNB2",id:232},{Term:"Acute myeloid leukemia",Overlap:"1/67",P_value:.8869,Adjusted_P_value:.9627,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.4582,Combined_Score:.055,Genes:"CCNA2",id:233},{Term:"Renal cell carcinoma",Overlap:"1/69",P_value:.894,Adjusted_P_value:.9663,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.4447,Combined_Score:.0498,Genes:"RBX1",id:234},{Term:"AMPK signaling pathway",Overlap:"2/120",P_value:.8999,Adjusted_P_value:.9686,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.512,Combined_Score:.054,Genes:"CCNA2;RAB8A",id:235},{Term:"Inositol phosphate metabolism",Overlap:"1/73",P_value:.907,Adjusted_P_value:.9687,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.4199,Combined_Score:.041,Genes:"TPI1",id:236},{Term:"Human papillomavirus infection",Overlap:"7/331",P_value:.9077,Adjusted_P_value:.9687,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6508,Combined_Score:.063,Genes:"CCNA2;ATP6V0B;PKM;MX1;ISG15;RBPJ;ATP6V0C",id:237},{Term:"Pancreatic cancer",Overlap:"1/76",P_value:.9156,Adjusted_P_value:.9731,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.4031,Combined_Score:.0355,Genes:"RAC2",id:238},{Term:"MicroRNAs in cancer",Overlap:"6/310",P_value:.9338,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.5942,Combined_Score:.0407,Genes:"DNMT1;HNRNPK;STMN1;VIM;SOX4;RHOA",id:239},{Term:"ErbB signaling pathway",Overlap:"1/85",P_value:.9371,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.3597,Combined_Score:.0234,Genes:"AREG",id:240},{Term:"Insulin secretion",Overlap:"1/86",P_value:.9391,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.3554,Combined_Score:.0223,Genes:"FXYD2",id:241},{Term:"Coronavirus disease",Overlap:"4/232",P_value:.9413,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.5286,Combined_Score:.032,Genes:"SYK;MX1;CYBB;ISG15",id:242},{Term:"Choline metabolism in cancer",Overlap:"1/98",P_value:.9588,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.3113,Combined_Score:.0131,Genes:"RAC2",id:243},{Term:"Glycerophospholipid metabolism",Overlap:"1/98",P_value:.9588,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.3113,Combined_Score:.0131,Genes:"PLD4",id:244},{Term:"mRNA surveillance pathway",Overlap:"1/98",P_value:.9588,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.3113,Combined_Score:.0131,Genes:"PPP1CA",id:245},{Term:"AGE-RAGE signaling pathway in diabetic complications",Overlap:"1/100",P_value:.9614,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.305,Combined_Score:.012,Genes:"CYBB",id:246},{Term:"Ribosome",Overlap:"2/158",P_value:.9637,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.3865,Combined_Score:.0143,Genes:"MRPL20;MRPS6",id:247},{Term:"Longevity regulating pathway",Overlap:"1/102",P_value:.9639,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.2989,Combined_Score:.011,Genes:"ATG101",id:248},{Term:"JAK-STAT signaling pathway",Overlap:"2/162",P_value:.9675,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.3768,Combined_Score:.0124,Genes:"IFNG;IL3RA",id:249},{Term:"Insulin resistance",Overlap:"1/108",P_value:.9703,Adjusted_P_value:.9819,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.282,Combined_Score:.0085,Genes:"PPP1CA",id:250},{Term:"Cushing syndrome",Overlap:"1/155",P_value:.9936,Adjusted_P_value:.9994,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.1955,Combined_Score:.0013,Genes:"RASD1",id:251},{Term:"Neuroactive ligand-receptor interaction",Overlap:"4/341",P_value:.9954,Adjusted_P_value:.9994,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.3556,Combined_Score:.0016,Genes:"NPW;SCT;P2RY14;GZMA",id:252},{Term:"Olfactory transduction",Overlap:"3/440",P_value:.9999,Adjusted_P_value:.9999,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.2043,Combined_Score:0,Genes:"RGS2;CALM3;CALM2",id:253}]}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p:T,i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function yl(e){let t,a,l,n;return t=new Pe({props:{$$slots:{default:[Fl]},$$scope:{ctx:e}}}),l=new Pe({props:{$$slots:{default:[Nl]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const n={};1&a&&(n.$$scope={dirty:a,ctx:e}),t.$set(n);const _={};1&a&&(_.$$scope={dirty:a,ctx:e}),l.$set(_)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function xl(e){let t,a;return t=new pe({props:{$$slots:{content:[yl],default:[Dl]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function Ul(e){let t,a,l,n;return t=new De({props:{title:"MSigDB_Hallmark_2020",open:!0,$$slots:{default:[Ml]},$$scope:{ctx:e}}}),l=new De({props:{title:"KEGG_2021_Human",$$slots:{default:[xl]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const n={};1&a&&(n.$$scope={dirty:a,ctx:e}),t.$set(n);const _={};1&a&&(_.$$scope={dirty:a,ctx:e}),l.$set(_)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Hl(e){let t,a,l,n,_,d;return t=new ve({props:{label:"Significant Markers"}}),l=new ve({props:{label:"Volcano Plot"}}),_=new ve({props:{label:"Dot Plot"}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment),n=C(),m(_.$$.fragment)},m(e,r){A(t,e,r),s(e,a,r),A(l,e,r),s(e,n,r),A(_,e,r),d=!0},p:T,i(e){d||(r(t.$$.fragment,e),r(l.$$.fragment,e),r(_.$$.fragment,e),d=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),c(_.$$.fragment,e),d=!1},d(e){e&&(u(a),u(n)),g(t,e),g(l,e),g(_,e)}}}function wl(e){let t;return{c(){t=$("The markers are found using Seurat's FindMarkers function, and filtered by: p_val_adj < 0.05 & avg_log2FC > 0.1")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function Il(e){let t,a,l,n;return t=new ml({props:{$$slots:{default:[wl]},$$scope:{ctx:e}}}),l=new ul({props:{src:"../ClusterMarkers/samples.markers/Cluster/c18/markers.txt",data:[{p_val:0,avg_log2FC:1.2929,pct_1:.932,pct_2:.175,p_val_adj:0,cluster:"c18",gene:"GNLY",id:0},{p_val:6536e-88,avg_log2FC:.6606,pct_1:.992,pct_2:.855,p_val_adj:1.266e-80,cluster:"c18",gene:"RPS12",id:1},{p_val:6672e-86,avg_log2FC:.7337,pct_1:.972,pct_2:.75,p_val_adj:1292e-81,cluster:"c18",gene:"RPL34",id:2},{p_val:108e-80,avg_log2FC:.7088,pct_1:.967,pct_2:.778,p_val_adj:2092e-77,cluster:"c18",gene:"RPS3",id:3},{p_val:3644e-78,avg_log2FC:.6289,pct_1:.977,pct_2:.793,p_val_adj:7058e-74,cluster:"c18",gene:"RPL32",id:4},{p_val:5.25e-70,avg_log2FC:.5843,pct_1:.982,pct_2:.839,p_val_adj:1017e-68,cluster:"c18",gene:"RPL13",id:5},{p_val:8.973e-70,avg_log2FC:.5476,pct_1:.992,pct_2:.895,p_val_adj:1738e-68,cluster:"c18",gene:"RPL10",id:6},{p_val:2122e-69,avg_log2FC:.587,pct_1:.983,pct_2:.867,p_val_adj:4111e-65,cluster:"c18",gene:"EEF1A1",id:7},{p_val:4209e-68,avg_log2FC:.5826,pct_1:.982,pct_2:.825,p_val_adj:8153e-64,cluster:"c18",gene:"RPL30",id:8},{p_val:1613e-67,avg_log2FC:.6895,pct_1:.947,pct_2:.74,p_val_adj:3.125e-60,cluster:"c18",gene:"RPS18",id:9},{p_val:3093e-67,avg_log2FC:.7956,pct_1:.863,pct_2:.57,p_val_adj:5.991e-60,cluster:"c18",gene:"RPL3",id:10},{p_val:3583e-67,avg_log2FC:.6875,pct_1:.921,pct_2:.68,p_val_adj:6.941e-60,cluster:"c18",gene:"RPS23",id:11},{p_val:4903e-67,avg_log2FC:.4558,pct_1:.995,pct_2:.95,p_val_adj:9.497e-60,cluster:"c18",gene:"RPL41",id:12},{p_val:2177e-66,avg_log2FC:.4888,pct_1:.993,pct_2:.929,p_val_adj:4216e-62,cluster:"c18",gene:"RPS27",id:13},{p_val:2875e-65,avg_log2FC:.5661,pct_1:.982,pct_2:.833,p_val_adj:5569e-61,cluster:"c18",gene:"RPL39",id:14},{p_val:9042e-65,avg_log2FC:.72,pct_1:.894,pct_2:.668,p_val_adj:1752e-60,cluster:"c18",gene:"RPS14",id:15},{p_val:103e-63,avg_log2FC:.6237,pct_1:.947,pct_2:.761,p_val_adj:1995e-60,cluster:"c18",gene:"RPL19",id:16},{p_val:1079e-64,avg_log2FC:.5383,pct_1:.985,pct_2:.851,p_val_adj:2091e-60,cluster:"c18",gene:"RPS15A",id:17},{p_val:3.076e-60,avg_log2FC:.6323,pct_1:.927,pct_2:.726,p_val_adj:5958e-59,cluster:"c18",gene:"RPS4X",id:18},{p_val:5736e-62,avg_log2FC:.619,pct_1:.946,pct_2:.749,p_val_adj:1111e-57,cluster:"c18",gene:"RPS27A",id:19},{p_val:2946e-61,avg_log2FC:.6614,pct_1:.901,pct_2:.643,p_val_adj:5707e-57,cluster:"c18",gene:"RPS3A",id:20},{p_val:1357e-56,avg_log2FC:.6082,pct_1:.962,pct_2:.784,p_val_adj:2628e-52,cluster:"c18",gene:"RPS19",id:21},{p_val:1383e-55,avg_log2FC:.691,pct_1:.878,pct_2:.657,p_val_adj:2678e-51,cluster:"c18",gene:"RPL26",id:22},{p_val:3261e-55,avg_log2FC:.6203,pct_1:.891,pct_2:.682,p_val_adj:6318e-51,cluster:"c18",gene:"RPS21",id:23},{p_val:3232e-52,avg_log2FC:.5338,pct_1:.969,pct_2:.769,p_val_adj:6261e-48,cluster:"c18",gene:"RPL11",id:24},{p_val:4643e-51,avg_log2FC:1.0127,pct_1:.543,pct_2:.277,p_val_adj:8994e-47,cluster:"c18",gene:"IL7R",id:25},{p_val:528e-49,avg_log2FC:.5755,pct_1:.941,pct_2:.715,p_val_adj:1023e-45,cluster:"c18",gene:"RPL18A",id:26},{p_val:1287e-49,avg_log2FC:.6485,pct_1:.865,pct_2:.612,p_val_adj:2494e-45,cluster:"c18",gene:"TPT1",id:27},{p_val:2113e-48,avg_log2FC:.7242,pct_1:.785,pct_2:.523,p_val_adj:4093e-44,cluster:"c18",gene:"RPL6",id:28},{p_val:4567e-48,avg_log2FC:.6235,pct_1:.861,pct_2:.614,p_val_adj:8847e-44,cluster:"c18",gene:"RPS7",id:29},{p_val:5633e-47,avg_log2FC:.7194,pct_1:.762,pct_2:.481,p_val_adj:1091e-42,cluster:"c18",gene:"RPLP0",id:30},{p_val:1363e-46,avg_log2FC:.6539,pct_1:.827,pct_2:.588,p_val_adj:264e-41,cluster:"c18",gene:"RPL7A",id:31},{p_val:1732e-46,avg_log2FC:.6679,pct_1:.795,pct_2:.542,p_val_adj:3354e-42,cluster:"c18",gene:"RPL14",id:32},{p_val:1.411e-40,avg_log2FC:.6248,pct_1:.838,pct_2:.585,p_val_adj:2733e-39,cluster:"c18",gene:"RPL29",id:33},{p_val:262e-40,avg_log2FC:.65,pct_1:.766,pct_2:.512,p_val_adj:5074e-37,cluster:"c18",gene:"RPS6",id:34},{p_val:7962e-41,avg_log2FC:.5819,pct_1:.818,pct_2:.569,p_val_adj:1542e-36,cluster:"c18",gene:"RPS13",id:35},{p_val:152e-39,avg_log2FC:.5717,pct_1:.847,pct_2:.592,p_val_adj:2944e-36,cluster:"c18",gene:"RPS25",id:36},{p_val:5851e-40,avg_log2FC:.602,pct_1:.794,pct_2:.556,p_val_adj:1133e-35,cluster:"c18",gene:"RPL18",id:37},{p_val:5676e-39,avg_log2FC:.639,pct_1:.733,pct_2:.478,p_val_adj:11e-32,cluster:"c18",gene:"RPL5",id:38},{p_val:3544e-38,avg_log2FC:.5004,pct_1:.888,pct_2:.704,p_val_adj:6866e-34,cluster:"c18",gene:"RPS15",id:39},{p_val:4546e-37,avg_log2FC:.511,pct_1:.84,pct_2:.606,p_val_adj:8.806e-30,cluster:"c18",gene:"RPS8",id:40},{p_val:6836e-37,avg_log2FC:.552,pct_1:.848,pct_2:.617,p_val_adj:1324e-32,cluster:"c18",gene:"RPL36",id:41},{p_val:9245e-37,avg_log2FC:.765,pct_1:.589,pct_2:.356,p_val_adj:1791e-32,cluster:"c18",gene:"RPSA",id:42},{p_val:4891e-35,avg_log2FC:.5483,pct_1:.832,pct_2:.612,p_val_adj:9474e-31,cluster:"c18",gene:"RPL35A",id:43},{p_val:1408e-32,avg_log2FC:.5115,pct_1:.818,pct_2:.603,p_val_adj:2728e-28,cluster:"c18",gene:"RPL37",id:44},{p_val:4623e-32,avg_log2FC:.6395,pct_1:.649,pct_2:.426,p_val_adj:8954e-28,cluster:"c18",gene:"RPL23A",id:45},{p_val:1264e-31,avg_log2FC:.4964,pct_1:.81,pct_2:.61,p_val_adj:2449e-27,cluster:"c18",gene:"FAU",id:46},{p_val:1766e-31,avg_log2FC:.4896,pct_1:.838,pct_2:.638,p_val_adj:3421e-27,cluster:"c18",gene:"RPL12",id:47},{p_val:2177e-30,avg_log2FC:.5602,pct_1:.69,pct_2:.452,p_val_adj:4217e-26,cluster:"c18",gene:"RPS5",id:48},{p_val:5575e-30,avg_log2FC:.6121,pct_1:.642,pct_2:.429,p_val_adj:108e-24,cluster:"c18",gene:"RPL35",id:49},{p_val:1026e-29,avg_log2FC:.8581,pct_1:.417,pct_2:.233,p_val_adj:1987e-25,cluster:"c18",gene:"RPL36A",id:50},{p_val:2012e-29,avg_log2FC:.7155,pct_1:.543,pct_2:.342,p_val_adj:3897e-25,cluster:"c18",gene:"RPL10A",id:51},{p_val:1206e-27,avg_log2FC:.5345,pct_1:.706,pct_2:.494,p_val_adj:2.336e-20,cluster:"c18",gene:"RPLP2",id:52},{p_val:1261e-27,avg_log2FC:.483,pct_1:.79,pct_2:.621,p_val_adj:2.442e-20,cluster:"c18",gene:"RPS24",id:53},{p_val:1475e-27,avg_log2FC:.3225,pct_1:.97,pct_2:.873,p_val_adj:2.858e-20,cluster:"c18",gene:"RPL28",id:54},{p_val:5582e-26,avg_log2FC:.6082,pct_1:.604,pct_2:.414,p_val_adj:1081e-21,cluster:"c18",gene:"RPS9",id:55},{p_val:4128e-25,avg_log2FC:.4949,pct_1:.738,pct_2:.554,p_val_adj:7997e-21,cluster:"c18",gene:"RPL8",id:56},{p_val:6172e-25,avg_log2FC:.4862,pct_1:.756,pct_2:.576,p_val_adj:1196e-20,cluster:"c18",gene:"RPS29",id:57},{p_val:9.964e-20,avg_log2FC:.2867,pct_1:.975,pct_2:.887,p_val_adj:0,cluster:"c18",gene:"RPLP1",id:58},{p_val:1449e-22,avg_log2FC:.6117,pct_1:.475,pct_2:.3,p_val_adj:0,cluster:"c18",gene:"EEF1B2",id:59},{p_val:6662e-22,avg_log2FC:.4911,pct_1:.589,pct_2:.394,p_val_adj:0,cluster:"c18",gene:"RPL9",id:60},{p_val:3325e-21,avg_log2FC:.6398,pct_1:.611,pct_2:.485,p_val_adj:0,cluster:"c18",gene:"RPS26",id:61},{p_val:3596e-20,avg_log2FC:.5165,pct_1:.54,pct_2:.368,p_val_adj:0,cluster:"c18",gene:"NACA",id:62},{p_val:3351e-19,avg_log2FC:.5895,pct_1:.465,pct_2:.313,p_val_adj:0,cluster:"c18",gene:"RPS2",id:63},{p_val:0,avg_log2FC:.5438,pct_1:.493,pct_2:.338,p_val_adj:0,cluster:"c18",gene:"RPL22",id:64},{p_val:0,avg_log2FC:.568,pct_1:.472,pct_2:.324,p_val_adj:0,cluster:"c18",gene:"RPL37A",id:65},{p_val:0,avg_log2FC:1.5636,pct_1:.074,pct_2:.025,p_val_adj:0,cluster:"c18",gene:"RPS4Y1",id:66},{p_val:0,avg_log2FC:1.319,pct_1:.104,pct_2:.042,p_val_adj:0,cluster:"c18",gene:"LYAR",id:67},{p_val:0,avg_log2FC:1.094,pct_1:.256,pct_2:.153,p_val_adj:0,cluster:"c18",gene:"KLRB1",id:68},{p_val:0,avg_log2FC:.5123,pct_1:.505,pct_2:.36,p_val_adj:0,cluster:"c18",gene:"GNB2L1",id:69},{p_val:0,avg_log2FC:.4439,pct_1:.573,pct_2:.431,p_val_adj:0,cluster:"c18",gene:"RPL15",id:70},{p_val:0,avg_log2FC:.2963,pct_1:.635,pct_2:.478,p_val_adj:0,cluster:"c18",gene:"CYTB",id:71},{p_val:0,avg_log2FC:.1327,pct_1:.483,pct_2:.318,p_val_adj:0,cluster:"c18",gene:"NKG7",id:72},{p_val:0,avg_log2FC:.5497,pct_1:.386,pct_2:.263,p_val_adj:0,cluster:"c18",gene:"RPL38",id:73},{p_val:0,avg_log2FC:.2655,pct_1:.781,pct_2:.674,p_val_adj:0,cluster:"c18",gene:"ND3",id:74},{p_val:0,avg_log2FC:.5481,pct_1:.404,pct_2:.283,p_val_adj:0,cluster:"c18",gene:"RPL27",id:75},{p_val:0,avg_log2FC:.4097,pct_1:.545,pct_2:.409,p_val_adj:0,cluster:"c18",gene:"RPL24",id:76},{p_val:0,avg_log2FC:.2721,pct_1:.642,pct_2:.513,p_val_adj:0,cluster:"c18",gene:"ATP8",id:77},{p_val:0,avg_log2FC:.1526,pct_1:.916,pct_2:.824,p_val_adj:0,cluster:"c18",gene:"COX2",id:78},{p_val:0,avg_log2FC:.2509,pct_1:.723,pct_2:.608,p_val_adj:0,cluster:"c18",gene:"ATP6",id:79},{p_val:0,avg_log2FC:.5654,pct_1:.305,pct_2:.203,p_val_adj:0,cluster:"c18",gene:"EEF1G",id:80},{p_val:0,avg_log2FC:.8731,pct_1:.152,pct_2:.085,p_val_adj:1e-4,cluster:"c18",gene:"SNHG6",id:81},{p_val:0,avg_log2FC:.2224,pct_1:.616,pct_2:.495,p_val_adj:1e-4,cluster:"c18",gene:"ND4L",id:82},{p_val:0,avg_log2FC:1.5416,pct_1:.053,pct_2:.02,p_val_adj:1e-4,cluster:"c18",gene:"NCR3",id:83},{p_val:0,avg_log2FC:.4362,pct_1:.411,pct_2:.303,p_val_adj:2e-4,cluster:"c18",gene:"RPL21",id:84},{p_val:0,avg_log2FC:1.1129,pct_1:.089,pct_2:.043,p_val_adj:3e-4,cluster:"c18",gene:"NOSIP",id:85},{p_val:0,avg_log2FC:.4179,pct_1:.447,pct_2:.339,p_val_adj:4e-4,cluster:"c18",gene:"RPS16",id:86},{p_val:0,avg_log2FC:.4953,pct_1:.325,pct_2:.231,p_val_adj:9e-4,cluster:"c18",gene:"RPL36AL",id:87},{p_val:0,avg_log2FC:.4479,pct_1:.361,pct_2:.265,p_val_adj:.0021,cluster:"c18",gene:"PFDN5",id:88},{p_val:0,avg_log2FC:1.9648,pct_1:.023,pct_2:.006,p_val_adj:.0039,cluster:"c18",gene:"FXYD7",id:89},{p_val:0,avg_log2FC:.4214,pct_1:.332,pct_2:.242,p_val_adj:.0113,cluster:"c18",gene:"RPL4",id:90},{p_val:0,avg_log2FC:.5682,pct_1:.226,pct_2:.153,p_val_adj:.0116,cluster:"c18",gene:"LRRC75A-AS1",id:91},{p_val:0,avg_log2FC:.3918,pct_1:.38,pct_2:.287,p_val_adj:.0139,cluster:"c18",gene:"UBA52",id:92},{p_val:0,avg_log2FC:.3359,pct_1:.414,pct_2:.319,p_val_adj:.0329,cluster:"c18",gene:"EEF1D",id:93},{p_val:0,avg_log2FC:1.2389,pct_1:.05,pct_2:.021,p_val_adj:.0389,cluster:"c18",gene:"PTGER2",id:94}]}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function El(e){let t;return{c(){t=$("The volcano plot is generated using ggplot2. The significant markers are highlighted in blue and red, indicating up-regulated and down-regulated genes, respectively. The non-significant genes are shown in grey.")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function kl(e){let t,a,l,n;return t=new ml({props:{$$slots:{default:[El]},$$scope:{ctx:e}}}),l=new Ia({props:{src:"../ClusterMarkers/samples.markers/Cluster/c18/volcano.png",download:"../ClusterMarkers/samples.markers/Cluster/c18/volcano.pdf",width:900,height:1200}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Xl(e){let t;return{c(){t=$("The dot plot is generated using Seurat's DotPlot function. The top significant markers are used as input. The sizes of the dots are proportional to the percentage of cells expressing the gene in each group. The colors of the dots are proportional to the average expression of the gene in each group.")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function Vl(e){let t,a,l,n;return t=new ml({props:{$$slots:{default:[Xl]},$$scope:{ctx:e}}}),l=new Ia({props:{src:"../ClusterMarkers/samples.markers/Cluster/c18/dotplot.png",download:"../ClusterMarkers/samples.markers/Cluster/c18/dotplot.pdf",width:600,height:1200}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Kl(e){let t,a,l,n,_,d;return t=new Pe({props:{$$slots:{default:[Il]},$$scope:{ctx:e}}}),l=new Pe({props:{$$slots:{default:[kl]},$$scope:{ctx:e}}}),_=new Pe({props:{$$slots:{default:[Vl]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment),n=C(),m(_.$$.fragment)},m(e,r){A(t,e,r),s(e,a,r),A(l,e,r),s(e,n,r),A(_,e,r),d=!0},p(e,a){const n={};1&a&&(n.$$scope={dirty:a,ctx:e}),t.$set(n);const d={};1&a&&(d.$$scope={dirty:a,ctx:e}),l.$set(d);const s={};1&a&&(s.$$scope={dirty:a,ctx:e}),_.$set(s)},i(e){d||(r(t.$$.fragment,e),r(l.$$.fragment,e),r(_.$$.fragment,e),d=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),c(_.$$.fragment,e),d=!1},d(e){e&&(u(a),u(n)),g(t,e),g(l,e),g(_,e)}}}function Yl(e){let t;return{c(){t=$("The enrichment analysis is done using Enrichr. The significant markers are used as input.")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function zl(e){let t,a,l,n;return t=new ve({props:{label:"Plot"}}),l=new ve({props:{label:"Table"}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p:T,i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function ql(e){let t;return{c(){t=$("The enrichment analysis results of the top biological pathways associated with the input gene set. Each bar represents a pathway, with the length of the bar indicating the number of input genes overlapping with genes in that pathway. The color intensity of the bars reflects the statistical significance of the enrichment (p-value).")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function Ql(e){let t,a,l,n;return t=new ml({props:{$$slots:{default:[ql]},$$scope:{ctx:e}}}),l=new Ia({props:{src:"../ClusterMarkers/samples.markers/Cluster/c18/Enrichr-MSigDB_Hallmark_2020.png",download:"../ClusterMarkers/samples.markers/Cluster/c18/Enrichr-MSigDB_Hallmark_2020.pdf",width:1e3,height:1e3}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Wl(e){let t,a;return t=new ul({props:{src:"../ClusterMarkers/samples.markers/Cluster/c18/Enrichr-MSigDB_Hallmark_2020.txt",data:[{Term:"Myc Targets V1",Overlap:"11/200",P_value:0,Adjusted_P_value:0,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:13.6606,Combined_Score:268.7884,Genes:"EEF1B2;RPL34;RPS5;RPLP0;RPS6;RPL22;RPS3;RPL14;RPS2;RPL18;RPL6",id:0},{Term:"Allograft Rejection",Overlap:"5/200",P_value:.0027,Adjusted_P_value:.012,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.6154,Combined_Score:33.291,Genes:"RPS9;RPS19;RPS3A;RPL9;RPL39",id:1},{Term:"p53 Pathway",Overlap:"3/200",P_value:.07,Adjusted_P_value:.2099,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.2622,Combined_Score:8.6762,Genes:"RPL36;RPL18;RPS12",id:2},{Term:"Inflammatory Response",Overlap:"2/200",P_value:.2457,Adjusted_P_value:.5528,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.1404,Combined_Score:3.0044,Genes:"PTGER2;IL7R",id:3},{Term:"Unfolded Protein Response",Overlap:"1/113",P_value:.417,Adjusted_P_value:.616,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.88,Combined_Score:1.6445,Genes:"RPS14",id:4},{Term:"IL-2/STAT5 Signaling",Overlap:"1/199",P_value:.6141,Adjusted_P_value:.616,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0588,Combined_Score:.5162,Genes:"PTGER2",id:5},{Term:"KRAS Signaling Up",Overlap:"1/200",P_value:.616,Adjusted_P_value:.616,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0535,Combined_Score:.5105,Genes:"IL7R",id:6},{Term:"TNF-alpha Signaling via NF-kB",Overlap:"1/200",P_value:.616,Adjusted_P_value:.616,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0535,Combined_Score:.5105,Genes:"IL7R",id:7},{Term:"E2F Targets",Overlap:"1/200",P_value:.616,Adjusted_P_value:.616,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0535,Combined_Score:.5105,Genes:"LYAR",id:8}]}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p:T,i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function Zl(e){let t,a,l,n;return t=new Pe({props:{$$slots:{default:[Ql]},$$scope:{ctx:e}}}),l=new Pe({props:{$$slots:{default:[Wl]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const n={};1&a&&(n.$$scope={dirty:a,ctx:e}),t.$set(n);const _={};1&a&&(_.$$scope={dirty:a,ctx:e}),l.$set(_)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function Jl(e){let t,a;return t=new pe({props:{$$slots:{content:[Zl],default:[zl]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function en(e){let t,a,l,n;return t=new ve({props:{label:"Plot"}}),l=new ve({props:{label:"Table"}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p:T,i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function tn(e){let t;return{c(){t=$("The enrichment analysis results of the top biological pathways associated with the input gene set. Each bar represents a pathway, with the length of the bar indicating the number of input genes overlapping with genes in that pathway. The color intensity of the bars reflects the statistical significance of the enrichment (p-value).")},m(e,a){s(e,t,a)},d(e){e&&u(t)}}}function an(e){let t,a,l,n;return t=new ml({props:{$$slots:{default:[tn]},$$scope:{ctx:e}}}),l=new Ia({props:{src:"../ClusterMarkers/samples.markers/Cluster/c18/Enrichr-KEGG_2021_Human.png",download:"../ClusterMarkers/samples.markers/Cluster/c18/Enrichr-KEGG_2021_Human.pdf",width:1e3,height:1e3}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function ln(e){let t,a;return t=new ul({props:{src:"../ClusterMarkers/samples.markers/Cluster/c18/Enrichr-KEGG_2021_Human.txt",data:[{Term:"Ribosome",Overlap:"70/158",P_value:4555e-136,Adjusted_P_value:1959e-134,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:630.5409,Combined_Score:192143.252,Genes:"RPL4;RPL5;RPL30;RPL3;RPL32;RPL34;RPLP1;RPLP0;RPL10A;RPL8;RPL9;RPL6;RPS4X;RPS15;RPS14;RPL7A;RPS16;RPS19;RPL18A;RPS18;RPL36AL;RPL36;RPL35;RPLP2;RPL38;RPL37;RPL39;RPS13;RPS12;RPS9;RPS7;RPL21;RPS8;RPS5;RPS6;RPL22;RPS3A;RPSA;RPL37A;RPL24;RPL27;RPL26;RPL29;RPL28;UBA52;RPL10;RPL12;RPL11;RPL36A;RPS4Y1;RPS15A;RPS3;RPL14;RPL13;RPS2;RPL15;RPS27A;RPL18;RPL19;RPL41;RPL35A;RPL23A;RPS26;RPS25;RPS27;RPS29;FAU;RPS21;RPS24;RPS23",id:0},{Term:"Coronavirus disease",Overlap:"70/232",P_value:1419e-121,Adjusted_P_value:305e-119,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:341.237,Combined_Score:92596.4905,Genes:"RPL4;RPL5;RPL30;RPL3;RPL32;RPL34;RPLP1;RPLP0;RPL10A;RPL8;RPL9;RPL6;RPS4X;RPS15;RPS14;RPL7A;RPS16;RPS19;RPL18A;RPS18;RPL36AL;RPL36;RPL35;RPLP2;RPL38;RPL37;RPL39;RPS13;RPS12;RPS9;RPS7;RPL21;RPS8;RPS5;RPS6;RPL22;RPS3A;RPSA;RPL37A;RPL24;RPL27;RPL26;RPL29;RPL28;UBA52;RPL10;RPL12;RPL11;RPL36A;RPS4Y1;RPS15A;RPS3;RPL14;RPL13;RPS2;RPL15;RPS27A;RPL18;RPL19;RPL41;RPL35A;RPL23A;RPS26;RPS25;RPS27;RPS29;FAU;RPS21;RPS24;RPS23",id:1},{Term:"Parkinson disease",Overlap:"8/249",P_value:0,Adjusted_P_value:4e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:7.5028,Combined_Score:79.6068,Genes:"ATP8;ATP6;COX2;ND4L;CYTB;ND3;RPS27A;UBA52",id:2},{Term:"Oxidative phosphorylation",Overlap:"6/133",P_value:0,Adjusted_P_value:4e-4,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:10.4988,Combined_Score:105.973,Genes:"ATP8;ATP6;COX2;ND4L;CYTB;ND3",id:3},{Term:"Thermogenesis",Overlap:"7/232",P_value:1e-4,Adjusted_P_value:.001,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:6.9576,Combined_Score:62.8137,Genes:"ATP8;RPS6;ATP6;COX2;ND4L;CYTB;ND3",id:4},{Term:"Diabetic cardiomyopathy",Overlap:"6/203",P_value:4e-4,Adjusted_P_value:.003,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:6.7443,Combined_Score:52.4884,Genes:"ATP8;ATP6;COX2;ND4L;CYTB;ND3",id:5},{Term:"Pathways of neurodegeneration",Overlap:"8/475",P_value:.0019,Adjusted_P_value:.0104,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.8274,Combined_Score:23.9648,Genes:"ATP8;ATP6;COX2;ND4L;CYTB;ND3;RPS27A;UBA52",id:6},{Term:"Prion disease",Overlap:"6/273",P_value:.0019,Adjusted_P_value:.0104,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.9585,Combined_Score:30.9932,Genes:"ATP8;ATP6;COX2;ND4L;CYTB;ND3",id:7},{Term:"Huntington disease",Overlap:"6/306",P_value:.0034,Adjusted_P_value:.0162,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.4056,Combined_Score:25.043,Genes:"ATP8;ATP6;COX2;ND4L;CYTB;ND3",id:8},{Term:"Amyotrophic lateral sclerosis",Overlap:"6/364",P_value:.0078,Adjusted_P_value:.0325,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.6809,Combined_Score:17.8642,Genes:"ATP8;ATP6;COX2;ND4L;CYTB;ND3",id:9},{Term:"Alzheimer disease",Overlap:"6/369",P_value:.0083,Adjusted_P_value:.0325,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.6293,Combined_Score:17.3826,Genes:"ATP8;ATP6;COX2;ND4L;CYTB;ND3",id:10},{Term:"Legionellosis",Overlap:"2/57",P_value:.0301,Adjusted_P_value:.1078,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:7.7615,Combined_Score:27.1934,Genes:"EEF1A1;EEF1G",id:11},{Term:"Mitophagy",Overlap:"2/68",P_value:.0415,Adjusted_P_value:.1374,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:6.4643,Combined_Score:20.5647,Genes:"RPS27A;UBA52",id:12},{Term:"Cardiac muscle contraction",Overlap:"2/87",P_value:.0644,Adjusted_P_value:.1978,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.0145,Combined_Score:13.7529,Genes:"COX2;CYTB",id:13},{Term:"Ubiquitin mediated proteolysis",Overlap:"2/140",P_value:.143,Adjusted_P_value:.4011,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.0804,Combined_Score:5.9901,Genes:"RPS27A;UBA52",id:14},{Term:"Retrograde endocannabinoid signaling",Overlap:"2/148",P_value:.1562,Adjusted_P_value:.4011,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.9104,Combined_Score:5.4034,Genes:"ND4L;ND3",id:15},{Term:"Primary immunodeficiency",Overlap:"1/38",P_value:.1656,Adjusted_P_value:.4011,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:5.7125,Combined_Score:10.2705,Genes:"IL7R",id:16},{Term:"Non-alcoholic fatty liver disease",Overlap:"2/155",P_value:.1679,Adjusted_P_value:.4011,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.7763,Combined_Score:4.9539,Genes:"COX2;CYTB",id:17},{Term:"Malaria",Overlap:"1/50",P_value:.2121,Adjusted_P_value:.48,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:4.3109,Combined_Score:6.6853,Genes:"KLRB1",id:18},{Term:"Kaposi sarcoma-associated herpesvirus infection",Overlap:"2/193",P_value:.2334,Adjusted_P_value:.5018,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.2197,Combined_Score:3.2295,Genes:"RPS27A;UBA52",id:19},{Term:"Renin secretion",Overlap:"1/69",P_value:.2804,Adjusted_P_value:.5742,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:3.1034,Combined_Score:3.9459,Genes:"PTGER2",id:20},{Term:"Leishmaniasis",Overlap:"1/77",P_value:.3074,Adjusted_P_value:.6009,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.7756,Combined_Score:3.274,Genes:"EEF1A1",id:21},{Term:"Shigellosis",Overlap:"2/246",P_value:.3265,Adjusted_P_value:.6104,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.7329,Combined_Score:1.9398,Genes:"RPS27A;UBA52",id:22},{Term:"Inflammatory mediator regulation of TRP channels",Overlap:"1/98",P_value:.3736,Adjusted_P_value:.6461,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.1724,Combined_Score:2.139,Genes:"PTGER2",id:23},{Term:"Hematopoietic cell lineage",Overlap:"1/99",P_value:.3766,Adjusted_P_value:.6461,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.1501,Combined_Score:2.0999,Genes:"IL7R",id:24},{Term:"Parathyroid hormone synthesis, secretion and action",Overlap:"1/106",P_value:.3971,Adjusted_P_value:.6461,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:2.0061,Combined_Score:1.8527,Genes:"NACA",id:25},{Term:"HIF-1 signaling pathway",Overlap:"1/109",P_value:.4057,Adjusted_P_value:.6461,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.9501,Combined_Score:1.7592,Genes:"RPS6",id:26},{Term:"FoxO signaling pathway",Overlap:"1/131",P_value:.4652,Adjusted_P_value:.6662,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.6182,Combined_Score:1.2386,Genes:"IL7R",id:27},{Term:"Natural killer cell mediated cytotoxicity",Overlap:"1/131",P_value:.4652,Adjusted_P_value:.6662,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.6182,Combined_Score:1.2386,Genes:"NCR3",id:28},{Term:"Insulin signaling pathway",Overlap:"1/137",P_value:.4803,Adjusted_P_value:.6662,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.5464,Combined_Score:1.134,Genes:"RPS6",id:29},{Term:"Apelin signaling pathway",Overlap:"1/137",P_value:.4803,Adjusted_P_value:.6662,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.5464,Combined_Score:1.134,Genes:"RPS6",id:30},{Term:"PI3K-Akt signaling pathway",Overlap:"2/354",P_value:.5034,Adjusted_P_value:.6764,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.1946,Combined_Score:.82,Genes:"RPS6;IL7R",id:31},{Term:"mTOR signaling pathway",Overlap:"1/154",P_value:.521,Adjusted_P_value:.6789,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.3734,Combined_Score:.8954,Genes:"RPS6",id:32},{Term:"JAK-STAT signaling pathway",Overlap:"1/162",P_value:.539,Adjusted_P_value:.6817,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.3046,Combined_Score:.8062,Genes:"IL7R",id:33},{Term:"RNA transport",Overlap:"1/186",P_value:.5892,Adjusted_P_value:.7239,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.134,Combined_Score:.5998,Genes:"EEF1A1",id:34},{Term:"Pathogenic Escherichia coli infection",Overlap:"1/197",P_value:.6104,Adjusted_P_value:.7265,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0697,Combined_Score:.5281,Genes:"RPS3",id:35},{Term:"Proteoglycans in cancer",Overlap:"1/205",P_value:.6251,Adjusted_P_value:.7265,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:1.0274,Combined_Score:.4827,Genes:"RPS6",id:36},{Term:"cAMP signaling pathway",Overlap:"1/216",P_value:.6444,Adjusted_P_value:.7271,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9743,Combined_Score:.4281,Genes:"PTGER2",id:37},{Term:"Human cytomegalovirus infection",Overlap:"1/225",P_value:.6595,Adjusted_P_value:.7271,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.9347,Combined_Score:.3891,Genes:"PTGER2",id:38},{Term:"Salmonella infection",Overlap:"1/249",P_value:.6967,Adjusted_P_value:.7489,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.8432,Combined_Score:.3048,Genes:"RPS3",id:39},{Term:"Pathways in cancer",Overlap:"2/531",P_value:.7221,Adjusted_P_value:.7573,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7877,Combined_Score:.2565,Genes:"PTGER2;IL7R",id:40},{Term:"Cytokine-cytokine receptor interaction",Overlap:"1/295",P_value:.7571,Adjusted_P_value:.7751,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.7096,Combined_Score:.1975,Genes:"IL7R",id:41},{Term:"Neuroactive ligand-receptor interaction",Overlap:"1/341",P_value:.8056,Adjusted_P_value:.8056,Old_P_value:0,Old_Adjusted_P_value:0,Odds_Ratio:.6122,Combined_Score:.1324,Genes:"PTGER2",id:42}]}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p:T,i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function nn(e){let t,a,l,n;return t=new Pe({props:{$$slots:{default:[an]},$$scope:{ctx:e}}}),l=new Pe({props:{$$slots:{default:[ln]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const n={};1&a&&(n.$$scope={dirty:a,ctx:e}),t.$set(n);const _={};1&a&&(_.$$scope={dirty:a,ctx:e}),l.$set(_)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function _n(e){let t,a;return t=new pe({props:{$$slots:{content:[nn],default:[en]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},p(e,a){const l={};1&a&&(l.$$scope={dirty:a,ctx:e}),t.$set(l)},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function dn(e){let t,a,l,n;return t=new De({props:{title:"MSigDB_Hallmark_2020",open:!0,$$slots:{default:[Jl]},$$scope:{ctx:e}}}),l=new De({props:{title:"KEGG_2021_Human",$$slots:{default:[_n]},$$scope:{ctx:e}}}),{c(){m(t.$$.fragment),a=C(),m(l.$$.fragment)},m(e,_){A(t,e,_),s(e,a,_),A(l,e,_),n=!0},p(e,a){const n={};1&a&&(n.$$scope={dirty:a,ctx:e}),t.$set(n);const _={};1&a&&(_.$$scope={dirty:a,ctx:e}),l.$set(_)},i(e){n||(r(t.$$.fragment,e),r(l.$$.fragment,e),n=!0)},o(e){c(t.$$.fragment,e),c(l.$$.fragment,e),n=!1},d(e){e&&u(a),g(t,e),g(l,e)}}}function sn(e){let t,a,l,n,i,o,p,v,P,O,R,f,$,S,b,h,T,L,j,B,M,D,G,F,N,y,x,U,H,w;return p=new pe({props:{$$slots:{content:[Sl],default:[Al]},$$scope:{ctx:e}}}),f=new ml({props:{$$slots:{default:[bl]},$$scope:{ctx:e}}}),S=new Me({props:{align:"start",$$slots:{default:[Ul]},$$scope:{ctx:e}}}),D=new pe({props:{$$slots:{content:[Kl],default:[Hl]},$$scope:{ctx:e}}}),x=new ml({props:{$$slots:{default:[Yl]},$$scope:{ctx:e}}}),H=new Me({props:{align:"start",$$slots:{default:[dn]},$$scope:{ctx:e}}}),{c(){t=_("h1"),t.textContent="Cluster: c17",a=_("a"),a.innerHTML="",l=C(),n=_("h2"),n.textContent="Markers",i=_("a"),i.innerHTML="",o=C(),m(p.$$.fragment),v=C(),P=_("h2"),P.textContent="Enrichment Analysis",O=_("a"),O.innerHTML="",R=C(),m(f.$$.fragment),$=C(),m(S.$$.fragment),b=C(),h=_("h1"),h.textContent="Cluster: c18",T=_("a"),T.innerHTML="",L=C(),j=_("h2"),j.textContent="Markers",B=_("a"),B.innerHTML="",M=C(),m(D.$$.fragment),G=C(),F=_("h2"),F.textContent="Enrichment Analysis",N=_("a"),N.innerHTML="",y=C(),m(x.$$.fragment),U=C(),m(H.$$.fragment),d(a,"id","prt-h1-16-cluster-c17"),d(a,"class","pipen-report-toc-anchor"),d(i,"id","prt-h2-32-markers"),d(i,"class","pipen-report-toc-anchor"),d(O,"id","prt-h2-33-enrichment-analysis"),d(O,"class","pipen-report-toc-anchor"),d(T,"id","prt-h1-17-cluster-c18"),d(T,"class","pipen-report-toc-anchor"),d(B,"id","prt-h2-34-markers"),d(B,"class","pipen-report-toc-anchor"),d(N,"id","prt-h2-35-enrichment-analysis"),d(N,"class","pipen-report-toc-anchor")},m(e,_){s(e,t,_),s(e,a,_),s(e,l,_),s(e,n,_),s(e,i,_),s(e,o,_),A(p,e,_),s(e,v,_),s(e,P,_),s(e,O,_),s(e,R,_),A(f,e,_),s(e,$,_),A(S,e,_),s(e,b,_),s(e,h,_),s(e,T,_),s(e,L,_),s(e,j,_),s(e,B,_),s(e,M,_),A(D,e,_),s(e,G,_),s(e,F,_),s(e,N,_),s(e,y,_),A(x,e,_),s(e,U,_),A(H,e,_),w=!0},p(e,[t]){const a={};1&t&&(a.$$scope={dirty:t,ctx:e}),p.$set(a);const l={};1&t&&(l.$$scope={dirty:t,ctx:e}),f.$set(l);const n={};1&t&&(n.$$scope={dirty:t,ctx:e}),S.$set(n);const _={};1&t&&(_.$$scope={dirty:t,ctx:e}),D.$set(_);const d={};1&t&&(d.$$scope={dirty:t,ctx:e}),x.$set(d);const s={};1&t&&(s.$$scope={dirty:t,ctx:e}),H.$set(s)},i(e){w||(r(p.$$.fragment,e),r(f.$$.fragment,e),r(S.$$.fragment,e),r(D.$$.fragment,e),r(x.$$.fragment,e),r(H.$$.fragment,e),w=!0)},o(e){c(p.$$.fragment,e),c(f.$$.fragment,e),c(S.$$.fragment,e),c(D.$$.fragment,e),c(x.$$.fragment,e),c(H.$$.fragment,e),w=!1},d(e){e&&(u(t),u(a),u(l),u(n),u(i),u(o),u(v),u(P),u(O),u(R),u($),u(b),u(h),u(T),u(L),u(j),u(B),u(M),u(G),u(F),u(N),u(y),u(U)),g(p,e),g(f,e),g(S,e),g(D,e),g(x,e),g(H,e)}}}class rn extends t{constructor(e){super(),a(this,e,null,sn,l,{})}}var on=[{slug:"prt-h1-0-cluster-c1",text:"Cluster: c1",children:[{slug:"prt-h2-0-markers",text:"Markers",children:[],page:0},{slug:"prt-h2-1-enrichment-analysis",text:"Enrichment Analysis",children:[],page:0}],page:0},{slug:"prt-h1-1-cluster-c2",text:"Cluster: c2",children:[{slug:"prt-h2-2-markers",text:"Markers",children:[],page:0},{slug:"prt-h2-3-enrichment-analysis",text:"Enrichment Analysis",children:[],page:0}],page:0},{slug:"prt-h1-2-cluster-c3",text:"Cluster: c3",children:[{slug:"prt-h2-4-markers",text:"Markers",children:[],page:0},{slug:"prt-h2-5-enrichment-analysis",text:"Enrichment Analysis",children:[],page:0}],page:0},{slug:"prt-h1-3-cluster-c4",text:"Cluster: c4",children:[{slug:"prt-h2-6-markers",text:"Markers",children:[],page:0},{slug:"prt-h2-7-enrichment-analysis",text:"Enrichment Analysis",children:[],page:0}],page:0},{slug:"prt-h1-4-cluster-c5",text:"Cluster: c5",children:[{slug:"prt-h2-8-markers",text:"Markers",children:[],page:0},{slug:"prt-h2-9-enrichment-analysis",text:"Enrichment Analysis",children:[],page:0}],page:0},{slug:"prt-h1-5-cluster-c6",text:"Cluster: c6",children:[{slug:"prt-h2-10-markers",text:"Markers",children:[],page:0},{slug:"prt-h2-11-enrichment-analysis",text:"Enrichment Analysis",children:[],page:0}],page:0},{slug:"prt-h1-6-cluster-c7",text:"Cluster: c7",children:[{slug:"prt-h2-12-markers",text:"Markers",children:[],page:0},{slug:"prt-h2-13-enrichment-analysis",text:"Enrichment Analysis",children:[],page:0}],page:0},{slug:"prt-h1-7-cluster-c8",text:"Cluster: c8",children:[{slug:"prt-h2-14-markers",text:"Markers",children:[],page:0},{slug:"prt-h2-15-enrichment-analysis",text:"Enrichment Analysis",children:[],page:0}],page:0},{slug:"prt-h1-8-cluster-c9",text:"Cluster: c9",children:[{slug:"prt-h2-16-markers",text:"Markers",children:[],page:1},{slug:"prt-h2-17-enrichment-analysis",text:"Enrichment Analysis",children:[],page:1}],page:1},{slug:"prt-h1-9-cluster-c10",text:"Cluster: c10",children:[{slug:"prt-h2-18-markers",text:"Markers",children:[],page:1},{slug:"prt-h2-19-enrichment-analysis",text:"Enrichment Analysis",children:[],page:1}],page:1},{slug:"prt-h1-10-cluster-c11",text:"Cluster: c11",children:[{slug:"prt-h2-20-markers",text:"Markers",children:[],page:1},{slug:"prt-h2-21-enrichment-analysis",text:"Enrichment Analysis",children:[],page:1}],page:1},{slug:"prt-h1-11-cluster-c12",text:"Cluster: c12",children:[{slug:"prt-h2-22-markers",text:"Markers",children:[],page:1},{slug:"prt-h2-23-enrichment-analysis",text:"Enrichment Analysis",children:[],page:1}],page:1},{slug:"prt-h1-12-cluster-c13",text:"Cluster: c13",children:[{slug:"prt-h2-24-markers",text:"Markers",children:[],page:1},{slug:"prt-h2-25-enrichment-analysis",text:"Enrichment Analysis",children:[],page:1}],page:1},{slug:"prt-h1-13-cluster-c14",text:"Cluster: c14",children:[{slug:"prt-h2-26-markers",text:"Markers",children:[],page:1},{slug:"prt-h2-27-enrichment-analysis",text:"Enrichment Analysis",children:[],page:1}],page:1},{slug:"prt-h1-14-cluster-c15",text:"Cluster: c15",children:[{slug:"prt-h2-28-markers",text:"Markers",children:[],page:1},{slug:"prt-h2-29-enrichment-analysis",text:"Enrichment Analysis",children:[],page:1}],page:1},{slug:"prt-h1-15-cluster-c16",text:"Cluster: c16",children:[{slug:"prt-h2-30-markers",text:"Markers",children:[],page:1},{slug:"prt-h2-31-enrichment-analysis",text:"Enrichment Analysis",children:[],page:1}],page:1},{slug:"prt-h1-16-cluster-c17",text:"Cluster: c17",children:[{slug:"prt-h2-32-markers",text:"Markers",children:[],page:2},{slug:"prt-h2-33-enrichment-analysis",text:"Enrichment Analysis",children:[],page:2}],page:2},{slug:"prt-h1-17-cluster-c18",text:"Cluster: c18",children:[{slug:"prt-h2-34-markers",text:"Markers",children:[],page:2},{slug:"prt-h2-35-enrichment-analysis",text:"Enrichment Analysis",children:[],page:2}],page:2}],cn={pipeline:{name:"Immunopipe",desc:"Immunopipe (v1.4.3): Integrative analysis for scRNA-seq and scTCR-seq data"},versions:'\n    <a href="https://github.com/pwwang/pipen" target="_blank">\n        pipen v0.15.7\n    </a> and\n    <a href="https://github.com/pwwang/pipen-report" target="_blank">\n        pipen-report v0.21.4\n    </a>\n',entries:[{name:"SampleInfo",desc:"List sample information and perform statistics",npages:1,report_toc:!0,order:0,runinfo:{session:"# Generated by pipen_runinfo v0.8.2\n# Lang: R\nR version 4.3.3 (2024-02-29)\nPlatform: x86_64-conda-linux-gnu (64-bit)\nRunning under: Debian GNU/Linux 11 (bullseye)\n\nMatrix products: default\nBLAS/LAPACK: /opt/conda/lib/libopenblasp-r0.3.28.so;  LAPACK version 3.12.0\n\nlocale:\n [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       \n [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   \n [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          \n[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   \n\ntime zone: Etc/UTC\ntzcode source: system (glibc)\n\nattached base packages:\n[1] stats     graphics  grDevices utils     datasets  methods   base     \n\nother attached packages:\n[1] ggrepel_0.9.6    ggprism_1.0.5    ggplot2_3.5.1    tidyr_1.3.1     \n[5] dplyr_1.1.4      tidyselect_1.2.1 jsonlite_1.8.9   rlang_1.1.5     \n[9] logger_0.4.0    \n\nloaded via a namespace (and not attached):\n [1] vctrs_0.6.5      cli_3.6.4        purrr_1.0.2      generics_0.1.3  \n [5] glue_1.8.0       labeling_0.4.3   colorspace_2.1-1 scales_1.3.0    \n [9] grid_4.3.3       munsell_0.5.1    tibble_3.2.1     lifecycle_1.0.4 \n[13] compiler_4.3.3   Rcpp_1.0.14      pkgconfig_2.0.3  farver_2.1.2    \n[17] digest_0.6.37    R6_2.6.0         pillar_1.10.1    magrittr_2.0.3  \n[21] withr_3.0.2      gtable_0.3.6    \n",time:"# Generated by pipen-runinfo v0.8.2\n\nreal 1.78\nuser 10.68\nsys 0.14\npercent 606%\n",device:"# Generated by pipen-runinfo v0.8.2\n\nScheduler\n---------\nlocal\n\nHostname\n--------\nmforgehn1.mayo.edu\n\nCPU\n----\nArchitecture:                         x86_64\nCPU op-mode(s):                       32-bit, 64-bit\nByte Order:                           Little Endian\nAddress sizes:                        46 bits physical, 57 bits virtual\nCPU(s):                               96\nOn-line CPU(s) list:                  0-95\nThread(s) per core:                   2\nCore(s) per socket:                   24\nSocket(s):                            2\nNUMA node(s):                         2\nVendor ID:                            GenuineIntel\nCPU family:                           6\nModel:                                106\nModel name:                           Intel(R) Xeon(R) Gold 6342 CPU @ 2.80GHz\nStepping:                             6\nCPU MHz:                              3500.000\nCPU max MHz:                          3500.0000\nCPU min MHz:                          800.0000\nBogoMIPS:                             5600.00\nVirtualization:                       VT-x\nL1d cache:                            2.3 MiB\nL1i cache:                            1.5 MiB\nL2 cache:                             60 MiB\nL3 cache:                             72 MiB\nNUMA node0 CPU(s):                    0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48,50,52,54,56,58,60,62,64,66,68,70,72,74,76,78,80,82,84,86,88,90,92,94\nNUMA node1 CPU(s):                    1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,45,47,49,51,53,55,57,59,61,63,65,67,69,71,73,75,77,79,81,83,85,87,89,91,93,95\nVulnerability Gather data sampling:   Mitigation; Microcode\nVulnerability Itlb multihit:          Not affected\nVulnerability L1tf:                   Not affected\nVulnerability Mds:                    Not affected\nVulnerability Meltdown:               Not affected\nVulnerability Mmio stale data:        Mitigation; Clear CPU buffers; SMT vulnerable\nVulnerability Reg file data sampling: Not affected\nVulnerability Retbleed:               Not affected\nVulnerability Spec rstack overflow:   Not affected\nVulnerability Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl\nVulnerability Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization\nVulnerability Spectre v2:             Mitigation; Enhanced / Automatic IBRS; IBPB conditional; RSB filling; PBRSB-eIBRS SW sequence; BHI SW loop, KVM SW loop\nVulnerability Srbds:                  Not affected\nVulnerability Tsx async abort:        Not affected\nFlags:                                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 invpcid_single intel_ppin ssbd mba ibrs ibpb stibp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm rdt_a avx512f avx512dq rdseed adx smap avx512ifma clflushopt clwb intel_pt avx512cd sha_ni avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local split_lock_detect wbnoinvd dtherm ida arat pln pts avx512vbmi umip pku ospke avx512_vbmi2 gfni vaes vpclmulqdq avx512_vnni avx512_bitalg tme avx512_vpopcntdq la57 rdpid fsrm md_clear pconfig flush_l1d arch_capabilities\n\nMemory\n------\n"}},{name:"SeuratPreparing",desc:"Load, prepare and apply QC to data, using `Seurat`",npages:1,report_toc:!0,order:2,runinfo:{session:"# Generated by pipen_runinfo v0.8.2\n# Lang: R\nR version 4.3.3 (2024-02-29)\nPlatform: x86_64-conda-linux-gnu (64-bit)\nRunning under: Debian GNU/Linux 11 (bullseye)\n\nMatrix products: default\nBLAS/LAPACK: /opt/conda/lib/libopenblasp-r0.3.28.so;  LAPACK version 3.12.0\n\nlocale:\n [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       \n [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   \n [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          \n[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   \n\ntime zone: Etc/UTC\ntzcode source: system (glibc)\n\nattached base packages:\n[1] stats     graphics  grDevices utils     datasets  methods   base     \n\nother attached packages:\n [1] dplyr_1.1.4        ggplot2_3.5.1      bracer_1.2.1       future_1.34.0     \n [5] Seurat_5.1.0       SeuratObject_5.0.2 sp_2.2-0           digest_0.6.37     \n [9] jsonlite_1.8.9     rlang_1.1.5        logger_0.4.0      \n\nloaded via a namespace (and not attached):\n  [1] RColorBrewer_1.1-3          magrittr_2.0.3             \n  [3] spatstat.utils_3.1-2        farver_2.1.2               \n  [5] zlibbioc_1.48.0             vctrs_0.6.5                \n  [7] ROCR_1.0-11                 spatstat.explore_3.3-4     \n  [9] RCurl_1.98-1.16             S4Arrays_1.2.0             \n [11] htmltools_0.5.8.1           SparseArray_1.2.2          \n [13] sctransform_0.4.1           parallelly_1.42.0          \n [15] KernSmooth_2.23-26          htmlwidgets_1.6.4          \n [17] ica_1.0-3                   plyr_1.8.9                 \n [19] plotly_4.10.4               zoo_1.8-12                 \n [21] igraph_2.0.3                mime_0.12                  \n [23] lifecycle_1.0.4             pkgconfig_2.0.3            \n [25] Matrix_1.6-5                R6_2.6.0                   \n [27] fastmap_1.2.0               GenomeInfoDbData_1.2.11    \n [29] MatrixGenerics_1.14.0       fitdistrplus_1.2-2         \n [31] shiny_1.10.0                colorspace_2.1-1           \n [33] patchwork_1.3.0             S4Vectors_0.40.2           \n [35] tensor_1.5                  RSpectra_0.16-2            \n [37] irlba_2.3.5.1               GenomicRanges_1.54.1       \n [39] labeling_0.4.3              progressr_0.15.1           \n [41] spatstat.sparse_3.1-0       httr_1.4.7                 \n [43] polyclip_1.10-7             abind_1.4-5                \n [45] compiler_4.3.3              withr_3.0.2                \n [47] fastDummies_1.7.5           R.utils_2.12.3             \n [49] MASS_7.3-60.0.1             DelayedArray_0.28.0        \n [51] tools_4.3.3                 lmtest_0.9-40              \n [53] httpuv_1.6.15               future.apply_1.11.3        \n [55] goftest_1.2-3               R.oo_1.27.0                \n [57] glmGamPoi_1.14.0            glue_1.8.0                 \n [59] nlme_3.1-167                promises_1.3.2             \n [61] grid_4.3.3                  Rtsne_0.17                 \n [63] cluster_2.1.8               reshape2_1.4.4             \n [65] generics_0.1.3              gtable_0.3.6               \n [67] spatstat.data_3.1-4         R.methodsS3_1.8.2          \n [69] tidyr_1.3.1                 data.table_1.16.4          \n [71] XVector_0.42.0              BiocGenerics_0.48.1        \n [73] spatstat.geom_3.3-5         RcppAnnoy_0.0.22           \n [75] ggrepel_0.9.6               RANN_2.6.2                 \n [77] pillar_1.10.1               stringr_1.5.1              \n [79] spam_2.11-1                 RcppHNSW_0.6.0             \n [81] later_1.4.1                 splines_4.3.3              \n [83] lattice_0.22-6              survival_3.8-3             \n [85] deldir_2.0-4                tidyselect_1.2.1           \n [87] miniUI_0.1.1.1              pbapply_1.7-2              \n [89] gridExtra_2.3               IRanges_2.36.0             \n [91] SummarizedExperiment_1.32.0 scattermore_1.2            \n [93] RhpcBLASctl_0.23-42         stats4_4.3.3               \n [95] Biobase_2.62.0              matrixStats_1.1.0          \n [97] stringi_1.8.4               lazyeval_0.2.2             \n [99] codetools_0.2-20            tibble_3.2.1               \n[101] cli_3.6.4                   uwot_0.2.2                 \n[103] xtable_1.8-4                reticulate_1.40.0          \n[105] munsell_0.5.1               harmony_1.2.0              \n[107] Rcpp_1.0.14                 GenomeInfoDb_1.38.1        \n[109] globals_0.16.3              spatstat.random_3.3-2      \n[111] png_0.1-8                   spatstat.univar_3.1-1      \n[113] parallel_4.3.3              dotCall64_1.2              \n[115] bitops_1.0-9                listenv_0.9.1              \n[117] viridisLite_0.4.2           scales_1.3.0               \n[119] ggridges_0.5.6              crayon_1.5.3               \n[121] leiden_0.4.3.1              purrr_1.0.2                \n[123] cowplot_1.1.3              \n",time:"# Generated by pipen-runinfo v0.8.2\n\nreal 4921.02\nuser 13708.15\nsys 1959.18\npercent 318%\n",device:"# Generated by pipen-runinfo v0.8.2\n\nScheduler\n---------\nlocal\n\nHostname\n--------\nmforgehn1.mayo.edu\n\nCPU\n----\nArchitecture:                         x86_64\nCPU op-mode(s):                       32-bit, 64-bit\nByte Order:                           Little Endian\nAddress sizes:                        46 bits physical, 57 bits virtual\nCPU(s):                               96\nOn-line CPU(s) list:                  0-95\nThread(s) per core:                   2\nCore(s) per socket:                   24\nSocket(s):                            2\nNUMA node(s):                         2\nVendor ID:                            GenuineIntel\nCPU family:                           6\nModel:                                106\nModel name:                           Intel(R) Xeon(R) Gold 6342 CPU @ 2.80GHz\nStepping:                             6\nCPU MHz:                              2510.056\nCPU max MHz:                          3500.0000\nCPU min MHz:                          800.0000\nBogoMIPS:                             5600.00\nVirtualization:                       VT-x\nL1d cache:                            2.3 MiB\nL1i cache:                            1.5 MiB\nL2 cache:                             60 MiB\nL3 cache:                             72 MiB\nNUMA node0 CPU(s):                    0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48,50,52,54,56,58,60,62,64,66,68,70,72,74,76,78,80,82,84,86,88,90,92,94\nNUMA node1 CPU(s):                    1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,45,47,49,51,53,55,57,59,61,63,65,67,69,71,73,75,77,79,81,83,85,87,89,91,93,95\nVulnerability Gather data sampling:   Mitigation; Microcode\nVulnerability Itlb multihit:          Not affected\nVulnerability L1tf:                   Not affected\nVulnerability Mds:                    Not affected\nVulnerability Meltdown:               Not affected\nVulnerability Mmio stale data:        Mitigation; Clear CPU buffers; SMT vulnerable\nVulnerability Reg file data sampling: Not affected\nVulnerability Retbleed:               Not affected\nVulnerability Spec rstack overflow:   Not affected\nVulnerability Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl\nVulnerability Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization\nVulnerability Spectre v2:             Mitigation; Enhanced / Automatic IBRS; IBPB conditional; RSB filling; PBRSB-eIBRS SW sequence; BHI SW loop, KVM SW loop\nVulnerability Srbds:                  Not affected\nVulnerability Tsx async abort:        Not affected\nFlags:                                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 invpcid_single intel_ppin ssbd mba ibrs ibpb stibp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm rdt_a avx512f avx512dq rdseed adx smap avx512ifma clflushopt clwb intel_pt avx512cd sha_ni avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local split_lock_detect wbnoinvd dtherm ida arat pln pts avx512vbmi umip pku ospke avx512_vbmi2 gfni vaes vpclmulqdq avx512_vnni avx512_bitalg tme avx512_vpopcntdq la57 rdpid fsrm md_clear pconfig flush_l1d arch_capabilities\n\nMemory\n------\n"}},{name:"TCellSelection",desc:"Separate T and non-T cells and select T cells.",npages:1,report_toc:!0,order:4,runinfo:{session:"# Generated by pipen_runinfo v0.8.2\n# Lang: R\nR version 4.3.3 (2024-02-29)\nPlatform: x86_64-conda-linux-gnu (64-bit)\nRunning under: Debian GNU/Linux 11 (bullseye)\n\nMatrix products: default\nBLAS/LAPACK: /opt/conda/lib/libopenblasp-r0.3.28.so;  LAPACK version 3.12.0\n\nlocale:\n [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       \n [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   \n [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          \n[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   \n\ntime zone: Etc/UTC\ntzcode source: system (glibc)\n\nattached base packages:\n[1] stats     graphics  grDevices utils     datasets  methods   base     \n\nother attached packages:\n [1] glue_1.8.0         factoextra_1.0.7   patchwork_1.3.0    ggpubr_0.6.0      \n [5] ggrepel_0.9.6      ggprism_1.0.5      ggplot2_3.5.1      tibble_3.2.1      \n [9] tidyr_1.3.1        dplyr_1.1.4        Seurat_5.1.0       SeuratObject_5.0.2\n[13] sp_2.2-0           jsonlite_1.8.9     rlang_1.1.5        logger_0.4.0      \n\nloaded via a namespace (and not attached):\n  [1] deldir_2.0-4           pbapply_1.7-2          gridExtra_2.3         \n  [4] magrittr_2.0.3         RcppAnnoy_0.0.22       matrixStats_1.1.0     \n  [7] ggridges_0.5.6         compiler_4.3.3         spatstat.geom_3.3-5   \n [10] png_0.1-8              vctrs_0.6.5            reshape2_1.4.4        \n [13] stringr_1.5.1          pkgconfig_2.0.3        fastmap_1.2.0         \n [16] backports_1.5.0        labeling_0.4.3         promises_1.3.2        \n [19] purrr_1.0.2            goftest_1.2-3          later_1.4.1           \n [22] spatstat.utils_3.1-2   broom_1.0.7            irlba_2.3.5.1         \n [25] parallel_4.3.3         cluster_2.1.8          R6_2.6.0              \n [28] ica_1.0-3              stringi_1.8.4          RColorBrewer_1.1-3    \n [31] spatstat.data_3.1-4    reticulate_1.40.0      car_3.1-3             \n [34] parallelly_1.42.0      spatstat.univar_3.1-1  lmtest_0.9-40         \n [37] scattermore_1.2        Rcpp_1.0.14            tensor_1.5            \n [40] future.apply_1.11.3    zoo_1.8-12             sctransform_0.4.1     \n [43] httpuv_1.6.15          Matrix_1.6-5           splines_4.3.3         \n [46] igraph_2.0.3           tidyselect_1.2.1       abind_1.4-5           \n [49] spatstat.random_3.3-2  codetools_0.2-20       miniUI_0.1.1.1        \n [52] spatstat.explore_3.3-4 listenv_0.9.1          lattice_0.22-6        \n [55] plyr_1.8.9             withr_3.0.2            shiny_1.10.0          \n [58] ROCR_1.0-11            Rtsne_0.17             future_1.34.0         \n [61] fastDummies_1.7.5      survival_3.8-3         polyclip_1.10-7       \n [64] fitdistrplus_1.2-2     pillar_1.10.1          carData_3.0-5         \n [67] KernSmooth_2.23-26     plotly_4.10.4          generics_0.1.3        \n [70] RcppHNSW_0.6.0         munsell_0.5.1          scales_1.3.0          \n [73] globals_0.16.3         xtable_1.8-4           lazyeval_0.2.2        \n [76] tools_4.3.3            data.table_1.16.4      RSpectra_0.16-2       \n [79] ggsignif_0.6.4         RANN_2.6.2             leiden_0.4.3.1        \n [82] dotCall64_1.2          cowplot_1.1.3          grid_4.3.3            \n [85] colorspace_2.1-1       nlme_3.1-167           Formula_1.2-5         \n [88] cli_3.6.4              spatstat.sparse_3.1-0  spam_2.11-1           \n [91] viridisLite_0.4.2      uwot_0.2.2             gtable_0.3.6          \n [94] rstatix_0.7.2          digest_0.6.37          progressr_0.15.1      \n [97] htmlwidgets_1.6.4      farver_2.1.2           htmltools_0.5.8.1     \n[100] lifecycle_1.0.4        httr_1.4.7             mime_0.12             \n[103] MASS_7.3-60.0.1       \n",time:"# Generated by pipen-runinfo v0.8.2\n\nreal 490.09\nuser 488.00\nsys 10.24\npercent 101%\n",device:"# Generated by pipen-runinfo v0.8.2\n\nScheduler\n---------\nlocal\n\nHostname\n--------\nmforgehn1.mayo.edu\n\nCPU\n----\nArchitecture:                         x86_64\nCPU op-mode(s):                       32-bit, 64-bit\nByte Order:                           Little Endian\nAddress sizes:                        46 bits physical, 57 bits virtual\nCPU(s):                               96\nOn-line CPU(s) list:                  0-95\nThread(s) per core:                   2\nCore(s) per socket:                   24\nSocket(s):                            2\nNUMA node(s):                         2\nVendor ID:                            GenuineIntel\nCPU family:                           6\nModel:                                106\nModel name:                           Intel(R) Xeon(R) Gold 6342 CPU @ 2.80GHz\nStepping:                             6\nCPU MHz:                              2292.413\nCPU max MHz:                          3500.0000\nCPU min MHz:                          800.0000\nBogoMIPS:                             5600.00\nVirtualization:                       VT-x\nL1d cache:                            2.3 MiB\nL1i cache:                            1.5 MiB\nL2 cache:                             60 MiB\nL3 cache:                             72 MiB\nNUMA node0 CPU(s):                    0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48,50,52,54,56,58,60,62,64,66,68,70,72,74,76,78,80,82,84,86,88,90,92,94\nNUMA node1 CPU(s):                    1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,45,47,49,51,53,55,57,59,61,63,65,67,69,71,73,75,77,79,81,83,85,87,89,91,93,95\nVulnerability Gather data sampling:   Mitigation; Microcode\nVulnerability Itlb multihit:          Not affected\nVulnerability L1tf:                   Not affected\nVulnerability Mds:                    Not affected\nVulnerability Meltdown:               Not affected\nVulnerability Mmio stale data:        Mitigation; Clear CPU buffers; SMT vulnerable\nVulnerability Reg file data sampling: Not affected\nVulnerability Retbleed:               Not affected\nVulnerability Spec rstack overflow:   Not affected\nVulnerability Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl\nVulnerability Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization\nVulnerability Spectre v2:             Mitigation; Enhanced / Automatic IBRS; IBPB conditional; RSB filling; PBRSB-eIBRS SW sequence; BHI SW loop, KVM SW loop\nVulnerability Srbds:                  Not affected\nVulnerability Tsx async abort:        Not affected\nFlags:                                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 invpcid_single intel_ppin ssbd mba ibrs ibpb stibp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm rdt_a avx512f avx512dq rdseed adx smap avx512ifma clflushopt clwb intel_pt avx512cd sha_ni avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local split_lock_detect wbnoinvd dtherm ida arat pln pts avx512vbmi umip pku ospke avx512_vbmi2 gfni vaes vpclmulqdq avx512_vnni avx512_bitalg tme avx512_vpopcntdq la57 rdpid fsrm md_clear pconfig flush_l1d arch_capabilities\n\nMemory\n------\n"}},{name:"SeuratClusterStats",desc:"Statistics of the clustering.",npages:1,report_toc:!0,order:8,runinfo:{session:"# Generated by pipen_runinfo v0.8.2\n# Lang: R\nR version 4.3.3 (2024-02-29)\nPlatform: x86_64-conda-linux-gnu (64-bit)\nRunning under: Debian GNU/Linux 11 (bullseye)\n\nMatrix products: default\nBLAS/LAPACK: /opt/conda/lib/libopenblasp-r0.3.28.so;  LAPACK version 3.12.0\n\nlocale:\n [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       \n [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   \n [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          \n[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   \n\ntime zone: Etc/UTC\ntzcode source: system (glibc)\n\nattached base packages:\n[1] grid      stats     graphics  grDevices utils     datasets  methods  \n[8] base     \n\nother attached packages:\n [1] clustree_0.5.1        ggraph_2.2.1          ComplexHeatmap_2.18.0\n [4] circlize_0.4.16       tidyseurat_0.8.0      ttservice_0.4.1      \n [7] ggrepel_0.9.6         ggprism_1.0.5         tibble_3.2.1         \n[10] Seurat_5.1.0          SeuratObject_5.0.2    sp_2.2-0             \n[13] ggplot2_3.5.1         tidyr_1.3.1           dplyr_1.1.4          \n[16] tidyselect_1.2.1      jsonlite_1.8.9        rlang_1.1.5          \n[19] logger_0.4.0         \n\nloaded via a namespace (and not attached):\n  [1] RColorBrewer_1.1-3     shape_1.4.6.1          magrittr_2.0.3        \n  [4] ggbeeswarm_0.7.2       spatstat.utils_3.1-2   farver_2.1.2          \n  [7] GlobalOptions_0.1.2    vctrs_0.6.5            ROCR_1.0-11           \n [10] memoise_2.0.1          spatstat.explore_3.3-4 htmltools_0.5.8.1     \n [13] sctransform_0.4.1      parallelly_1.42.0      KernSmooth_2.23-26    \n [16] htmlwidgets_1.6.4      ica_1.0-3              plyr_1.8.9            \n [19] cachem_1.1.0           plotly_4.10.4          zoo_1.8-12            \n [22] igraph_2.0.3           mime_0.12              lifecycle_1.0.4       \n [25] iterators_1.0.14       pkgconfig_2.0.3        Matrix_1.6-5          \n [28] R6_2.6.0               fastmap_1.2.0          clue_0.3-66           \n [31] fitdistrplus_1.2-2     future_1.34.0          shiny_1.10.0          \n [34] digest_0.6.37          colorspace_2.1-1       patchwork_1.3.0       \n [37] S4Vectors_0.40.2       tensor_1.5             RSpectra_0.16-2       \n [40] irlba_2.3.5.1          labeling_0.4.3         progressr_0.15.1      \n [43] fansi_1.0.6            spatstat.sparse_3.1-0  httr_1.4.7            \n [46] polyclip_1.10-7        abind_1.4-5            compiler_4.3.3        \n [49] withr_3.0.2            doParallel_1.0.17      backports_1.5.0       \n [52] viridis_0.6.5          fastDummies_1.7.5      ggforce_0.4.2         \n [55] MASS_7.3-60.0.1        rjson_0.2.23           tools_4.3.3           \n [58] vipor_0.4.7            lmtest_0.9-40          beeswarm_0.4.0        \n [61] httpuv_1.6.15          future.apply_1.11.3    goftest_1.2-3         \n [64] glue_1.8.0             nlme_3.1-167           promises_1.3.2        \n [67] checkmate_2.3.2        Rtsne_0.17             cluster_2.1.8         \n [70] reshape2_1.4.4         generics_0.1.3         gtable_0.3.6          \n [73] spatstat.data_3.1-4    data.table_1.16.4      tidygraph_1.3.0       \n [76] BiocGenerics_0.48.1    spatstat.geom_3.3-5    RcppAnnoy_0.0.22      \n [79] RANN_2.6.2             foreach_1.5.2          pillar_1.10.1         \n [82] stringr_1.5.1          spam_2.11-1            RcppHNSW_0.6.0        \n [85] later_1.4.1            splines_4.3.3          tweenr_2.0.3          \n [88] lattice_0.22-6         survival_3.8-3         deldir_2.0-4          \n [91] miniUI_0.1.1.1         pbapply_1.7-2          gridExtra_2.3         \n [94] IRanges_2.36.0         scattermore_1.2        stats4_4.3.3          \n [97] graphlayouts_1.2.2     matrixStats_1.1.0      stringi_1.8.4         \n[100] lazyeval_0.2.2         codetools_0.2-20       cli_3.6.4             \n[103] uwot_0.2.2             xtable_1.8-4           reticulate_1.40.0     \n[106] munsell_0.5.1          Rcpp_1.0.14            globals_0.16.3        \n[109] spatstat.random_3.3-2  png_0.1-8              ggrastr_1.0.2         \n[112] spatstat.univar_3.1-1  parallel_4.3.3         ellipsis_0.3.2        \n[115] dotCall64_1.2          listenv_0.9.1          viridisLite_0.4.2     \n[118] scales_1.3.0           ggridges_0.5.6         leiden_0.4.3.1        \n[121] purrr_1.0.2            crayon_1.5.3           GetoptLong_1.0.5      \n[124] cowplot_1.1.3         \n",time:"# Generated by pipen-runinfo v0.8.2\n\nreal 130.43\nuser 132.79\nsys 6.56\npercent 106%\n",device:"# Generated by pipen-runinfo v0.8.2\n\nScheduler\n---------\nlocal\n\nHostname\n--------\nmforgehn1.mayo.edu\n\nCPU\n----\nArchitecture:                         x86_64\nCPU op-mode(s):                       32-bit, 64-bit\nByte Order:                           Little Endian\nAddress sizes:                        46 bits physical, 57 bits virtual\nCPU(s):                               96\nOn-line CPU(s) list:                  0-95\nThread(s) per core:                   2\nCore(s) per socket:                   24\nSocket(s):                            2\nNUMA node(s):                         2\nVendor ID:                            GenuineIntel\nCPU family:                           6\nModel:                                106\nModel name:                           Intel(R) Xeon(R) Gold 6342 CPU @ 2.80GHz\nStepping:                             6\nCPU MHz:                              3500.000\nCPU max MHz:                          3500.0000\nCPU min MHz:                          800.0000\nBogoMIPS:                             5600.00\nVirtualization:                       VT-x\nL1d cache:                            2.3 MiB\nL1i cache:                            1.5 MiB\nL2 cache:                             60 MiB\nL3 cache:                             72 MiB\nNUMA node0 CPU(s):                    0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48,50,52,54,56,58,60,62,64,66,68,70,72,74,76,78,80,82,84,86,88,90,92,94\nNUMA node1 CPU(s):                    1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,45,47,49,51,53,55,57,59,61,63,65,67,69,71,73,75,77,79,81,83,85,87,89,91,93,95\nVulnerability Gather data sampling:   Mitigation; Microcode\nVulnerability Itlb multihit:          Not affected\nVulnerability L1tf:                   Not affected\nVulnerability Mds:                    Not affected\nVulnerability Meltdown:               Not affected\nVulnerability Mmio stale data:        Mitigation; Clear CPU buffers; SMT vulnerable\nVulnerability Reg file data sampling: Not affected\nVulnerability Retbleed:               Not affected\nVulnerability Spec rstack overflow:   Not affected\nVulnerability Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl\nVulnerability Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization\nVulnerability Spectre v2:             Mitigation; Enhanced / Automatic IBRS; IBPB conditional; RSB filling; PBRSB-eIBRS SW sequence; BHI SW loop, KVM SW loop\nVulnerability Srbds:                  Not affected\nVulnerability Tsx async abort:        Not affected\nFlags:                                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 invpcid_single intel_ppin ssbd mba ibrs ibpb stibp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm rdt_a avx512f avx512dq rdseed adx smap avx512ifma clflushopt clwb intel_pt avx512cd sha_ni avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local split_lock_detect wbnoinvd dtherm ida arat pln pts avx512vbmi umip pku ospke avx512_vbmi2 gfni vaes vpclmulqdq avx512_vnni avx512_bitalg tme avx512_vpopcntdq la57 rdpid fsrm md_clear pconfig flush_l1d arch_capabilities\n\nMemory\n------\n"}},{name:"Immunarch",desc:"Exploration of Single-cell and Bulk T-cell/Antibody Immune Repertoires",npages:3,report_toc:!0,order:9,runinfo:{session:"# Generated by pipen_runinfo v0.8.2\n# Lang: R\nR version 4.3.3 (2024-02-29)\nPlatform: x86_64-conda-linux-gnu (64-bit)\nRunning under: Debian GNU/Linux 11 (bullseye)\n\nMatrix products: default\nBLAS/LAPACK: /opt/conda/lib/libopenblasp-r0.3.28.so;  LAPACK version 3.12.0\n\nlocale:\n [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       \n [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   \n [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          \n[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   \n\ntime zone: Etc/UTC\ntzcode source: system (glibc)\n\nattached base packages:\n[1] stats     graphics  grDevices utils     datasets  methods   base     \n\nother attached packages:\n [1] circlize_0.4.16   tibble_3.2.1      glue_1.8.0        ggprism_1.0.5    \n [5] immunarch_0.9.1   patchwork_1.3.0   data.table_1.16.4 dtplyr_1.3.1     \n [9] ggplot2_3.5.1     tidyr_1.3.1       dplyr_1.1.4       jsonlite_1.8.9   \n[13] rlang_1.1.5       logger_0.4.0     \n\nloaded via a namespace (and not attached):\n  [1] gridExtra_2.3       phangorn_2.12.1     readxl_1.4.3       \n  [4] magrittr_2.0.3      compiler_4.3.3      flexmix_2.3-19     \n  [7] vctrs_0.6.5         reshape2_1.4.4      ggalluvial_0.12.5  \n [10] quadprog_1.5-8      stringr_1.5.1       pkgconfig_2.0.3    \n [13] shape_1.4.6.1       fastmap_1.2.0       backports_1.5.0    \n [16] labeling_0.4.3      ggraph_2.2.1        promises_1.3.2     \n [19] tzdb_0.4.0          UpSetR_1.4.0        purrr_1.0.2        \n [22] modeltools_0.2-23   ggseqlogo_0.2       cachem_1.1.0       \n [25] later_1.4.1         uuid_1.2-1          fpc_2.2-13         \n [28] tweenr_2.0.3        broom_1.0.7         parallel_4.3.3     \n [31] prabclus_2.3-4      cluster_2.1.8       R6_2.6.0           \n [34] stringi_1.8.4       RColorBrewer_1.1-3  rlist_0.4.6.2      \n [37] car_3.1-3           diptest_0.77-1      cellranger_1.1.0   \n [40] assertthat_0.2.1    Rcpp_1.0.14         iterators_1.0.14   \n [43] readr_2.1.5         httpuv_1.6.15       Matrix_1.6-5       \n [46] nnet_7.3-20         igraph_2.0.3        tidyselect_1.2.1   \n [49] stringdist_0.9.15   abind_1.4-5         viridis_0.6.5      \n [52] doParallel_1.0.17   codetools_0.2-20    lattice_0.22-6     \n [55] plyr_1.8.9          shiny_1.10.0        withr_3.0.2        \n [58] polyclip_1.10-7     mclust_6.1.1        kernlab_0.9-33     \n [61] pillar_1.10.1       ggpubr_0.6.0        carData_3.0-5      \n [64] foreach_1.5.2       stats4_4.3.3        generics_0.1.3     \n [67] hms_1.1.3           munsell_0.5.1       scales_1.3.0       \n [70] xtable_1.8-4        class_7.3-23        pheatmap_1.0.12    \n [73] tools_4.3.3         robustbase_0.99-4-1 ggsignif_0.6.4     \n [76] graphlayouts_1.2.2  fastmatch_1.1-6     tidygraph_1.3.0    \n [79] grid_4.3.3          ape_5.8-1           colorspace_2.1-1   \n [82] nlme_3.1-167        ggforce_0.4.2       Formula_1.2-5      \n [85] cli_3.6.4           viridisLite_0.4.2   gtable_0.3.6       \n [88] DEoptimR_1.1-3-1    rstatix_0.7.2       digest_0.6.37      \n [91] ggrepel_0.9.6       farver_2.1.2        memoise_2.0.1      \n [94] htmltools_0.5.8.1   lifecycle_1.0.4     factoextra_1.0.7   \n [97] mime_0.12           GlobalOptions_0.1.2 shinythemes_1.2.0  \n[100] MASS_7.3-60.0.1    \n",time:"# Generated by pipen-runinfo v0.8.2\n\nreal 509.14\nuser 1354.17\nsys 26.19\npercent 271%\n",device:"# Generated by pipen-runinfo v0.8.2\n\nScheduler\n---------\nlocal\n\nHostname\n--------\nmforgehn1.mayo.edu\n\nCPU\n----\nArchitecture:                         x86_64\nCPU op-mode(s):                       32-bit, 64-bit\nByte Order:                           Little Endian\nAddress sizes:                        46 bits physical, 57 bits virtual\nCPU(s):                               96\nOn-line CPU(s) list:                  0-95\nThread(s) per core:                   2\nCore(s) per socket:                   24\nSocket(s):                            2\nNUMA node(s):                         2\nVendor ID:                            GenuineIntel\nCPU family:                           6\nModel:                                106\nModel name:                           Intel(R) Xeon(R) Gold 6342 CPU @ 2.80GHz\nStepping:                             6\nCPU MHz:                              3500.000\nCPU max MHz:                          3500.0000\nCPU min MHz:                          800.0000\nBogoMIPS:                             5600.00\nVirtualization:                       VT-x\nL1d cache:                            2.3 MiB\nL1i cache:                            1.5 MiB\nL2 cache:                             60 MiB\nL3 cache:                             72 MiB\nNUMA node0 CPU(s):                    0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48,50,52,54,56,58,60,62,64,66,68,70,72,74,76,78,80,82,84,86,88,90,92,94\nNUMA node1 CPU(s):                    1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,45,47,49,51,53,55,57,59,61,63,65,67,69,71,73,75,77,79,81,83,85,87,89,91,93,95\nVulnerability Gather data sampling:   Mitigation; Microcode\nVulnerability Itlb multihit:          Not affected\nVulnerability L1tf:                   Not affected\nVulnerability Mds:                    Not affected\nVulnerability Meltdown:               Not affected\nVulnerability Mmio stale data:        Mitigation; Clear CPU buffers; SMT vulnerable\nVulnerability Reg file data sampling: Not affected\nVulnerability Retbleed:               Not affected\nVulnerability Spec rstack overflow:   Not affected\nVulnerability Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl\nVulnerability Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization\nVulnerability Spectre v2:             Mitigation; Enhanced / Automatic IBRS; IBPB conditional; RSB filling; PBRSB-eIBRS SW sequence; BHI SW loop, KVM SW loop\nVulnerability Srbds:                  Not affected\nVulnerability Tsx async abort:        Not affected\nFlags:                                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 invpcid_single intel_ppin ssbd mba ibrs ibpb stibp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm rdt_a avx512f avx512dq rdseed adx smap avx512ifma clflushopt clwb intel_pt avx512cd sha_ni avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local split_lock_detect wbnoinvd dtherm ida arat pln pts avx512vbmi umip pku ospke avx512_vbmi2 gfni vaes vpclmulqdq avx512_vnni avx512_bitalg tme avx512_vpopcntdq la57 rdpid fsrm md_clear pconfig flush_l1d arch_capabilities\n\nMemory\n------\n"}},{name:"CloneResidency",desc:"Identification of clone residency",npages:1,report_toc:!0,order:10,runinfo:{session:"# Generated by pipen_runinfo v0.8.2\n# Lang: R\nR version 4.3.3 (2024-02-29)\nPlatform: x86_64-conda-linux-gnu (64-bit)\nRunning under: Debian GNU/Linux 11 (bullseye)\n\nMatrix products: default\nBLAS/LAPACK: /opt/conda/lib/libopenblasp-r0.3.28.so;  LAPACK version 3.12.0\n\nlocale:\n [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       \n [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   \n [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          \n[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   \n\ntime zone: Etc/UTC\ntzcode source: system (glibc)\n\nattached base packages:\n[1] stats     graphics  grDevices utils     datasets  methods   base     \n\nother attached packages:\n [1] ComplexUpset_1.3.3  ggVennDiagram_1.2.3 ggprism_1.0.5      \n [4] ggplot2_3.5.1       ggnewscale_0.5.0    glue_1.8.0         \n [7] tibble_3.2.1        tidyr_1.3.1         dplyr_1.1.4        \n[10] jsonlite_1.8.9      rlang_1.1.5         logger_0.4.0       \n\nloaded via a namespace (and not attached):\n [1] gtable_0.3.6       compiler_4.3.3     Rcpp_1.0.14        tidyselect_1.2.1  \n [5] scales_1.3.0       R6_2.6.0           labeling_0.4.3     generics_0.1.3    \n [9] patchwork_1.3.0    classInt_0.4-11    sf_1.0-19          units_0.8-5       \n[13] munsell_0.5.1      DBI_1.2.3          RColorBrewer_1.1-3 pillar_1.10.1     \n[17] RVenn_1.1.0        cli_3.6.4          withr_3.0.2        magrittr_2.0.3    \n[21] class_7.3-23       digest_0.6.37      grid_4.3.3         lifecycle_1.0.4   \n[25] vctrs_0.6.5        KernSmooth_2.23-26 proxy_0.4-27       farver_2.1.2      \n[29] e1071_1.7-16       colorspace_2.1-1   purrr_1.0.2        tools_4.3.3       \n[33] pkgconfig_2.0.3   \n",time:"# Generated by pipen-runinfo v0.8.2\n\nreal 173.62\nuser 178.49\nsys 3.50\npercent 104%\n",device:"# Generated by pipen-runinfo v0.8.2\n\nScheduler\n---------\nlocal\n\nHostname\n--------\nmforgehn1.mayo.edu\n\nCPU\n----\nArchitecture:                         x86_64\nCPU op-mode(s):                       32-bit, 64-bit\nByte Order:                           Little Endian\nAddress sizes:                        46 bits physical, 57 bits virtual\nCPU(s):                               96\nOn-line CPU(s) list:                  0-95\nThread(s) per core:                   2\nCore(s) per socket:                   24\nSocket(s):                            2\nNUMA node(s):                         2\nVendor ID:                            GenuineIntel\nCPU family:                           6\nModel:                                106\nModel name:                           Intel(R) Xeon(R) Gold 6342 CPU @ 2.80GHz\nStepping:                             6\nCPU MHz:                              3300.000\nCPU max MHz:                          3500.0000\nCPU min MHz:                          800.0000\nBogoMIPS:                             5600.00\nVirtualization:                       VT-x\nL1d cache:                            2.3 MiB\nL1i cache:                            1.5 MiB\nL2 cache:                             60 MiB\nL3 cache:                             72 MiB\nNUMA node0 CPU(s):                    0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48,50,52,54,56,58,60,62,64,66,68,70,72,74,76,78,80,82,84,86,88,90,92,94\nNUMA node1 CPU(s):                    1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,45,47,49,51,53,55,57,59,61,63,65,67,69,71,73,75,77,79,81,83,85,87,89,91,93,95\nVulnerability Gather data sampling:   Mitigation; Microcode\nVulnerability Itlb multihit:          Not affected\nVulnerability L1tf:                   Not affected\nVulnerability Mds:                    Not affected\nVulnerability Meltdown:               Not affected\nVulnerability Mmio stale data:        Mitigation; Clear CPU buffers; SMT vulnerable\nVulnerability Reg file data sampling: Not affected\nVulnerability Retbleed:               Not affected\nVulnerability Spec rstack overflow:   Not affected\nVulnerability Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl\nVulnerability Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization\nVulnerability Spectre v2:             Mitigation; Enhanced / Automatic IBRS; IBPB conditional; RSB filling; PBRSB-eIBRS SW sequence; BHI SW loop, KVM SW loop\nVulnerability Srbds:                  Not affected\nVulnerability Tsx async abort:        Not affected\nFlags:                                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 invpcid_single intel_ppin ssbd mba ibrs ibpb stibp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm rdt_a avx512f avx512dq rdseed adx smap avx512ifma clflushopt clwb intel_pt avx512cd sha_ni avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local split_lock_detect wbnoinvd dtherm ida arat pln pts avx512vbmi umip pku ospke avx512_vbmi2 gfni vaes vpclmulqdq avx512_vnni avx512_bitalg tme avx512_vpopcntdq la57 rdpid fsrm md_clear pconfig flush_l1d arch_capabilities\n\nMemory\n------\n"}},{name:"ClusterMarkers",desc:"Markers for clusters of all or selected T cells.",npages:3,report_toc:!0,order:11,runinfo:{session:"# Generated by pipen_runinfo v0.8.2\n# Lang: R\nR version 4.3.3 (2024-02-29)\nPlatform: x86_64-conda-linux-gnu (64-bit)\nRunning under: Debian GNU/Linux 11 (bullseye)\n\nMatrix products: default\nBLAS/LAPACK: /opt/conda/lib/libopenblasp-r0.3.28.so;  LAPACK version 3.12.0\n\nlocale:\n [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       \n [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   \n [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          \n[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   \n\ntime zone: Etc/UTC\ntzcode source: system (glibc)\n\nattached base packages:\n[1] stats     graphics  grDevices utils     datasets  methods   base     \n\nother attached packages:\n [1] UpSetR_1.4.0        ggVennDiagram_1.2.3 tidyseurat_0.8.0   \n [4] ttservice_0.4.1     future_1.34.0       ggrepel_0.9.6      \n [7] ggprism_1.0.5       ggplot2_3.5.1       enrichR_3.4        \n[10] Seurat_5.1.0        SeuratObject_5.0.2  sp_2.2-0           \n[13] tibble_3.2.1        tidyr_1.3.1         dplyr_1.1.4        \n[16] tidyselect_1.2.1    digest_0.6.37       jsonlite_1.8.9     \n[19] rlang_1.1.5         logger_0.4.0       \n\nloaded via a namespace (and not attached):\n  [1] deldir_2.0-4           pbapply_1.7-2          gridExtra_2.3         \n  [4] magrittr_2.0.3         RcppAnnoy_0.0.22       matrixStats_1.1.0     \n  [7] ggridges_0.5.6         compiler_4.3.3         spatstat.geom_3.3-5   \n [10] png_0.1-8              vctrs_0.6.5            reshape2_1.4.4        \n [13] stringr_1.5.1          pkgconfig_2.0.3        fastmap_1.2.0         \n [16] ellipsis_0.3.2         labeling_0.4.3         promises_1.3.2        \n [19] purrr_1.0.2            WriteXLS_6.7.0         RVenn_1.1.0           \n [22] goftest_1.2-3          later_1.4.1            spatstat.utils_3.1-2  \n [25] irlba_2.3.5.1          parallel_4.3.3         cluster_2.1.8         \n [28] R6_2.6.0               ica_1.0-3              stringi_1.8.4         \n [31] RColorBrewer_1.1-3     spatstat.data_3.1-4    limma_3.58.1          \n [34] reticulate_1.40.0      parallelly_1.42.0      spatstat.univar_3.1-1 \n [37] lmtest_0.9-40          scattermore_1.2        Rcpp_1.0.14           \n [40] tensor_1.5             future.apply_1.11.3    zoo_1.8-12            \n [43] sctransform_0.4.1      httpuv_1.6.15          Matrix_1.6-5          \n [46] splines_4.3.3          igraph_2.0.3           abind_1.4-5           \n [49] spatstat.random_3.3-2  codetools_0.2-20       miniUI_0.1.1.1        \n [52] spatstat.explore_3.3-4 curl_6.0.1             listenv_0.9.1         \n [55] lattice_0.22-6         plyr_1.8.9             withr_3.0.2           \n [58] shiny_1.10.0           ROCR_1.0-11            Rtsne_0.17            \n [61] fastDummies_1.7.5      survival_3.8-3         polyclip_1.10-7       \n [64] fitdistrplus_1.2-2     pillar_1.10.1          KernSmooth_2.23-26    \n [67] plotly_4.10.4          generics_0.1.3         RcppHNSW_0.6.0        \n [70] munsell_0.5.1          scales_1.3.0           globals_0.16.3        \n [73] xtable_1.8-4           glue_1.8.0             lazyeval_0.2.2        \n [76] tools_4.3.3            data.table_1.16.4      RSpectra_0.16-2       \n [79] RANN_2.6.2             leiden_0.4.3.1         dotCall64_1.2         \n [82] cowplot_1.1.3          grid_4.3.3             colorspace_2.1-1      \n [85] nlme_3.1-167           patchwork_1.3.0        presto_1.0.0          \n [88] cli_3.6.4              spatstat.sparse_3.1-0  fansi_1.0.6           \n [91] spam_2.11-1            viridisLite_0.4.2      uwot_0.2.2            \n [94] gtable_0.3.6           progressr_0.15.1       rjson_0.2.23          \n [97] htmlwidgets_1.6.4      farver_2.1.2           htmltools_0.5.8.1     \n[100] lifecycle_1.0.4        httr_1.4.7             statmod_1.5.0         \n[103] mime_0.12              MASS_7.3-60.0.1       \n",time:"# Generated by pipen-runinfo v0.8.2\n\nreal 241.42\nuser 193.57\nsys 7.97\npercent 83%\n",device:"# Generated by pipen-runinfo v0.8.2\n\nScheduler\n---------\nlocal\n\nHostname\n--------\nmforgehn1.mayo.edu\n\nCPU\n----\nArchitecture:                         x86_64\nCPU op-mode(s):                       32-bit, 64-bit\nByte Order:                           Little Endian\nAddress sizes:                        46 bits physical, 57 bits virtual\nCPU(s):                               96\nOn-line CPU(s) list:                  0-95\nThread(s) per core:                   2\nCore(s) per socket:                   24\nSocket(s):                            2\nNUMA node(s):                         2\nVendor ID:                            GenuineIntel\nCPU family:                           6\nModel:                                106\nModel name:                           Intel(R) Xeon(R) Gold 6342 CPU @ 2.80GHz\nStepping:                             6\nCPU MHz:                              3300.000\nCPU max MHz:                          3500.0000\nCPU min MHz:                          800.0000\nBogoMIPS:                             5600.00\nVirtualization:                       VT-x\nL1d cache:                            2.3 MiB\nL1i cache:                            1.5 MiB\nL2 cache:                             60 MiB\nL3 cache:                             72 MiB\nNUMA node0 CPU(s):                    0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48,50,52,54,56,58,60,62,64,66,68,70,72,74,76,78,80,82,84,86,88,90,92,94\nNUMA node1 CPU(s):                    1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,45,47,49,51,53,55,57,59,61,63,65,67,69,71,73,75,77,79,81,83,85,87,89,91,93,95\nVulnerability Gather data sampling:   Mitigation; Microcode\nVulnerability Itlb multihit:          Not affected\nVulnerability L1tf:                   Not affected\nVulnerability Mds:                    Not affected\nVulnerability Meltdown:               Not affected\nVulnerability Mmio stale data:        Mitigation; Clear CPU buffers; SMT vulnerable\nVulnerability Reg file data sampling: Not affected\nVulnerability Retbleed:               Not affected\nVulnerability Spec rstack overflow:   Not affected\nVulnerability Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl\nVulnerability Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization\nVulnerability Spectre v2:             Mitigation; Enhanced / Automatic IBRS; IBPB conditional; RSB filling; PBRSB-eIBRS SW sequence; BHI SW loop, KVM SW loop\nVulnerability Srbds:                  Not affected\nVulnerability Tsx async abort:        Not affected\nFlags:                                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 invpcid_single intel_ppin ssbd mba ibrs ibpb stibp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm rdt_a avx512f avx512dq rdseed adx smap avx512ifma clflushopt clwb intel_pt avx512cd sha_ni avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local split_lock_detect wbnoinvd dtherm ida arat pln pts avx512vbmi umip pku ospke avx512_vbmi2 gfni vaes vpclmulqdq avx512_vnni avx512_bitalg tme avx512_vpopcntdq la57 rdpid fsrm md_clear pconfig flush_l1d arch_capabilities\n\nMemory\n------\n"}},{name:"CellsDistribution",desc:"Distribution of cells (i.e. in a TCR clone) from different groups for each cluster",npages:1,report_toc:!0,order:12}]};function un(e){let t,a;return t=new aa({props:{dir:"up"}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function pn(e){let t,a;return t=new aa({props:{dir:"down"}}),{c(){m(t.$$.fragment)},m(e,l){A(t,e,l),a=!0},i(e){a||(r(t.$$.fragment,e),a=!0)},o(e){c(t.$$.fragment,e),a=!1},d(e){g(t,e)}}}function vn(e){let t,a,l,n,_,d=e[1]>0&&un();a=new rn({});let p=e[1]<e[3].npages-1&&pn();return{c(){d&&d.c(),t=C(),m(a.$$.fragment),l=C(),p&&p.c(),n=P()},m(e,r){d&&d.m(e,r),s(e,t,r),A(a,e,r),s(e,l,r),p&&p.m(e,r),s(e,n,r),_=!0},p(e,a){e[1]>0?d?2&a&&r(d,1):(d=un(),d.c(),r(d,1),d.m(t.parentNode,t)):d&&(i(),c(d,1,1,(()=>{d=null})),o()),e[1]<e[3].npages-1?p?10&a&&r(p,1):(p=pn(),p.c(),r(p,1),p.m(n.parentNode,n)):p&&(i(),c(p,1,1,(()=>{p=null})),o())},i(e){_||(r(d),r(a.$$.fragment,e),r(p),_=!0)},o(e){c(d),c(a.$$.fragment,e),c(p),_=!1},d(e){e&&(u(t),u(l),u(n)),d&&d.d(e),g(a,e),p&&p.d(e)}}}function Pn(e){let t,a,l;function n(t){e[4](t)}let _={logo:e[0],logotext:e[3].desc,versions:cn.versions,entries:cn.entries,pipeline_name:cn.pipeline.name,report_toc:e[3].report_toc,runinfo:e[3].runinfo,toc:on,page:e[1],$$slots:{default:[vn]},$$scope:{ctx:e}};return void 0!==e[2]&&(_.isSideNavOpen=e[2]),t=new Zt({props:_}),O.push((()=>R(t,"isSideNavOpen",n))),{c(){m(t.$$.fragment)},m(e,a){A(t,e,a),l=!0},p(e,[l]){const n={};1&l&&(n.logo=e[0]),8&l&&(n.logotext=e[3].desc),8&l&&(n.report_toc=e[3].report_toc),8&l&&(n.runinfo=e[3].runinfo),2&l&&(n.page=e[1]),42&l&&(n.$$scope={dirty:l,ctx:e}),!a&&4&l&&(a=!0,n.isSideNavOpen=e[2],f((()=>a=!1))),t.$set(n)},i(e){l||(r(t.$$.fragment,e),l=!0)},o(e){c(t.$$.fragment,e),l=!1},d(e){g(t,e)}}}function mn(e,t,a){let l,{name:n}=t,{page:_=0}=t,d=!1;for(const e of cn.entries)if(e.procs){for(const t of e.procs)if(t.name==n){l=t;break}}else if(e.name==n){l=e;break}return v((()=>{const e=new URL(window.location.href);let t;e.hash.length>1&&(t=document.getElementById(`${e.hash.slice(1)}`)),t&&t.scrollIntoView({block:"start",behavior:"smooth"})})),e.$$set=e=>{"name"in e&&a(0,n=e.name),"page"in e&&a(1,_=e.page)},[n,_,d,l,function(e){d=e,a(2,d)}]}e("default",new class extends t{constructor(e){super(),a(this,e,mn,Pn,l,{name:0,page:1})}}({target:document.body,props:{name:"ClusterMarkers",page:2}}))}}}));
